{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "4807ac91",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import shutil\n",
    "import tempfile\n",
    "from tqdm import tqdm\n",
    "import json\n",
    "import re\n",
    "from collections import defaultdict\n",
    "from rank_bm25 import BM25Okapi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "3bef030c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'qid': '67e6ce6e18b1e36f2e0000cf',\n",
       " 'question': 'How many primary genetic associations were identified through pQTL mapping within the Pharma Proteomics Project?',\n",
       " 'all_retreived_articles': [{'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36865558',\n",
       "   'title': 'Integration of ',\n",
       "   'abstract': 'Both gene expression and protein concentrations are regulated by genetic variants. Exploring the regulation of both eQTLs and pQTLs simultaneously in a context- and cell-type dependent manner may help to unravel mechanistic basis for genetic regulation of pQTLs. Here, we performed meta-analysis of '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39418314',\n",
       "   'title': 'Evaluating the Causal Effect of Circulating Proteome on Glycemic Traits: Evidence From Mendelian Randomization.',\n",
       "   'abstract': 'Exploring the mechanisms underlying abnormal glycemic traits is important for deciphering type 2 diabetes and characterizing novel drug targets. This study aimed to decipher the causal associations of circulating proteins with fasting glucose (FG), 2-h glucose after an oral glucose challenge (2hGlu), fasting insulin (FI), and glycated hemoglobin (HbA1c) using large-scale proteome-wide Mendelian randomization (MR) analyses. Genetic data on plasma proteomes were obtained from 10 proteomic genome-wide association studies. Both cis-protein quantitative trait loci (pQTLs) and cis + trans-pQTLs MR analyses were conducted. Bayesian colocalization, Steiger filtering analysis, assessment of protein-altering variants, and mapping expression QTLs to pQTLs were performed to investigate the reliability of the MR findings. Protein-protein interaction, pathway enrichment analysis, and evaluation of drug targets were performed. Thirty-three proteins were identified with causal effects on FG, FI, or HbA1c but not 2hGlu in the cis-pQTL analysis, and 93 proteins had causal effects on glycemic traits in the cis + trans-pQTLs analysis. Most proteins were either considered druggable or drug targets. In conclusion, many novel circulating protein biomarkers were identified to be causally associated with glycemic traits. These biomarkers enhance the understanding of molecular etiology and provide insights into the screening, monitoring, and treatment of diabetes.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40038097',\n",
       "   'title': 'Exploring druggable targets and inflammation-mediated pathways in cancer: a Mendelian randomization analysis integrating transcriptomic and proteomic data.',\n",
       "   'abstract': 'Cancer remains a predominant global health challenge, necessitating the ongoing exploration of novel biomarkers and therapeutic targets to improve diagnosis and treatment. By integrating expression quantitative trait loci (eQTL) and protein quantitative trait loci (pQTL) data with genome-wide association studies (GWAS) data, we performed Mendelian randomization (MR) analysis to identify potential druggable targets at the gene expression and protein levels for multiple cancers. We conducted mediation analysis to explore whether inflammatory factors mediate the pathways linking identified druggable targets to cancer. Phenome-wide MR analysis, drug prediction, and molecular docking were employed to evaluate the medicinal potential. We finally identified five druggable targets: CDKN1A, FES, and PDIA3 were associated with breast cancer, whereas TP53 and VAMP8 were associated with prostate cancer. Mediation analysis identified six inflammatory proteins as potential mediators in the causal pathways from these druggable targets to cancer: caspase 8, interleukin-1-alpha, C-X-C motif chemokine 1, C-C motif chemokine 23, TNF-related apoptosis-inducing ligand, and interleukin-6. Subsequent analyses further provided evidence supporting the pharmaceutical potential of these five targets. Our study identified five druggable targets causally associated with breast and prostate cancers, with six inflammatory proteins acting as potential mediators, providing novel insights into the treatment of these cancers.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/28129359',\n",
       "   'title': 'High-Throughput Characterization of Blood Serum Proteomics of IBD Patients with Respect to Aging and Genetic Factors.',\n",
       "   'abstract': \"To date, no large scale, systematic description of the blood serum proteome has been performed in inflammatory bowel disease (IBD) patients. By using microarray technology, a more complete description of the blood proteome of IBD patients is feasible. It may help to achieve a better understanding of the disease. We analyzed blood serum profiles of 1128 proteins in IBD patients of European descent (84 Crohn's Disease (CD) subjects and 88 Ulcerative Colitis (UC) subjects) as well as 15 healthy control subjects, and linked protein variability to patient age (all cohorts) and genetic components (genotype data generated from CD patients). We discovered new, previously unreported aging-associated proteomic traits (such as serum Albumin level), confirmed previously reported results from different tissues (i.e., upregulation of APOE with aging), and found loss of regulation of MMP7 in CD patients. In carrying out a genome wide genotype-protein association study (proteomic Quantitative Trait Loci, pQTL) within the CD patients, we identified 41 distinct proteomic traits influenced by cis pQTLs (underlying SNPs are referred to as pSNPs). Significant overlaps between pQTLs and cis eQTLs corresponding to the same gene were observed and in some cases the QTL were related to inflammatory disease susceptibility. Importantly, we discovered that serum protein levels of MST1 (Macrophage Stimulating 1) were regulated by SNP rs3197999 (p = 5.96E-10, FDR<5%), an accepted GWAS locus for IBD. Filling the knowledge gap of molecular mechanisms between GWAS hits and disease susceptibility requires systematically dissecting the impact of the locus at the cell, mRNA expression, and protein levels. The technology and analysis tools that are now available for large-scale molecular studies can elucidate how alterations in the proteome driven by genetic polymorphisms cause or provide protection against disease. Herein, we demonstrated this directly by integrating proteomic and pQTLs with existing GWAS, mRNA expression, and eQTL datasets to provide insights into the biological processes underlying IBD and pinpoint causal genetic variants along with their downstream molecular consequences.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39040677',\n",
       "   'title': 'Role of circulating inflammatory protein in the development of diabetic renal complications: proteome-wide Mendelian randomization and colocalization analyses.',\n",
       "   'abstract': 'Diabetes ranks among the most widespread diseases globally, with the kidneys being particularly susceptible to its vascular complications. The identification of proteins for pathogenesis and novel drug targets remains imperative. This study aims to investigate roles of circulating inflammatory proteins in diabetic renal complications. Data on the proteins were derived from a genome-wide protein quantitative trait locus (pQTL) study, while data on diabetic renal complications came from the FinnGen study. In this study, proteome-wide Mendelian randomization (MR) and colocalization analyses were used to assess the relationship between circulating inflammatory proteins and diabetic renal complications. MR approach indicated that elevated levels of interleukin 12B (IL-12B) (OR 1.691, 95%CI 1.179-2.427, P=4.34×10 Our study established associations between specific circulating inflammatory proteins and the risk of diabetic renal complications, suggesting these proteins as targets for further investigation into the pathogenesis and potential therapeutic interventions for T1D and T2D with renal complications.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35870639',\n",
       "   'title': 'Identification of 969 protein quantitative trait loci in an African American population with kidney disease attributed to hypertension.',\n",
       "   'abstract': 'Investigations into the causal underpinnings of disease processes can be aided by the incorporation of genetic information. Genetic studies require populations varied in both ancestry and prevalent disease in order to optimize discovery and ensure generalizability of findings to the global population. Here, we report the genetic determinants of the serum proteome in 466 African Americans with chronic kidney disease attributed to hypertension from the richly phenotyped African American Study of Kidney Disease and Hypertension (AASK) study. Using the largest aptamer-based protein profiling platform to date (6,790 proteins or protein complexes), we identified 969 genetic associations with 900 unique proteins; including 52 novel cis (local) associations and 379 novel trans (distant) associations. The genetic effects of previously published cis-protein quantitative trait loci (pQTLs) were found to be highly reproducible, and we found evidence that our novel genetic signals colocalize with gene expression and disease processes. Many trans- pQTLs were found to reflect associations mediated by the circulating cis protein, and the common trans-pQTLs are enriched for processes involving extracellular vesicles, highlighting a plausible mechanism for distal regulation of the levels of secreted proteins. Thus, our study generates a valuable resource of genetic associations linking variants to protein levels and disease in an understudied patient population to inform future studies of drug targets and physiology.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30874442',\n",
       "   'title': 'Omics and the Search for Blood Biomarkers in Chronic Obstructive Pulmonary Disease. Insights from COPDGene.',\n",
       "   'abstract': 'There is an unmet need for blood biomarkers in diagnosis and prognosis of chronic obstructive pulmonary disease (COPD). The search for these biomarkers has been revolutionized by high-throughput sequencing techniques and multiplex platforms that can measure thousands of gene transcripts, proteins, or metabolites. We review COPDGene (Genetic Epidemiology of COPD) project publications that include DNA methylation, transcriptomic, proteomic, and metabolomic blood biomarkers and discuss their impact on COPD. Key contributions from COPDGene include identification of DNA methylation effects from smoking and genetic variation, new transcriptomic signatures in the blood, identification of protein biomarkers associated with severity and progression (e.g., sRAGE [soluble receptor for advanced glycosylation end products], inflammatory cytokines IL-6 and IL-8), and identification of small molecules (ceramides and sphingomyelin) that may be pathogenic. COPDGene studies have revealed that some of the COPD genome-wide association study polymorphisms are strongly associated with blood biomarkers (e.g., rs2070600 in '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39145416',\n",
       "   'title': 'Mendelian randomization of plasma lipidome, inflammatory proteome and heart failure.',\n",
       "   'abstract': 'Heart failure (HF) is a global health issue, with lipid metabolism and inflammation critically implicated in its progression. This study harnesses cutting-edge, expanded genetic information for lipid and inflammatory protein profiles, employing Mendelian randomization (MR) to uncover genetic risk factors for HF. We assessed genetic susceptibility to HF across 179 lipidomes and 91 inflammatory proteins using instrumental variables (IVs) from recent genome-wide association studies (GWASs) and proteome-wide quantitative trait loci (pQTL) studies. GWASs involving 47\\xa0309 HF cases and 930\\xa0014 controls were obtained from the Heart Failure Molecular Epidemiology for Therapeutic Targets (HERMES) Consortium. Data on 179 lipids from 7174 individuals in a Finnish cohort and 91 inflammatory proteins from a European pQTL study involving 14\\xa0824 individuals are available in the HGRI-EBI catalogue. A two-sample MR approach evaluated the associations, and a two-step mediation analysis explored the mediation role of inflammatory proteins in the lipid-HF pathway. Sensitivity analyses, including MR-RAPS (robust adjusted profile score) and MR-Egger, ensured result robustness. Genetic IVs for 162 lipids and 74 inflammatory proteins were successfully identified. MR analysis revealed a genetic association between HF and 31 lipids. Among them, 18 lipids, including sterol ester (27:1/18:0), cholesterol, 9 phosphatidylcholines, phosphatidylinositol (16:0_20:4) and 6 triacylglycerols, were identified as HF risk factors [odds ratio (OR)\\xa0=\\xa01.037-1.368]. Cholesterol exhibited the most significant association with elevated HF risk [OR\\xa0=\\xa01.368, 95% confidence interval (CI)\\xa0=\\xa01.044-1.794, P\\xa0=\\xa00.023]. In the inflammatory proteome, leukaemia inhibitory factor receptor (OR\\xa0=\\xa00.841, 95% CI\\xa0=\\xa00.789-0.897, P\\xa0=\\xa01.08E-07), fibroblast growth factor 19 (OR\\xa0=\\xa00.905, 95% CI\\xa0=\\xa00.830-0.988, P\\xa0=\\xa00.025) and urokinase-type plasminogen activator (OR\\xa0=\\xa00.938, 95% CI\\xa0=\\xa00.886-0.994, P\\xa0=\\xa00.030) were causally negatively correlated with HF, whereas interleukin-20 receptor subunit alpha (OR\\xa0=\\xa01.333, 95% CI\\xa0=\\xa01.094-1.625, P\\xa0=\\xa00.004) was causally positively correlated with HF. Mediation analysis revealed leukaemia inhibitory factor receptor (mediation proportion: 23.5%-25.2%) and urokinase-type plasminogen activator (mediation proportion: 9.5%-10.7%) as intermediaries in the lipid-inflammation-HF pathway. No evidence of directional horizontal pleiotropy was observed (P\\xa0>\\xa00.05). This study identifies a genetic connection between certain lipids, particularly cholesterol, and HF, highlighting inflammatory proteins that influence HF risk and mediate this relationship, suggesting new therapeutic targets and insights into genetic drivers in HF.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39614878',\n",
       "   'title': 'Genetic insights into kidney stone formation: a Mendelian randomization study of protein quantitative trait loci.',\n",
       "   'abstract': 'Kidney stones are a common urological condition caused by a complex interaction of genetic, metabolic, and environmental factors. Recent genomic research has shed light on the genetic basis of kidney stone susceptibility.This study aims to identify protein quantitative trait loci (pQTL) associated with kidney stone formation and explore their causal relationships using Mendelian randomization.We conducted two-sample Mendelian Randomization (MR) analyses utilizing Genome-Wide Association Study (GWAS) summary data to assess the causal impact of pQTL on kidney stone formation. Data sources included the UK Biobank dataset \"ukb-b-8297\" and an external validation dataset \"ukb-b-13537\". We employed inverse variance weighting (IVW) as the primary MR method, supplemented by sensitivity analyses such as MR-PRESSO, Leave-One-Out, and Cochran Q tests to validate the robustness of our findings.Our analyses identified significant associations between several pQTL and kidney stones. Key proteins such as CD27, CXCL9, and TNFRSF1A exhibited significant centrality in the protein-protein interaction (PPI) network, suggesting their critical roles in kidney stone pathogenesis. The KEGG pathway enrichment analysis revealed significant pathways, including cytokine-cytokine receptor interaction and osteoclast differentiation, highlighting the involvement of immune response and inflammatory processes in kidney stone formation.This study underscores the significance of pQTL in kidney stone research, identifying key proteins and pathways that may serve as biomarkers or therapeutic targets. The findings provide insights into the genetic and molecular mechanisms underlying kidney stone formation, offering potential avenues for future research and therapeutic interventions.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38103512',\n",
       "   'title': 'Multi-omic insight into the molecular networks of mitochondrial dysfunction in the pathogenesis of inflammatory bowel disease.',\n",
       "   'abstract': \"Mitochondrial dysfunction has been linked to the development of inflammatory bowel disease (IBD), but the genetic pathophysiology was not fully elucidated. We employed Mendelian randomization and colocalization analyses to investigate the associations between mitochondrial-related genes and IBD via integrating multi-omics. Summary-level data of mitochondrial gene methylation, expression and protein abundance levels were obtained from corresponding methylation, expression and protein quantitative trait loci studies, respectively. We obtained genetic associations with IBD and its two subtypes from the Inflammatory Bowel Disease Genetics Consortium (discovery), the UK Biobank (replication), and the FinnGen study (replication). We performed summary-data-based Mendelian randomization analysis to assess the associations of mitochondrial gene-related molecular features with IBD. Colocalization analysis was further conducted to assess whether the identified signal pairs shared a causal genetic variant. After integrating the multi-omics data between mQTL-eQTL and eQTL-pQTL, we identified two mitochondrial genes, i.e., PARK7 and ACADM, with tier 1 evidence for their associations with IBD and ulcerative colitis (UC). PDK1 and FISI genes were associated with UC risk with tier 2 and tier 3 evidence, respectively. The methylation of cg05467918 in ACADM was associated with lower expression of ACADM, which fits with the positive effect of cg05467918 methylation on UC risk. Consistently, the inverse associations between gene methylation and gene expression were also observed in PARK7 (cg10385390) and PDK1 (cg17679246), which were corroborated with the protective role in UC. At circulating protein level, genetically predicted higher levels of PARK7 (OR 0.36, 95% CI 0.25-0.52) and HINT1 (OR 0.47, 95% CI 0.30-0.74) were inversely associated with IBD risk; genetically predicted higher level of HINT1 was associated with a decreased risk of Crohn's disease (CD) (OR 0.26, 95% CI 0.14-0.49) and a higher level of ACADM (OR 0.67, 95% CI 0.55-0.83), PDK1 (OR 0.63, 95% CI 0.49-0.81), FIS1 (OR 0.63, 95% CI 0.47-0.83) was associated with a decreased risk of UC. We found that the mitochondrial PARK7 gene was putatively associated with IBD risk, and mitochondrial FIS1, PDK1, and ACADM genes were associated with UC risk with evidence from multi-omics levels. This study identified mitochondrial genes in relation to IBD, which may enhance the understanding of the pathogenic mechanisms of IBD development. XL is supported by the Natural Science Fund for Distinguished Young Scholars of Zhejiang Province (LR22H260001) and Healthy Zhejiang One Million People Cohort (K-20230085).\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38780787',\n",
       "   'title': 'Dynamic changes in RNA m',\n",
       "   'abstract': 'RNA modifications are essential for the establishment of cellular identity. Although increasing evidence indicates that RNA modifications regulate the innate immune response, their role in monocyte-to-macrophage differentiation and polarisation is unclear. While m'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40286918',\n",
       "   'title': 'Multi-omics reveal the role of nociception-related genes TNXB, CTNND1 and CBL in depression.',\n",
       "   'abstract': 'Recent studies have suggested a link between nociception and depression. However, the specific genes involved remain unclear. This study investigates this genetic link using multi-omics data. We collected nociception-related genes from the GeneCards database and integrated quantitative trait loci (mQTLs, eQTLs and pQTLs) data for gene expression, DNA methylation and protein expression. GWAS data from the IEU database served as the discovery cohort for depression, with FinnGen and GWAS Catalog data used for validation. Summary data-based Mendelian Randomization (SMR) analysis was employed to examine the interactions between nociception-related genes and depression, and colocalization analysis identified shared causal variants. The associations between depression and target gene expression in specific tissues and specific cell types were assessed using the GTEx v8 dataset and single-cell eQTL data. SMR analysis revealed 215 mQTLs, 12 eQTLs, and 1 pQTL associated with depression in the discovery cohort. By integrating multi-omics evidence, we found that the hypermethylation of the TNXB gene (cg02272968, cg02432444, cg27624229) and the hypomethylation of the CTNND1 gene (cg16127573) and the P2RY6 gene (cg12889420) were found to upregulate their expression, potentially increasing the risk of depression. GTEx eQTL analysis confirmed CBL expression in the substantia nigra positively correlates with depression risk. However, none of the key genes were confirmed in the single-cell eQTL analysis. Our study emphasizes the importance of nociception-related genes, particularly TNXB, CTNND1 and CBL in the pathogenesis of depression. Future research should build on these findings for potential prevention and treatment strategies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37148340',\n",
       "   'title': 'Identifying novel genes for amyotrophic lateral sclerosis by integrating human brain proteomes with genome-wide association data.',\n",
       "   'abstract': 'Genome-Wide Association Studies (GWAS) have identified numerous risk genes for Amyotrophic Lateral Sclerosis (ALS); however, the mechanisms by which these loci confer ALS risk are uncertain. This study aims to identify novel causal proteins in the brains of patients with ALS using an integrative analytical pipeline. Using the datasets of Protein Quantitative Trait Loci (pQTL) (N Using PWAS, we found that the altered protein abundance of 12 genes in the brain was associated with ALS. Three genes (SCFD1, SARM1 and CAMLG) were identified as lead causal genes for ALS with solid evidence (False discovery rate\\u2009<\\u20090.05, in MR analysis; PPH4\\u2009>\\u200980% for Bayesian colocalization). Specifically, an increased abundance of SCFD1 and CAMLG led to an increased risk of ALS, whereas a higher abundance of SARM1 led to a decreased risk of developing ALS. TWAS showed that SCFD1 and CAMLG were related to ALS at the transcriptional level. SCFD1, CAMLG, and SARM1 exhibited robust associations and causality with ALS. The study findings provide novel clues for identifying potential therapeutic targets in ALS. Further studies are required to explore the mechanisms underlying the identified genes.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32895551',\n",
       "   'title': 'Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases.',\n",
       "   'abstract': 'The human proteome is a major source of therapeutic targets. Recent genetic association analyses of the plasma proteome enable systematic evaluation of the causal consequences of variation in plasma protein levels. Here we estimated the effects of 1,002 proteins on 225 phenotypes using two-sample Mendelian randomization (MR) and colocalization. Of 413 associations supported by evidence from MR, 130 (31.5%) were not supported by results of colocalization analyses, suggesting that genetic confounding due to linkage disequilibrium is widespread in naïve phenome-wide association studies of proteins. Combining MR and colocalization evidence in cis-only analyses, we identified 111 putatively causal effects between 65 proteins and 52 disease-related phenotypes ( https://www.epigraphdb.org/pqtl/ ). Evaluation of data from historic drug development programs showed that target-indication pairs with MR and colocalization support were more likely to be approved, evidencing the value of this approach in identifying and prioritizing potential therapeutic targets.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30606113',\n",
       "   'title': 'Genetic control of longissimus dorsi muscle gene expression variation and joint analysis with phenotypic quantitative trait loci in pigs.',\n",
       "   'abstract': 'Economically important growth and meat quality traits in pigs are controlled by cascading molecular events occurring during development and continuing throughout the conversion of muscle to meat. However, little is known about the genes and molecular mechanisms involved in this process. Evaluating transcriptomic profiles of skeletal muscle during the initial steps leading to the conversion of muscle to meat can identify key regulators of polygenic phenotypes. In addition, mapping transcript abundance through genome-wide association analysis using high-density marker genotypes allows identification of genomic regions that control gene expression, referred to as expression quantitative trait loci (eQTL). In this study, we perform eQTL analyses to identify potential candidate genes and molecular markers regulating growth and meat quality traits in pigs. Messenger RNA transcripts obtained with RNA-seq of longissimus dorsi muscle from 168\\u2009F2 animals from a Duroc x Pietrain pig resource population were used to estimate gene expression variation subject to genetic control by mapping eQTL. A total of 339 eQTL were mapped (FDR\\u2009≤\\u20090.01) with 191 exhibiting local-acting regulation. Joint analysis of eQTL with phenotypic QTL (pQTL) segregating in our population revealed 16 genes significantly associated with 21 pQTL for meat quality, carcass composition and growth traits. Ten of these pQTL were for meat quality phenotypes that co-localized with one eQTL on SSC2 (8.8-Mb region) and 11 eQTL on SSC15 (121-Mb region). Biological processes identified for co-localized eQTL genes include calcium signaling (FERM, MRLN, PKP2 and CHRNA9), energy metabolism (SUCLG2 and PFKFB3) and redox hemostasis (NQO1 and CEP128), and results support an important role for activation of the PI3K-Akt-mTOR signaling pathway during the initial conversion of muscle to meat. Co-localization of eQTL with pQTL identified molecular markers significantly associated with both economically important phenotypes and gene transcript abundance. This study reveals candidate genes contributing to variation in pig production traits, and provides new knowledge regarding the genetic architecture of meat quality phenotypes.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30026924',\n",
       "   'title': 'BED: a Biological Entity Dictionary based on a graph data model.',\n",
       "   'abstract': 'The understanding of molecular processes involved in a specific biological system can be significantly improved by combining and comparing different data sets and knowledge resources. However, these information sources often use different identification systems and an identifier conversion step is required before any integration effort. Mapping between identifiers is often provided by the reference information resources and several tools have been implemented to simplify their use. However, most of these tools do not combine the information provided by individual resources to increase the completeness of the mapping process. Also, deprecated identifiers from former versions of databases are not taken into account. Finally, finding automatically the most relevant path to map identifiers from one scope to the other is often not trivial. The Biological Entity Dictionary (BED) addresses these three challenges by relying on a graph data model describing possible relationships between entities and their identifiers. This model has been implemented using Neo4j and an R package provides functions to query the graph but also to create and feed a custom instance of the database. This design combined with a local installation of the graph database and a cache system make BED very efficient to convert large lists of identifiers.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40149725',\n",
       "   'title': 'Multi-Omics Integration Reveals Mitochondrial Gene Regulation as a Determinant of Tuberculosis Susceptibility: A Mendelian Randomization Approach.',\n",
       "   'abstract': ''},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40217113',\n",
       "   'title': 'Error-corrected flow-based sequencing at whole-genome scale and its application to circulating cell-free DNA profiling.',\n",
       "   'abstract': 'Differentiating sequencing errors from true variants is a central genomics challenge, calling for error suppression strategies that balance costs and sensitivity. For example, circulating cell-free DNA (ccfDNA) sequencing for cancer monitoring is limited by sparsity of circulating tumor DNA, abundance of genomic material in samples and preanalytical error rates. Whole-genome sequencing (WGS) can overcome the low abundance of ccfDNA by integrating signals across the mutation landscape, but higher costs limit its wide adoption. Here, we applied deep (~120×) lower-cost WGS (Ultima Genomics) for tumor-informed circulating tumor DNA detection within the part-per-million range. We further leveraged lower-cost sequencing by developing duplex error-corrected WGS of ccfDNA, achieving 7.7\\u2009×\\u200910'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38172904',\n",
       "   'title': \"Proteo-genomics of soluble TREM2 in cerebrospinal fluid provides novel insights and identifies novel modulators for Alzheimer's disease.\",\n",
       "   'abstract': \"Triggering receptor expressed on myeloid cells 2 (TREM2) plays a critical role in microglial activation, survival, and apoptosis, as well as in Alzheimer's disease (AD) pathogenesis. We previously reported the MS4A locus as a key modulator for soluble TREM2 (sTREM2) in cerebrospinal fluid (CSF). To identify additional novel genetic modifiers of sTREM2, we performed the largest genome-wide association study (GWAS) and identified four loci for CSF sTREM2 in 3,350 individuals of European ancestry. Through multi-ethnic fine mapping, we identified two independent missense variants (p.M178V in MS4A4A and p.A112T in MS4A6A) that drive the association in MS4A locus and showed an epistatic effect for sTREM2 levels and AD risk. The novel TREM2 locus on chr 6 contains two rare missense variants (rs75932628 p.R47H, P=7.16×10\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39794838',\n",
       "   'title': \"Association of antihypertensive drug target genes with alzheimer's disease: a mendelian randomization study.\",\n",
       "   'abstract': \"Epidemiological and genetic studies have elucidated associations between antihypertensive medication and Alzheimer's disease (AD), with the directionality of these associations varying upon the specific class of antihypertensive agents. Genetic instruments for the expression of antihypertensive drug target genes were identified using expression quantitative trait loci (eQTL) in blood, which are associated with systolic blood pressure (SBP). Exposure was derived from existing eQTL data in blood from the eQTLGen consortium and in the brain from the PsychENCODE and subsequently replicated in GTEx V8 and BrainMeta V2. We performed two-sample Mendelian randomization (MR) to estimate the potential effect of different antihypertensive drug classes on AD using summary statistics from a meta-analysis (111,326 cases and 677,663 controls) and further replicated in FinnGen cohorts (9301 cases and 367,976 controls). The reverse causality detection, assessing horizontal pleiotropy, Bayesian co-localization, phenotype scanning, and protein quantitative trait loci (pQTL) analysis were implemented to consolidate the MR findings further. A 1-standard deviation (SD) lower expression of the angiotensin-converting enzyme (ACE) gene in blood was associated with a lower SBP of 3.92 (95% confidence interval (CI), 2.69-5.15) mmHg but an increased risk of AD (odds ratio (OR), 2.46; 95% CI, 1.82-3.33). A similar direction of association was also observed between ACE expression in prefrontal cortex (OR, 1.19; 95% CI, 1.10-1.28), frontal cortex (OR, 1.19; 95% CI, 1.11-1.27), cerebellum (OR, 1.13; 95% CI, 1.09-1.17), cortex (OR, 1.59; 95% CI, 1.33-1.28) and ACE protein levels in plasma (OR, 1.13; 95% CI, 1.09-1.17) and AD risk. Colocalization supports these results. Similar results were found in external validation. We found no evidence for an association between genetically estimated blood pressure (BP) and AD risk. There findings suggest an adverse association of lower ACE messenger RNA and protein levels with an elevated risk of AD, irrespective of its BP-lowering effects. These findings warrant greater pharmacovigilance and further investigation into the effect of ACE inhibitors, particularly those that are centrally acting, on neurodegenerative symptoms in patients with AD.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35931049',\n",
       "   'title': 'A multi-layer functional genomic analysis to understand noncoding genetic variation in lipids.',\n",
       "   'abstract': 'A major challenge of genome-wide association studies (GWASs) is to translate phenotypic associations into biological insights. Here, we integrate a large GWAS on blood lipids involving 1.6 million individuals from five ancestries with a wide array of functional genomic datasets to discover regulatory mechanisms underlying lipid associations. We first prioritize lipid-associated genes with expression quantitative trait locus (eQTL) colocalizations and then add chromatin interaction data to narrow the search for functional genes. Polygenic enrichment analysis across 697 annotations from a host of tissues and cell types confirms the central role of the liver in lipid levels and highlights the selective enrichment of adipose-specific chromatin marks in high-density lipoprotein cholesterol and triglycerides. Overlapping transcription factor (TF) binding sites with lipid-associated loci identifies TFs relevant in lipid biology. In addition, we present an integrative framework to prioritize causal variants at GWAS loci, producing a comprehensive list of candidate causal genes and variants with multiple layers of functional evidence. We highlight two of the prioritized genes, CREBRF and RRBP1, which show convergent evidence across functional datasets supporting their roles in lipid biology.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37415601',\n",
       "   'title': 'Identifying potential risk genes and pathways for neuropsychiatric and substance use disorders using intermediate molecular mediator information.',\n",
       "   'abstract': 'Neuropsychiatric and substance use disorders (NPSUDs) have a complex etiology that includes environmental and polygenic risk factors with significant cross-trait genetic correlations. Genome-wide association studies (GWAS) of NPSUDs yield numerous association signals. However, for most of these regions, we do not yet have a firm understanding of either the specific risk variants or the effects of these variants. Post-GWAS methods allow researchers to use GWAS summary statistics and molecular mediators (transcript, protein, and methylation abundances) infer the effect of these mediators on risk for disorders. One group of post-GWAS approaches is commonly referred to as transcriptome/proteome/methylome-wide association studies, which are abbreviated as T/P/MWAS (or collectively as XWAS). Since these approaches use biological mediators, the multiple testing burden is reduced to the number of genes (∼20,000) instead of millions of GWAS SNPs, which leads to increased signal detection. In this work, our aim is to uncover likely risk genes for NPSUDs by performing XWAS analyses in two tissues-blood and brain. First, to identify putative causal risk genes, we performed an XWAS using the Summary-data-based Mendelian randomization, which uses GWAS summary statistics, reference xQTL data, and a reference LD panel. Second, given the large comorbidities among NPSUDs and the shared cis-xQTLs between blood and the brain, we improved XWAS signal detection for underpowered analyses by performing joint concordance analyses between XWAS results i) across the two tissues and ii) across NPSUDs. All XWAS signals i) were adjusted for heterogeneity in dependent instruments (HEIDI) (non-causality) '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40170147',\n",
       "   'title': 'Cross-ancestry genome-wide association study identifies new susceptibility genes for preeclampsia.',\n",
       "   'abstract': 'Preeclampsia (PE) is a heterogeneous, multi-organ pregnancy disorder that poses a significant health burden globally, with its pathogenesis remaining unclear. This study aimed to identify novel susceptibility genes for PE through a cross-ancestry genome-wide association study (GWAS). We performed meta-analysis to summarize the PE GWAS data from the United Kingdom, Finland, and Japan. Subsequently, the multi-ancestry sum of the single-effects model was used to perform cross-ancestry fine-mapping. The functional mapping and annotation (FUMA)-expression quantitative trait loci (eQTL) mapping method, transcriptome-wide association study (TWAS)- functional summary-based imputation (FUSION) method, genome-wide complex trait analysis (GCTA)-multivariate set-based association test (mBAT)-combo method, and polygenic priority score (PoPS) method were employed to screen for candidate genes. We utilized biomarker expression level imputation using summary-level statistics (BLISS), based on summary-level protein quantitative trait loci (pQTL) data, to conduct a multi-ancestry proteome-wide association study (PWAS) analysis, followed by candidate drug prediction. Six novel susceptibility genes associated with PE risk were identified: NPPA, SWAP70, NPR3, FGF5, REPIN1, and ACAA1. High expression of the NPPA and SWAP70 and low expression of the remaining genes were associated with a reduced risk of PE. Furthermore, we identified drugs that target NPPA, NPR3, and REPIN1. Our study identified NPPA, SWAP70, NPR3, FGF5, REPIN1, and ACAA1 as novel genes whose predicted expression was linked to the risk of PE, offering new insights into the genetic framework of this condition.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36180795',\n",
       "   'title': 'Stroke genetics informs drug discovery and risk prediction across ancestries.',\n",
       "   'abstract': 'Previous genome-wide association studies (GWASs) of stroke\\xa0-\\xa0the second leading cause of death worldwide\\xa0-\\xa0were conducted predominantly in populations of European ancestry'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39434152',\n",
       "   'title': 'Causal and mediating effects of lipid and facial aging: association study integrating GWAS, eQTL, mQTL, and pQTL data.',\n",
       "   'abstract': 'Increasing evidence suggests a potential causal association between lipid levels and facial aging. The aim of this study was to investigate the relationship between levels of specific lipids and facial aging via Mendelian randomization methods. Additionally, this study aimed to identify mediators and explore relevant genes and drug targets. In this study, genome-wide association data on plasma lipids from 7,174 Finnish individuals in the UK Biobank were used. Two-sample Mendelian randomization was applied to assess the causal effects of specific lipids on facial aging. Sensitivity and pleiotropy analyses were conducted to ensure the robustness and reliability of the results. Multivariate Mendelian randomization was conducted to account for the potential impact of confounding factors. Furthermore, summary-data-based Mendelian randomization was used to identify relevant genes, which were validated through multiomics data. Finally, drug‒gene interactions were explored via molecular docking techniques. Two-sample Mendelian randomization analysis revealed a causal relationship between lipid levels and facial aging. According to the multivariate Mendelian randomization results, smoking was found to mediate this association, and these lipids remained significantly associated with facial aging, even after accounting for environmental confounders. Using summary-data-based Mendelian randomization, CYP21A2, CCND1, PSMA4, and MED1 were identified as potential gene targets, with MED1 further validated through pQTL and mQTL data. Additionally, the MED1 protein was found to bind spontaneously with astragalin, fenofibrate, and ginsenoside. The results revealed a causal relationship between lipid levels and facial aging, revealing key gene targets that were still significantly associated with facial aging after controlling for environmental confounders. Additionally, the interactions between MED1 and certain drugs may indicate potential pathways for therapeutic interventions related to facial aging.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39284832',\n",
       "   'title': 'Proteomic Mendelian randomization to identify protein biomarkers of telomere length.',\n",
       "   'abstract': 'Shortening of telomere length (TL) is correlated with many age-related disorders and is a hallmark of biological aging. This study used proteome-wide Mendelian randomization to identify the protein biomarkers associated with telomere length. Protein quantitative trait loci (pQTL) were derived from two studies, the deCODE Health study (4907 plasma proteins) and the UK Biobank Pharma Proteomics Project (2923 plasma proteins). Summary data from genome-wide association studies (GWAS) for TL were obtained from the UK Biobank (472,174 cases) and GWAS Catalog (418,401 cases). The association between proteins and TL was further assessed using colocalization and summary data-based Mendelian randomization (SMR) analyses. The protein-protein network, druggability assessment, and phenome-wide MR were used to further evaluate the potential biological effects, druggability, and safety of the target proteins. Proteome-wide MR analysis identified 22 plasma proteins that were causally associated with telomere length. Five of these proteins (APOE, SPRED2, MAX, RALY, and PSMB1) had the highest evidence of association with TL and should be prioritized. This study revealed telomere length-related protein biomarkers, providing new insights into the development of new treatment targets for chronic diseases and anti-aging intervention strategies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36747854',\n",
       "   'title': 'Harnessing Transcriptomic Signals for Amyotrophic Lateral Sclerosis to Identify Novel Drugs and Enhance Risk Prediction.',\n",
       "   'abstract': 'Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease. This study integrates the latest ALS genome-wide association study (GWAS) summary statistics with functional genomic annotations with the aim of providing mechanistic insights into ALS risk loci, inferring drug repurposing opportunities, and enhancing prediction of ALS risk and clinical characteristics. Genes associated with ALS were identified using GWAS summary statistic methodology including SuSiE SNP-based fine-mapping, and transcriptome- and proteome-wide association study (TWAS/PWAS) analyses. Using several approaches, gene associations were integrated with the DrugTargetor drug-gene interaction database to identify drugs that could be repurposed for the treatment of ALS. Furthermore, ALS gene associations from TWAS were combined with observed blood expression in two external ALS case-control datasets to calculate polytranscriptomic scores and evaluate their utility for prediction of ALS risk and clinical characteristics, including site of onset, age at onset, and survival. SNP-based fine-mapping, TWAS and PWAS identified 117 genes associated with ALS, with TWAS and PWAS providing novel mechanistic insights. Drug repurposing analyses identified five drugs significantly enriched for interactions with ALS associated genes, with directional analyses highlighting α-glucosidase inhibitors may exacerbate ALS pathology. Additionally, drug class enrichment analysis showed calcium channel blockers may reduce ALS risk. Across the two observed expression target samples, ALS polytranscriptomic scores significantly predicted ALS risk ( Functionally-informed analyses of ALS GWAS summary statistics identified novel mechanistic insights into ALS aetiology, highlighted several therapeutic research avenues, and enabled statistically significant prediction of ALS risk.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39578729',\n",
       "   'title': 'Plasma pQTL and brain eQTL integration identifies PNKP as a therapeutic target and reveals mechanistic insights into migraine pathophysiology.',\n",
       "   'abstract': \"Migraine is a prevalent neurological disorder affecting 14.1% of the global population. Despite advances in genetic research, further investigation is needed to identify therapeutic targets and better understand its mechanisms. In this study, we aimed to identify drug targets and explore the relationships between gene expression, protein levels, and migraine pathophysiology. We utilized cis-pQTL data from deCODE Genetics, combined with migraine GWAS data from the GERA\\u2009+\\u2009UKB cohort as the discovery cohort and the FinnGen R10 cohort as the replication cohort. SMR and MR analyses identified migraine-associated protein loci. Brain eQTL data from GTEx v8 and BrainMeta v2 were used to explore causal relationships between gene expression, protein levels, and migraine risk. Mediation analysis assessed the role of metabolites, and PheWAS evaluated potential side effects. Four loci were identified: PNKP, MRVI1, CALCB, and INPP5B. PNKP and MRVI1 showed a high level of evidence and opposing effects at the gene and protein levels. PNKP gene expression in certain brain regions was protective against migraine, while its plasma protein levels were positively associated with migraine risk. MRVI1 showed protective effects at the protein level but had the opposite effect at the gene expression level. Mediation analysis revealed that the glutamate to pyruvate ratio and 3-CMPFP mediated PNKP's effects on migraine. PheWAS indicated associations between PNKP and body composition traits, suggesting drug safety considerations. PNKP and MRVI1 exhibit dual mechanisms of action at the gene and protein levels, potentially involving distinct mechanistic pathways. Among them, PNKP emerges as a promising drug target for migraine treatment, supported by multi-layered validation.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39263038',\n",
       "   'title': 'Identifying genetically-supported drug repurposing targets for non-small cell lung cancer through mendelian randomization of the druggable genome.',\n",
       "   'abstract': 'Lung cancer is responsible for most cancer-related deaths, and non-small cell lung cancer (NSCLC) accounts for the majority of cases. Targeted therapy has made promising advancements in systemic treatment for NSCLC over the last two decades, but inadequate drug targets with clinically proven survival benefits limit its universal application in clinical practice compared to chemotherapy and immunotherapy. There is an urgent need to explore new drug targets to expand the beneficiary group. This study aims to identify druggable genes and to predict the efficacy and prognostic value of the corresponding targeted drugs in NSCLC. Two-sample mendelian randomization (MR) of druggable genes was performed to predict the efficacy of their corresponding targeted therapy for NSCLC. Subsequent sensitivity analyses were performed to assess potential confounders. Accessible RNA sequencing data were incorporated for subsequent verifications, and Kaplan-Meier survival curves of different gene expressions were used to explore the prognostic value of candidate druggable genes. MR screening encompassing 4,863 expression quantitative trait loci (eQTL) and 1,072 protein quantitative trait loci (pQTL, with 453 proteins overlapping) were performed. Seven candidate druggable genes were identified, including  Drugs targeting genetically supported genomes are considerably more likely to yield promising efficacy and succeed in clinical trials. We provide compelling genetic evidence to prioritize drug development for NSCLC.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38644354',\n",
       "   'title': 'Systematic druggable genome-wide Mendelian randomization identifies therapeutic targets for sarcopenia.',\n",
       "   'abstract': 'There are no effective pharmacological treatments for sarcopenia. We aim to identify potential therapeutic targets for sarcopenia by integrating various publicly available datasets. We integrated druggable genome data, cis-eQTL/cis-pQTL from human blood and skeletal muscle tissue, and GWAS summary data of sarcopenia-related traits to analyse the potential causal relationships between drug target genes and sarcopenia using the Mendelian Randomization (MR) method. Sensitivity analyses and Bayesian colocalization were employed to validate the causal relationships. We also assessed the side effects or additional indications of the identified drug targets using a phenome-wide MR (Phe-MR) approach and investigated actionable drugs for target genes using available databases. MR analysis identified 17 druggable genes with potential causation to sarcopenia in human blood or skeletal muscle tissue. Six of them (HP, HLA-DRA, MAP 3K3, MFGE8, COL15A1, and AURKA) were further confirmed by Bayesian colocalization (PPH4\\xa0>\\xa090%). The up-regulation of HP [higher ALM (beta: 0.012, 95% CI: 0.007-0.018, P\\xa0=\\xa01.2*10 Our research indicated MAP 3K3, MFGE8, COL15A1, HP, and HLA-DRA may serve as promising targets for sarcopenia, while the effectiveness of zinc supplementation and collagenase clostridium histolyticum for sarcopenia requires further validation.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37639066',\n",
       "   'title': 'Abnormal Brain Protein Abundance and Cross-tissue mRNA Expression in Amyotrophic Lateral Sclerosis.',\n",
       "   'abstract': 'Due to the limitations of the present risk genes in understanding the etiology of amyotrophic lateral sclerosis (ALS), it is necessary to find additional causative genes utilizing novel approaches. In this study, we conducted a two-stage proteome-wide association study (PWAS) using ALS genome-wide association study (GWAS) data (N\\u2009=\\u2009152,268) and two distinct human brain protein quantitative trait loci (pQTL) datasets (ROSMAP N\\u2009=\\u2009376 and Banner N\\u2009=\\u2009152) to identify ALS risk genes and prioritized candidate genes with Mendelian randomization (MR) and Bayesian colocalization analysis. Next, we verified the aberrant expression of risk genes in multiple tissues, including lower motor neurons, skeletal muscle, and whole blood. Six ALS risk genes (SCFD1, SARM1, TMEM175, BCS1L, WIPI2, and DHRS11) were found during the PWAS discovery phase, and SARM1 and BCS1L were confirmed during the validation phase. The following MR (p\\u2009=\\u20092.10\\u2009×\\u200910'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39033731',\n",
       "   'title': 'Circulating inflammatory proteins and abdominal aortic aneurysm: A two-sample Mendelian randomization and colocalization analyses.',\n",
       "   'abstract': 'Inflammatory proteins are implicated in the progression of abdominal aortic aneurysms (AAA); however, it remains debated whether they are causal or consequential. This study aimed to assess the influence of circulating inflammatory proteins on AAA via two-sample Mendelian randomization (MR) and colocalization analysis. Summary data on 91 circulating inflammatory protein levels were extracted from a comprehensive protein quantitative trait loci (pQTL) study involving 14,824 individuals. Genetic associations with AAA were derived from the FinnGen study (3,869 cases and 381,977 controls). MR analysis was conducted to assess the relationships between proteins and AAA risk. Colocalization analysis was employed to explore potential shared causal variants between identified proteins and AAA. Using a two-sample bidirectional MR study, our findings suggested that genetically predicted elevated levels of TGFB1 (OR\\xa0=\\xa01.21, P\\xa0=\\xa00.003), SIRT2 (OR\\xa0=\\xa01.196, P\\xa0=\\xa00.031) and TNFSF14 (OR\\xa0=\\xa01.129, P\\xa0=\\xa00.034) were linked to an increased risk of AAA. Conversely, genetically predicted higher levels of CD40 (OR\\xa0=\\xa00.912, P\\xa0=\\xa00.049), IL2RB (OR\\xa0=\\xa00.839, P\\xa0=\\xa00.028) and KITLG (OR\\xa0=\\xa00.827, P\\xa0=\\xa00.008) were associated with a decreased risk of AAA. Colocalization analyses supported the association of TGFB1 and SIRT2 levels with AAA risk. The proteome-wide MR and colocalization study identified TGFB1 and SIRT2 as being associated with the risk of AAA, warranting further investigation as potential therapeutic targets.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39354406',\n",
       "   'title': 'Exploration of protein and genetic targets causing atrioventricular block: mendelian-randomization analyses based on eQTL data and pQTL data.',\n",
       "   'abstract': 'Atrioventricular block (AVB) is a heterogeneous group of arrhythmias. AVB can lead to sudden arrest of the heart and subsequent syncope or sudden cardiac death. Few scholars have investigated the underlying molecular mechanisms of AVB. Finding molecular markers can facilitate understanding of AVB and exploration of therapeutic targets. Two-sample Mendelian randomization (MR) analysis was undertaken with inverse variance weighted (IVW) model and Wald ratio as the primary approach. Reverse MR analysis was undertaken to identify the associated protein targets and gene targets. Expression quantitative trait loci (eQTL) data from the eQTLGen database and protein quantitative trait loci (pQTL) data from three previous large-scale proteomic studies on plasma were retrieved as exposure data. Genome-wide association study (GWAS) summary data (586 cases and 379,215 controls) for AVB were retrieved from the UK Biobank database. Colocalization analyses were undertaken to identify the effect of filtered markers on outcome data. Databases (DrugBank, Therapeutic Target, PubChem) were used to identify drugs that interacted with targets. We discovered that 692 genes and 42 proteins showed a significant correlation with the AVB phenotype. Proteins (cadherin-5, sTie-1, Notch 1) and genes (DNAJC30, ABO) were putative molecules to AVB. Drug-interaction analyses revealed anticancer drugs such as tyrosine-kinase inhibitors and TIMD3 inhibitors could cause AVB. Other substances (e.g. toxins, neurological drugs) could also cause AVB. We identified the proteins (cadherin-5, sTie-1, Notch 1) and gene (DNAJC30, ABO) targets associated with AVB pathogenesis. Anticancer drugs (tyrosine-kinase inhibitors, TIMD3 inhibitors), toxins, or neurological drugs could also cause AVB.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40045304',\n",
       "   'title': 'Multi-omics study on autophagic dysfunction molecular network in the pathogenesis of rheumatoid arthritis.',\n",
       "   'abstract': 'Autophagy is associated with the development of rheumatoid arthritis (RA), but its genetic pathological mechanisms remain incompletely understood. In this study, we employed summary-data-based Mendelian randomization (SMR) and co-localization analysis to systematically investigate the relationship between autophagy-related genes and RA. We obtained summary data on blood methylation (mQTL), gene expression (eQTL), and protein abundance (pQTL) from respective quantitative trait locus (QTL) studies. Genetic association data for RA were primarily derived from the FinnGen database, with validation performed using the UK Biobank (UKB) and GWAS Catalog databases. SMR analysis was conducted to evaluate the association between molecular characteristics of autophagy-related genes and RA. Subsequently, co-localization analysis was performed to determine whether the identified signals share the same causal genetic variants. After integrating mQTL-eQTL multi-omics data, we identified two key autophagy genes, BCL2L1 and RAF1, which may have a causal relationship with RA. Significant associations were found for BCL2L1 (cg12873919, cg13989999) and RAF1 (cg26432171) in the SMR analysis of autophagy-related mQTL, eQTL, and GWAS data (p SMR\\u2009<\\u20090.05). In the integrated mQTL-eQTL SMR analysis, cg12873919 (p SMR\\u2009=\\u20091.40E-07, OR\\u2009=\\u20090.82, 95% CI [0.76-0.88]), cg13989999 (p SMR\\u2009=\\u20091.43E-06, OR\\u2009=\\u20090.78, 95% CI [0.71-0.87]), and cg26432171 (p SMR\\u2009=\\u20099.18E-09, OR\\u2009=\\u20091.83, 95% CI [1.49-2.25]) were all significantly validated. Methylation of cg12873919 and cg13989999 in BCL2L1 was associated with increased BCL2L1 expression, consistent with their negative impact on RA risk. Conversely, the cg26432171 site in RAF1 showed a positive correlation between gene methylation and expression. In the eQTL-GWAS SMR analysis, MAPK3 expression (p SMR\\u2009=\\u20097.24E-05, OR\\u2009=\\u20090.91, 95% CI [0.87-0.95]) was negatively correlated with RA risk, a finding supported by co-localization analysis (PPH4\\u2009>\\u20090.5), suggesting that this gene may inhibit RA pathogenesis by regulating the autophagy process. Furthermore, protein level analysis also supported the protective role of MAPK3 (p SMR\\u2009=\\u20097.53E-05, OR\\u2009=\\u20090.89, 95% CI [0.84-0.94]). We identified that autophagy-related genes BCL2L1 and RAF1 may be associated with RA risk, providing strong evidence from multi-omics data. This study identifies autophagy genes related to RA, potentially offering new insights into the pathogenesis of RA.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39108314',\n",
       "   'title': \"Polygenic scores and Mendelian randomization identify plasma proteins causally implicated in Alzheimer's disease.\",\n",
       "   'abstract': \"An increasing body of evidence suggests that neuroinflammation is one of the key drivers of late-onset Alzheimer's disease (LOAD) pathology. Due to the increased permeability of the blood-brain barrier (BBB) in older adults, peripheral plasma proteins can infiltrate the central nervous system (CNS) and drive neuroinflammation through interactions with neurons and glial cells. Because these inflammatory factors are heritable, a greater understanding of their genetic relationship with LOAD could identify new biomarkers that contribute to LOAD pathology or offer protection against it. We used a genome-wide association study (GWAS) of 90 different plasma proteins ( We identified four plasma protein level PGSs that were significantly associated (FDR-adjusted  Our results show that plasma HGF has a negative causal relationship with LOAD liability that is driven by pleiotropic SNPs possibly involved in other pathways. These findings suggest a low transferability between PGS and MR approaches, and future research should explore ways in which LOAD and the plasma proteome may interact.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39824912',\n",
       "   'title': 'Integrative genetics and multiomics analysis reveal mechanisms and therapeutic targets in vitiligo highlighting JAK STAT pathway regulation of CTSS.',\n",
       "   'abstract': \"Vitiligo is a complex autoimmune disease characterized by the loss of melanocytes, leading to skin depigmentation. Despite advances in understanding its genetic and molecular basis, the precise mechanisms driving vitiligo remain elusive. Integrating multiple layers of omics data can provide a comprehensive view of disease pathogenesis and identify potential therapeutic targets. The study aims to delineate the genetic and molecular mechanisms of vitiligo pathogenesis using an integrative multiomics strategy. We focus on exploring the regulatory influence of the JAK/STAT pathway on Cathepsin S, a potential therapeutic target in vitiligo. Our GWAS-meta analysis pinpointed five druggable genes: ERBB3, RHOH, CDK10, MC1R, and NDUFAF3, and underwent drug target exploration and molecular docking. SMR analysis linked CTSS, CTSH, STX8, KIR2DL3, and GRHPR to vitiligo through pQTL and eQTL associations. Microarray and single-cell RNA-seq data showed differential expression of CTSS and STAT1/3 in vitiligo patients' blood and skin lesions. Our study offers novel perspectives on vitiligo's genetic and molecular basis, highlighting the JAK/STAT pathway's role in regulating CTSS for antigen processing in melanocytes. Further research is needed to confirm these results and assess the therapeutic potential of CTSS and related genes.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38240717',\n",
       "   'title': \"Joint analysis of proteome, transcriptome, and multi-trait analysis to identify novel Parkinson's disease risk genes.\",\n",
       "   'abstract': \"Genome-wide association studies (GWAS) have identified multiple risk variants for Parkinson's disease (PD). Nevertheless, how the risk variants confer the risk of PD remains largely unknown. We conducted a proteome-wide association study (PWAS) and summary-data-based mendelian randomization (SMR) analysis by integrating PD GWAS with proteome and protein quantitative trait loci (pQTL) data from human brain, plasma and CSF. We also performed a large transcriptome-wide association study (TWAS) and Fine-mapping of causal gene sets (FOCUS), leveraging joint-tissue imputation (JTI) prediction models of 22 tissues to identify and prioritize putatively causal genes. We further conducted PWAS, SMR, TWAS, and FOCUS using a multi-trait analysis of GWAS (MTAG) to identify additional PD risk genes to boost statistical power. In this large-scale study, we identified 16 genes whose genetically regulated protein abundance levels were associated with Parkinson's disease risk. We undertook a large-scale analysis of PD and correlated traits, through TWAS and FOCUS studies, and discovered 26 casual genes related to PD that had not been reported in previous TWAS. 5 genes (\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39210598',\n",
       "   'title': 'Extensive co-regulation of neighboring genes complicates the use of eQTLs in target gene prioritization.',\n",
       "   'abstract': 'Identifying causal genes underlying genome-wide association studies (GWASs) is a fundamental problem in human genetics. Although colocalization with gene expression quantitative trait loci (eQTLs) is often used to prioritize GWAS target genes, systematic benchmarking has been limited due to unavailability of large ground truth datasets. Here, we re-analyzed plasma protein QTL data from 3,301 individuals of the INTERVAL cohort together with 131 eQTL Catalog datasets. Focusing on variants located within or close to the affected protein identified 793 proteins with at least one cis-pQTL where we could assume that the most likely causal gene was the gene coding for the protein. We then benchmarked the ability of cis-eQTLs to recover these causal genes by comparing three Bayesian colocalization methods (coloc.susie, coloc.abf, and CLPP) and five Mendelian randomization (MR) approaches (three varieties of inverse-variance weighted MR, MR-RAPS, and MRLocus). We found that assigning fine-mapped pQTLs to their closest protein coding genes outperformed all colocalization methods regarding both precision (71.9%) and recall (76.9%). Furthermore, the colocalization method with the highest recall (coloc.susie - 46.3%) also had the lowest precision (45.1%). Combining evidence from multiple conditionally distinct colocalizing QTLs with MR increased precision to 81%, but this was accompanied by a large reduction in recall to 7.1%. Furthermore, the choice of the MR method greatly affected performance, with the standard inverse-variance-weighted MR often producing many false positives. Our results highlight that linking GWAS variants to target genes remains challenging with eQTL evidence alone, and prioritizing novel targets requires triangulation of evidence from multiple sources.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/34974068',\n",
       "   'title': 'A genomic approach identifies sRAGE as a putatively causal protein for asthma.',\n",
       "   'abstract': 'Asthma is a complex respiratory condition caused by environmental and genetic factors. Although lower concentrations of the anti-inflammatory protein soluble receptor for advanced glycation end products (sRAGE) have been associated with asthma in humans and mouse models, it is uncertain whether sRAGE plays a causal role in asthma. We designed a 2-stage study of sRAGE in relation to asthma with association analysis in FHS participants as well as causal inference testing using Mendelian randomization (MR). We measured plasma levels of sRAGE and performed cross-sectional analysis to examine the association between plasma sRAGE concentration and asthma status in 6546 FHS participants. We then used sRAGE protein advanced glycation end products (pQTLs) derived from a genome-wide association study of plasma sRAGE levels in ∼7000 FHS participants with UK Biobank asthma genome-wide association study in MR to consider sRAGE as a putatively causal protein for asthma. We also performed replication MR using an externally derived sRAGE pQTL from the INTERVAL study. Last, we conducted colocalization using cis-pQTL variants at the advanced glycosylation end-product specific receptor (AGER) locus with variants from the UK Biobank asthma genome-wide association study. Association analysis revealed that each 1 SD increment in sRAGE concentration was associated with a 14% lower odds of asthma in FHS participants (95% CI 0.76-0.96). MR identified sRAGE as putatively causal for and protective against asthma on the basis of self-reported (odds ratio [per 1 SE increment in inverse-rank-normalized sRAGE]\\xa0= 0.97, 95% CI 0.95-0.99; P\\xa0= .005) and doctor-diagnosed asthma (odds ratio\\xa0= 0.97, 95% CI 0.95-0.99; P\\xa0= .011). Through this genomic approach, we identified sRAGE as a putatively causal, biologically important, and protective protein in relation to asthma. Functional studies in\\xa0cell/animal models are needed to confirm our findings.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32617297',\n",
       "   'title': 'Assessing the genetic relationships between osteoarthritis and human plasma proteins: a large scale genetic correlation scan.',\n",
       "   'abstract': 'Osteoarthritis (OA) is a multifactorial complex disease. The impact of plasma proteins on OA remains elusive now. The UK Biobank genome-wide association study data of OA was used here. Genome-wide SNP genotyping was performed using the Affymetrix UK BiLEVE Axiom or UK Biobank Axiom array. Equally, the GWAS summary data of 3,622 plasma proteins was derived from a recently published study. Consequently, linkage disequilibrium score regression (LD score regression) analysis was performed to evaluate the genetic correlation between each plasma protein and different sites of OA. Several suggestive plasma proteins were identified for OA. For hand OA, evidence of genetic correlation was observed for inter-alpha-trypsin inhibitor heavy chain H1 (coefficient =-0.3854, P value =0.0198), multiple inositol polyphosphate phosphatase 1 (coefficient =-1.1721, P value =0.0303). For hip OA, 7 suggestive genetic correlation signals were observed, such as Transmembrane glycoprotein NMB (coefficient =0.6944, P value =0.0098), Endothelial cell-specific molecule 1 (coefficient =0.6337, P value =0.03). For Knee OA, 12 suggestive genetic correlation signals were identified, including Elafin (coefficient =-0.5562, P value =0.0092), Interleukin-16 (coefficient =0.3949, P value =0.0435). We investigated the genetic correlations between plasma proteins and different sites of OA in a systematic way. Our results provide novel evidence that OA is a heterogeneous disease.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38260300',\n",
       "   'title': 'Hippocampus Glutathione S Reductase Potentially Confers Genetic Resilience to Cognitive Decline in the AD-BXD Mouse Population.',\n",
       "   'abstract': \"Alzheimer's disease (AD) is a prevalent and costly age-related dementia. Heritable factors account for 58-79% of variation in late-onset AD, but substantial variation remains in age-of- onset, disease severity, and whether those with high-risk genotypes acquire AD. To emulate the diversity of human populations, we utilized the AD-BXD mouse panel. This genetically diverse resource combines AD genotypes with multiple BXD strains to discover new genetic drivers of AD resilience. Comparing AD-BXD carriers to noncarrier littermates, we computed a novel quantitative metric for resilience to cognitive decline in the AD-BXDs. Our quantitative AD resilience trait was heritable and genetic mapping identified a locus on chr8 associated with resilience to AD mutations that resulted in amyloid brain pathology. Using a hippocampus proteomics dataset, we nominated the mitochondrial glutathione S reductase protein (GR or GSHR) as a resilience factor, finding that the DBA/2J genotype was associated with substantially higher GR abundance. By mapping protein QTLs (pQTLs), we identified synaptic organization and mitochondrial proteins coregulated in trans with a cis-pQTL for GR. We found four coexpression modules correlated with the quantitative resilience score in aged 5XFAD mice using paracliques, which were related to cell structure, protein folding, and postsynaptic densities. Finally, we found significant positive associations between human \"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39261750',\n",
       "   'title': 'Proteome-wide Mendelian randomization identified potential drug targets for migraine.',\n",
       "   'abstract': 'Migraine is a highly prevalent and complex neurovascular disease. However, the currently available therapeutic drugs often fall to adequately meet clinical needs due to limited effectiveness and numerous undesirable side effects. This study aims to identify putative novel targets for migraine treatment through proteome-wide Mendelian randomization (MR). We utilized MR to estimate the causal effects of plasma proteins on migraine and its two subtypes, migraine with aura (MA) and without aura (MO). This analysis integrated plasma protein quantitative trait loci (pQTL) data with genome-wide association studies (GWAS) findings for these migraine phenotypes. Moreover, we conducted a phenome-wide MR assessment, enrichment analysis, protein-protein interaction networks construction, and mediation MR analysis to further validate the pharmaceutical potential of the identified protein targets. We identified 35 protein targets for migraine and its subtypes (p\\u2009<\\u20098.04\\u2009×\\u200910 A set of potential protein targets for migraine and its subtypes were identified. These proteins showed rare side effects and were responsible for biological mechanisms involved in migraine pathogenesis, indicating priority for the development of migraine treatments.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35128275',\n",
       "   'title': 'Detailed Analytical Characterization of a Bispecific IgG1 CrossMab Antibody of the Knob-into-Hole Format Applying Various Stress Conditions Revealed Pronounced Stability.',\n",
       "   'abstract': 'In recent years, a variety of new antibody formats have been developed. One of these formats allows the binding of one type of antibody to two different epitopes. This can for example be achieved by introduction of the \"knob-into-hole\" format and a combined CrossMab approach. Due to their complexity, these bispecific antibodies are expected to result in an enhanced variety of different degradation products. Reports on the stability of these molecules are still largely lacking. To address this, a panel of stress conditions, including elevated temperature, pH, oxidizing agents, and forced glycation via glucose incubation, to identify and functionally evaluate critical quality attributes in the complementary-determining and conserved regions of a bispecific antibody was applied in this study. The exertion of various stress conditions combined with an assessment by size exclusion chromatography, ion exchange chromatography, LC-MS/MS peptide mapping, and functional evaluation by cell-based assays was adequate to identify chemical modification sites and assess the stability and integrity, as well as the functionality of a bispecific antibody. Stress conditions induced size variants and post-translational modifications, such as isomerization, deamidation, and oxidation, albeit to a modest extent. Of note, all the observed stress conditions largely maintained functionality. In summary, this study revealed the pronounced stability of IgG1 \"knob-into-hole\" bispecific CrossMab antibodies compared to already marketed antibody products.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/24165562',\n",
       "   'title': 'Body composition and gene expression QTL mapping in mice reveals imprinting and interaction effects.',\n",
       "   'abstract': 'Shifts in body composition, such as accumulation of body fat, can be a symptom of many chronic human diseases; hence, efforts have been made to investigate the genetic mechanisms that underlie body composition. For example, a few quantitative trait loci (QTL) have been discovered using genome-wide association studies, which will eventually lead to the discovery of causal mutations that are associated with tissue traits. Although some body composition QTL have been identified in mice, limited research has been focused on the imprinting and interaction effects that are involved in these traits. Previously, we found that Myostatin genotype, reciprocal cross, and sex interacted with numerous chromosomal regions to affect growth traits. Here, we report on the identification of muscle, adipose, and morphometric phenotypic QTL (pQTL), translation and transcription QTL (tQTL) and expression QTL (eQTL) by applying a QTL model with additive, dominance, imprinting, and interaction effects. Using an F2 population of 1000 mice derived from the Myostatin-null C57BL/6 and M16i mouse lines, six imprinted pQTL were discovered on chromosomes 6, 9, 10, 11, and 18. We also identified two IGF1 and two Atp2a2 eQTL, which could be important trans-regulatory elements. pQTL, tQTL and eQTL that interacted with Myostatin, reciprocal cross, and sex were detected as well. Combining with the additive and dominance effect, these variants accounted for a large amount of phenotypic variation in this study. Our study indicates that both imprinting and interaction effects are important components of the genetic model of body composition traits. Furthermore, the integration of eQTL and traditional QTL mapping may help to explain more phenotypic variation than either alone, thereby uncovering more molecular details of how tissue traits are regulated.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39989961',\n",
       "   'title': 'Genome-wide mapping of complement system proteins for islet autoimmunity in the DAISY and TEDDY children.',\n",
       "   'abstract': 'Type 1 diabetes (T1D) is characterized by the autoimmune destruction of the insulin-producing beta cells, and there is no cure yet for the disease. While islet autoantibodies are well-recognized biomarkers that mark the onset of islet autoimmunity (IA) and are predictors of T1D, few additional biomarkers are available to monitor disease progression. Recent studies have reported the involvement of complement system proteins in the initiation and progression of IA in the study of T1D. However, the genetic factors of complement system proteins at the time of triggering of IA is unknown. Through complement system protein quantitative trait locus (pQTL) mapping analysis of 170 participants from the Diabetes Autoimmunity Study in the Young (DAISY), we identified 240 statistically significant pQTLs (false discovery rate, FDR < 0.1) from pooled and IA case-stratified analyses. Replication analysis conducted on 385 IA cases from The Environment Determinants of Diabetes in the Young (TEDDY) study confirmed 68 significant (FDR < 0.05) pQTLs in total for C8A, C8B, CFB, C4A, and MBL2. Furthermore, all replicated pQTLs of CFB and C4A were previously reported to be associated with T1D risk. We identified and replicated 68 pQTLs for five complement system proteins (C8A, C8B, CFB, C4A, and MBL2) in the young population. Among them, all replicated pQTLs of CFB and C4A are also associated with T1D risk. Our study provides evidence of complement system proteins as potential protein biomarkers underlying the development and progression of T1D.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37548539',\n",
       "   'title': 'The Genetic, Pharmacogenomic, and Immune Landscapes Associated with Protein Expression across Human Cancers.',\n",
       "   'abstract': 'Proteomics is a powerful approach that can rapidly enhance our understanding of cancer development. Detailed characterization of the genetic, pharmacogenomic, and immune landscape in relation to protein expression in patients with cancer could provide new insights into the functional roles of proteins in cancer. By taking advantage of the genotype data from The Cancer Genome Atlas and protein expression data from The Cancer Proteome Atlas, we characterized the effects of genetic variants on protein expression across 31 cancer types and identified approximately 100,000 protein quantitative trait loci (pQTL). Among these, over 8000 pQTLs were associated with patient overall survival. Furthermore, characterization of the impact of protein expression on more than 350 imputed anticancer drug responses in patients revealed nearly 230,000 significant associations. In addition, approximately 21,000 significant associations were identified between protein expression and immune cell abundance. Finally, a user-friendly data portal, GPIP (https://hanlaboratory.com/GPIP), was developed featuring multiple modules that enable researchers to explore, visualize, and browse multidimensional data. This detailed analysis reveals the associations between the proteomic landscape and genetic variation, patient outcome, the immune microenvironment, and drug response across cancer types, providing a resource that may offer valuable clinical insights and encourage further functional investigations of proteins in cancer. Comprehensive characterization of the relationship between protein expression and the genetic, pharmacogenomic, and immune landscape of tumors across cancer types provides a foundation for investigating the role of protein expression in cancer development and treatment.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39506937',\n",
       "   'title': 'Integrative genomic analysis of RNA-modification-single nucleotide polymorphisms associated with kidney function.',\n",
       "   'abstract': 'Increasing evidence suggests that RNA modification plays a significant role in the kidney and may be an ideal target for the treatment of kidney diseases. However, the specific mechanisms underlying RNA modifications in the pathogenesis of kidney disease remain unclear. Genome-wide association studies (GWAS) have identified numerous genetic loci involved in kidney function and RNA modifications. The identification and exploration of RNA modification-related single-nucleotide polymorphisms (RNAm-SNPs) associated with kidney function can help us to comprehensively understand the underlying mechanism of kidney disease and identify potential therapeutic targets. First, we examined the association of RNAm-SNPs with eGFR. Second, we performed expression quantitative trait locus (eQTL) and protein quantitative trait locus (pQTL) analyses to explore the functions of the identified RNAm-SNPs. Finally, we evaluated the causality between RNAm-SNP-associated gene expression and circulating proteins and kidney function using a Mendelian randomization (MR) analysis. A total of 252 RNA m-SNPs related to m This study reveals an association between RNAm-SNPs and kidney function. These SNPs regulate gene expression and protein levels through RNA modifications, eventually leading to kidney dysfunction. Our study provides novel insights that connect the genetic risk of kidney disease to RNA modification and suggests potential therapeutic targets for the prevention and treatment of kidney disease.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37745050',\n",
       "   'title': 'Exploration of potential novel drug targets and biomarkers for small cell lung cancer by plasma proteome screening.',\n",
       "   'abstract': ''},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37481073',\n",
       "   'title': 'Circulating microRNAs predict recurrence and death following venous thromboembolism.',\n",
       "   'abstract': 'Recurrent events frequently occur after venous thromboembolism (VTE) and remain difficult to predict based on established genetic, clinical, and proteomic contributors. The role of circulating microRNAs (miRNAs) has yet to be explored in detail. To identify circulating miRNAs predictive of recurrent VTE or death, and to interpret their mechanistic involvement. Data from 181 participants of a cohort study of acute VTE and 302 individuals with a history of VTE from a population-based cohort were investigated. Next-generation sequencing was performed on EDTA plasma samples to detect circulating miRNAs. The endpoint of interest was recurrent VTE or death. Penalized regression was applied to identify an outcome-relevant miRNA signature, and results were validated in the population-based cohort. The involvement of miRNAs in coregulatory networks was assessed using principal component analysis, and the associated clinical and molecular phenotypes were investigated. Mechanistic insights were obtained from target gene and pathway enrichment analyses. A total of 1950 miRNAs were detected across cohorts after postprocessing. In the discovery cohort, 50 miRNAs were associated with recurrent VTE or death (cross-validated C-index, 0.65). A weighted miRNA score predicted outcome over an 8-year follow-up period (HR Circulating miRNAs predicted the risk of recurrence or death after VTE over several years, both in the acute and chronic phases.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39565278',\n",
       "   'title': 'Brain multi-omic Mendelian randomisation to identify novel drug targets for gliomagenesis.',\n",
       "   'abstract': 'Genetic variants associated with molecular traits that are also associated with liability to glioma can provide causal evidence for the identification and prioritisation of drug targets. We performed comprehensive two-sample Mendelian randomisation (Wald ratio and/or IVW) and colocalisation analyses of molecular traits on glioma. Instrumentable traits (QTLs P\\u2009<\\u20095\\xa0×\\xa010-8) were identified amongst 11\\u2009985 gene expression measures, 13\\u2009285 splicing isoforms and 10\\u2009198 protein abundance measures, derived from 15 brain regions. Glioma summary-level data was extracted from a genome-wide association meta-analysis of 12\\u2009496 cases and 18\\u2009190 controls. We found evidence for causal effect of 22 molecular traits (across 18 genes/proteins) on glioma risk. Thirteen molecular traits have been previously linked with glioma risk and five were novel; HBEGF (5q31.3) expression and all glioma [OR 1.36 (95%CI 1.19-1.55); P\\u2009=\\u20094.41\\xa0×\\xa010-6]; a CEP192 (18p11.21) splice isoform and glioblastoma [OR 4.40 (95%CI 2.28-8.48); P\\u2009=\\u20099.78\\xa0×\\xa010-4]; a FAIM (3q22.3) splice isoform and all glioma [OR 2.72-3.43; P\\u2009=\\u20091.03\\xa0×\\xa010-5 to 1.09\\xa0×\\xa010-5]; a SLC8A1 (2p22.1) splice isoform and all glioma [OR 0.37 (95%CI 0.24-0.56; P\\u2009=\\u20095.72\\xa0×\\xa010-6]; D2HGDH (2q37.3) protein and all glioma [OR 0.86 (95%CI 0.80-0.92); P\\u2009=\\u20095.94\\xa0×\\xa010-6)]. We provide robust causal evidence for prioritising genes and their protein products in glioma research. Our results highlight the importance of alternative splicing as a mechanism in gliomagenesis and as an avenue for exploration of drug targets.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38685026',\n",
       "   'title': 'Large-scale genome-wide association studies reveal the genetic causal etiology between air pollutants and autoimmune diseases.',\n",
       "   'abstract': 'Epidemiological evidence links a close correlation between long-term exposure to air pollutants and autoimmune diseases, while the causality remained unknown. Two-sample Mendelian randomization (TSMR) was used to investigate the role of PM10, PM2.5, NO In TSMR, after correction of multiple testing, hypothyroidism was causally associated with higher exposure to NO This research underscores the necessity of rigorous air pollutant surveillance within public health studies to curb the prevalence of autoimmune diseases.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39979251',\n",
       "   'title': 'PSMB4: a potential biomarker and therapeutic target for depression, perspective from integration analysis of depression GWAS data and human plasma proteome.',\n",
       "   'abstract': 'Depression is a common and severe mental disorder that affects more than 300 million people worldwide. While it is known to have a moderate genetic component, identifying specific genes that contribute to the disorder has been challenging. Previous Genome-wide association studies (GWASs) have identified over 100 genomic loci that are significantly associated with depression. But finding useful therapeutic targets and diagnostic biomarkers from this information has proven difficult. To address this challenge, I conducted a plasma protein proteome-wide association study (PWAS) for depression, using human plasma protein QTL (pQTL) and depression GWAS data. I identified four proteins that were significantly associated with depression: BTN3A3 (P value\\u2009=\\u20096.41\\u2009×\\u200910'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36349687',\n",
       "   'title': 'Identifying causal serum protein-cardiometabolic trait relationships using whole genome sequencing.',\n",
       "   'abstract': 'Cardiometabolic diseases, such as type 2 diabetes and cardiovascular disease, have a high public health burden. Understanding the genetically determined regulation of proteins that are dysregulated in disease can help to dissect the complex biology underpinning them. Here, we perform a protein quantitative trait locus (pQTL) analysis of 248 serum proteins relevant to cardiometabolic processes in 2893 individuals. Meta-analyzing whole-genome sequencing (WGS) data from two Greek cohorts, MANOLIS (n\\u2009=\\u20091356; 22.5× WGS) and Pomak (n\\u2009=\\u20091537; 18.4× WGS), we detect 301 independently associated pQTL variants for 170 proteins, including 12 rare variants (minor allele frequency\\u2009<\\u20091%). We additionally find 15 pQTL variants that are rare in non-Finnish European populations but have drifted up in the frequency in the discovery cohorts here. We identify proteins causally associated with cardiometabolic traits, including Mep1b for high-density lipoprotein (HDL) levels, and describe a knock-out (KO) Mep1b mouse model. Our findings furnish insights into the genetic architecture of the serum proteome, identify new protein-disease relationships and demonstrate the importance of isolated populations in pQTL analysis.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/33411684',\n",
       "   'title': 'Identification of novel SNPs associated with coronary artery disease and birth weight using a pleiotropic cFDR method.',\n",
       "   'abstract': 'Clinical and epidemiological findings indicate an association between coronary artery disease (CAD) and low birth weight (BW). However, the mechanisms underlying this relationship are largely unknown. Here, we aimed to identify novel single-nucleotide polymorphisms (SNPs) associated with CAD, BW, and their shared pleiotropic loci, and to detect the potential causal relationship between CAD and BW. We first applied a genetic pleiotropic conditional false discovery rate (cFDR) method to two independent genome-wide association studies (GWAS) summary statistics of CAD and BW to estimate the pleiotropic enrichment between them. Then, bi-directional Mendelian randomization (MR) analyses were performed to clarify the causal association between these two traits. By incorporating related traits into a conditional analysis framework, we observed the significant pleiotropic enrichment between CAD and BW. By applying the cFDR level of 0.05, 109 variants were detected for CAD, 203 for BW, and 26 pleiotropic variants for both traits. We identified 11 CAD- and/or BW-associated SNPs that showed more than three of the metabolic quantitative trait loci (metaQTL), protein QTL (pQTL), methylation QTL (meQTL), or expression QTL (eQTL) effects. The pleiotropic SNP rs10774625, located at  We identified several pleiotropic loci between CAD and BW by leveraging GWAS results of related phenotypes and identified a potential causal relationship from BW to CAD. Our findings provide novel insights into the shared biological mechanisms and overlapping genetic heritability between CAD and BW.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37212950',\n",
       "   'title': 'Novel Drug Targets for Atrial Fibrillation Identified Through Mendelian Randomization Analysis of the Blood Proteome.',\n",
       "   'abstract': 'Novel, effective, and safe preventive therapy targets for AF are still needed. Circulating proteins with causal genetic evidence are promising candidates. We aimed to systematically screen circulating proteins for AF drug targets and determine their safety and efficacy using genetic methods. The protein quantitative trait loci (pQTL) of up to 1949 circulating proteins were retrieved from nine large genome-proteome-wide association studies. Two-sample Mendelian Randomization (MR) and colocalization analyses were used to estimate the causal effects of proteins on the risk of AF. Furthermore, phenome-wide MR was conducted to depict side effects and the drug-target databases were searched for drug validation and repurposing. Systematic MR screen identified 30 proteins as promising AF drug targets. Genetically predicted 12 proteins increased AF risk (TES, CFL2, MTHFD1, RAB1A, DUSP13, SRL, ANXA4, NEO1, FKBP7, SPON1, LPA, MANBA); 18 proteins decreased AF risk (PMVK, UBE2F, SYT11, CHMP3, PFKM, FBP1, TNFSF12, CTSZ, QSOX2, ALAD, EFEMP1, FLRT2, LRIG1, OLA1, SH3BGRL3, IL6R, B3GNT8, FCGR2A). DUSP13 and TNFSF12 possess strong colocalization evidence. For the proteins that were identified, extended phe-MR analysis was conducted to assess their side-effect profiles, while drug-target databases provided information on their approved or investigated indications. We identified 30 circulating proteins as potential preventive targets for AF.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/21750736',\n",
       "   'title': 'Characterization of serum proteins associated with IL28B genotype among patients with chronic hepatitis C.',\n",
       "   'abstract': 'Polymorphisms near the IL28B gene (e.g. rs12979860) encoding interferon λ3 have recently been associated with both spontaneous clearance and treatment response to pegIFN/RBV in chronic hepatitis C (CHC) patients. The molecular consequences of this genetic variation are unknown. To gain further insight into IL28B function we assessed the association of rs12979860 with expression of protein quantitative traits (pQTL analysis) generated using open-platform proteomics in serum from patients. 41 patients with genotype 1 chronic hepatitis C infection from the Duke Liver Clinic were genotyped for rs12979860. Proteomic profiles were generated by LC-MS/MS analysis following immunodepletion of serum with MARS14 columns and trypsin-digestion. Next, a latent factor model was used to classify peptides into metaproteins based on co-expression and using only those peptides with protein identifications. Metaproteins were then analyzed for association with IL28B genotype using one-way analysis of variance. There were a total of 4,186 peptides in the data set with positive identifications. These were matched with 253 proteins of which 110 had two or more associated, identified peptides. The IL28B treatment response genotype (rs12979860_CC) was significantly associated with lower serum levels of corticosteroid binding globulin (CBG; p\\u200a=\\u200a9.2×10(-6)), a major transport protein for glucocorticoids and progestins. Moreover, the CBG metaprotein was associated with treatment response (p\\u200a=\\u200a0.0148), but this association was attenuated when both IL28B genotype and CBG were included in the model, suggesting that the CBG association may be independent of treatment response. In this cohort of chronic hepatitis C patients, IL28B polymorphism was associated with serum levels of corticosteroid binding globulin, a major transporter of cortisol, however, CBG does not appear to mediate the association of IL28B with treatment response. Further investigation of this pathway is warranted to determine if it plays a role in other comorbidities of HCV-infection.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40059241',\n",
       "   'title': 'Identification of biomarkers and potential drug targets for esophageal cancer: a Mendelian randomization study.',\n",
       "   'abstract': 'Esophageal cancer (EC) is a common and deadly malignancy of the digestive system. Currently, effective treatments for EC are limited and patient prognosis remains poor. In this study, we utilized Mendelian Randomization (MR) to identify potential drug targets for EC by analyzing proteins linked to the disease risk. A total of 734 plasma proteins and 4,479 druggable genes were obtained from recent studies, and two-sample MR analyses were conducted to investigate causal relationships between these proteins and EC. The cis-pQTL data of the proteins was analyzed after filtering. The inverse variance weighted (IVW) method was the primary analytical approach in MR analysis. Steiger filtering, heterogeneity and pleiotropy tests, Summary-data-based Mendelian Randomization (SMR) analysis, and Bayesian co-localization analysis were implemented to consolidate the results further. Moreover, drugs corresponding to the identified proteins were found in the DrugBank database. Five proteins HPSE, ST3GAL1, CEL, KLK13, and GNRH2 were identified as highly associated with EC. HPSE and GNRH2 showed protective effects with odds ratios (OR) of 0.80 (95% confidence interval [CI], 0.70-0.92) and 0.73 (95% CI 0.54-0.98), respectively. In contrast, increased expression of ST3GAL1(OR, 1.37; 95% CI 1.04-1.82), CEL (OR, 1.27; 95% CI 1.08-1.49), and KLK13 (OR, 1.22; 95% CI 1.04-1.42) were all associated with a higher risk of EC. In addition, the HPSE protein showed moderate colocalization with EC [coloc.abf-posterior probability of hypothesis 4 (PPH4)\\u2009=\\u20090.637]. Furthermore, the sensitivity analyses indicated no heterogeneity or pleiotropy. Therefore, these findings present promising drug targets for EC and deserve further clinical investigation.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39867362',\n",
       "   'title': 'Regulation of protein abundance in normal human tissues.',\n",
       "   'abstract': 'We report a systematic quantification of 10,841 unique proteins from over 700 GTEx samples, representing five human tissues. Sex, age and genetic factors are associated with variation in protein abundance. In total, 1981 cis-protein quantitative trait loci (cis-pQTL) are identified, of which a majority of protein targets have not been assayed in the recent plasma-based proteogenomic studies. Integrating transcriptomic information from matching tissues delineates concordant as well as discordant expression patterns at RNA and protein levels. Juxtaposition of data from different tissues indicates both shared and tissue-specific genetic architecture that underlie protein abundance. Complementing genomic annotation, RNA-based eQTL studies, as well as the recent establishment of plasma-based proteogenomic characterization, tissue-pQTLs shed light on biology underlying genotype-phenotype association of complex traits and diseases.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39138392',\n",
       "   'title': 'Integration of QTL and comprehensive analysis in the circulating inflammatory cytokines for pan-cancer.',\n",
       "   'abstract': \"Chemokines and cytokines are components of the tumor microenvironment and also influence tumorigenesis and its composition. However, whether they genetically proxy tumorigenesis is unclear. For causal inferences, eQTL and pQTL were used to determine the role of chemokines and cytokines in pan-cancer. The impact on the tumor immune microenvironment was also explored. This study leveraged summary statistics from respective genome-wide association studies (GWAS) of 109 cytokines and chemokines in 18 types of solid tumors. Single nucleotide polymorphisms (SNPs) robustly associated with the cytokines and chemokines, located in or close to their coding gene (cis), were used as instrumental variables. A two-sample MR design was employed, followed by comprehensive sensitivity analyses to validate the robustness of results. The impact on immune infiltration was investigated using the TIMER and TISIDB websites. Survival analysis was conducted using the K-M plotter and TIMER 2.0 websites. The TISCH and GEO databases were used to carry out scRNA cell analysis.Analyzing relevant proteins using the STRING database and conducting enrichment pathways for GO analysis of the identified proteins. The results of the inverse-variance weighted (IVW) method using cis-protein QTL (cis-pQTL) instruments showed the causal effects of TNF in reducing the risk of squamous cell lung cancer (LUSC) and HGF in reducing the risk of head and neck cancer (HNSC).The results were consistent with the eQTL. HGF was associated with better overall survival (OS) in HNSC, regardless of the types of cells enriched. However, high expression of the ligand MET for HGF leads to a decrease in overall survival in LUSC. TNF was related to poor OS in LUSC with no significant impact. However, in CD8\\u2009+\\u2009T cell-enriched, eosinophil-enriched, macrophage-enriched, and NK cell-deficient types of LUSC, high expression of TNF leads to a poor prognosis, and there is statistical significance. The results showed a significant positive correlation between TNF and most immune cell infiltration, immunomodulator and chemokine in LUSC. HGF is positively correlated with the majority of immune cells except CD56\\u2009+\\u2009cells, as well as some immune regulatory factors and chemotactic factors. According to single-cell sequencing results, HGF is mainly secreted by fibroblasts and myofibroblasts in HNSC, while in LUSC, it is primarily secreted by macrophages and CD8\\u2009+\\u2009T cells secrete TNF. The GO/KEGG analysis suggests that proteins related to HGF are mainly involved in regulating peptidyl-tyrosine phosphorylation and positive regulation of the MAPK cascade. Proteins related to TNF are primarily associated with the regulation of I-kappaB kinase/NF-kappaB signaling and cytokine-mediated signaling pathway. HGF is primarily secreted by fibroblasts in HNSC and may have a protective effect on the occurrence and prognosis of HNSC. These effects are independent of immune cell influence, and this role may not necessarily be mediated through the HGF/MET pathway. On the other hand, TNF in LUSC is mainly secreted by immune cells like CD8\\u2009+\\u2009T cell, and it may have a protective effect on the occurrence of LUSC. However, it's impact on the prognosis of LUSC through the immune microenvironment may have a different effect.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/33604698',\n",
       "   'title': 'Multi-omics highlights ABO plasma protein as a causal risk factor for COVID-19.',\n",
       "   'abstract': 'SARS-CoV-2 is responsible for the coronavirus disease 2019 (COVID-19) and the current health crisis. Despite intensive research efforts, the genes and pathways that contribute to COVID-19 remain poorly understood. We, therefore, used an integrative genomics (IG) approach to identify candidate genes responsible for COVID-19 and its severity. We used Bayesian colocalization (COLOC) and summary-based Mendelian randomization to combine gene expression quantitative trait loci (eQTLs) from the Lung eQTL (n\\u2009=\\u20091,038) and eQTLGen (n\\u2009=\\u200931,784) studies with published COVID-19 genome-wide association study (GWAS) data from the COVID-19 Host Genetics Initiative. Additionally, we used COLOC to integrate plasma protein quantitative trait loci (pQTL) from the INTERVAL study (n\\u2009=\\u20093,301) with COVID-19 loci. Finally, we determined any causal associations between plasma proteins and COVID-19 using multi-variable two-sample Mendelian randomization (MR). The expression of 18 genes in lung and/or blood co-localized with COVID-19 loci. Of these, 12 genes were in suggestive loci (P'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38659056',\n",
       "   'title': 'Mendelian randomization and colocalization analysis reveal novel drug targets for myasthenia gravis.',\n",
       "   'abstract': 'Myasthenia gravis (MG) is a complex autoimmune disease affecting the neuromuscular junction with limited drug options, but the field of MG treatment recently benefits from novel biological agents. We performed a drug-targeted Mendelian randomization (MR) study to identify novel therapeutic targets of MG. Cis-expression quantitative loci (cis-eQTL), which proxy expression levels for 2176 druggable genes, were used for MR analysis. Causal relationships between genes and disease, identified by eQTL MR analysis, were verified by comprehensive sensitivity, colocalization, and protein quantitative loci (pQTL) MR analyses. The protein-protein interaction (PPI) analysis was also performed to extend targets, followed by enzyme-linked immunosorbent assay (ELISA) to explore the serum level of drug targets in MG patients. A phenome-wide MR analysis was then performed to assess side effects with a clinical trial review assessing druggability. The eQTL MR analysis has identified eight potential targets for MG, one for early-onset MG and seven for late-onset MG. Further colocalization analyses indicated that CD226, CDC42BPB, PRSS36, and TNFSF12 possess evidence for colocalization with MG or late-onset MG. pQTL MR analyses identified the causal relations of TNFSF12 and CD226 with MG and late-onset MG. Furthermore, PPI analysis has revealed the protein interaction between TNFSF12-TNFSF13(APRIL) and TNFSF12-TNFSF13B(BLyS). Elevated TNFSF13 serum level of MG patients was also identified by ELISA experiments. This study has ultimately proposed three promising therapeutic targets (TNFSF12, TNFSF13, TNFSF13B) of MG. Three drug targets associated with the BLyS/APRIL pathway have been identified. Multiple biological agents, including telitacicept and belimumab, are promising for MG therapy.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40073132',\n",
       "   'title': 'Genetically supported targets and drug repurposing for brain aging: A systematic study in the UK Biobank.',\n",
       "   'abstract': 'Brain age gap (BAG), the deviation between estimated brain age and chronological age, is a promising marker of brain health. However, the genetic architecture and reliable targets for brain aging remains poorly understood. In this study, we estimate magnetic resonance imaging (MRI)-based brain age using deep learning models trained on the UK Biobank and validated with three external datasets. A genome-wide association study for BAG identified two unreported loci and seven previously reported loci. By integrating Mendelian Randomization (MR) and colocalization analysis on eQTL and pQTL data, we prioritized seven genetically supported druggable genes, including '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40129067',\n",
       "   'title': 'Identification of RCC2 as a Risk Gene Associated With Basal Cell Carcinoma and Experimental Validation.',\n",
       "   'abstract': 'Basal cell carcinoma (BCC) is a common type of skin cancer that is increasing in prevalence worldwide. Previous genome-wide association studies (GWAS) have identified certain genetic loci associated with BCC. However, many potential disease-causing genes of BCC remain to be discovered. While the sonic hedgehog (SHH) signalling pathway and mutations in PTCH1/2 and SMO are well-established drivers of BCC pathogenesis, novel genetic factors may complement existing therapeutic targets such as vismodegib and sonidegib. The Mendelian Randomization (MR) study used multiple omics datasets including expression quantitative trait loci (eQTL), methylation quantitative trait loci (mQTL), and protein quantitative trait loci (pQTL) to identify genetic factors associated with an increased risk of developing BCC. Transcriptome analysis of the GEO database then verified the specific expression of key genes. In addition, in\\xa0vitro experiments were used to silence the key gene to observe the effect of this gene on the proliferation ability of A431 cells. Combined with the multi-omics MR Analysis results, six CpG sites were identified with the RCC2 gene associated with BCC risk. Additionally, single-cell transcriptome analysis confirmed the specific expression of RCC2 in the BCC cohort. In the in\\xa0vitro validation experiment, siRCC2-1/2 was transfected into the A431 cells, significantly decreasing the expression of RCC2 in the cells. Moreover, the proliferation of A431 cells was significantly inhibited after RCC2 was knocked down. We identified a risk gene RCC2 associated with BCC by MR-based bioinformatics analysis and demonstrated that inhibition of RCC2 inhibited the proliferation of A431 in\\xa0vitro. These findings provide new strategies for targeted therapy of BCC.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37304261',\n",
       "   'title': 'Circulating levels of cytokines and risk of cardiovascular disease: a Mendelian randomization study.',\n",
       "   'abstract': 'Epidemiological studies have linked various circulating cytokines to cardiovascular disease (CVD), which however remains uncertain whether these relationships represent causality or are due to bias. To address this question, we conducted a Mendelian randomization (MR) analysis to systematically investigate the causal effects of circulating cytokine levels on CVD development. This study leveraged the summary statistic from respective genome-wide association study (GWAS) of 47 cytokines and four types of CVD. The  The results of inverse-variance weighted method using  The present study provides supportive evidence that genetic predisposition to levels of certain cytokines causally affects the development of specific type of CVD. These findings have important implications for the creation of novel therapeutic strategies targeting these cytokines as a means of preventing and treating CVD.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39230170',\n",
       "   'title': 'Genome-Wide Identification of Cell Type-Specific Susceptibility Genes for SLE Through the Analysis of RNA Modification-Associated SNPs.',\n",
       "   'abstract': 'This study aimed to elucidate the functional genes associated with systemic lupus erythematosus (SLE) in various cell types through the utilization of RNAm-SNPs. Utilizing large-scale genetic data, we identified associations between RNAm-SNPs and SLE. The association between RNAm-SNPs and bulk and single-cell mRNA expression (eQTL) and protein levels (pQTL) were examined. Mendelian randomization and differential expression analyses were conducted to explore the links between gene expression, protein levels, and SLE. We identified 41 RNAm-SNPs that were significantly associated with SLE. The GWAS signals exhibited notable enrichment in m This study advances our understanding of the genetic basis of SLE by highlighting the significance of RNAm-SNPs and immune cell gene expression in SLE.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30595531',\n",
       "   'title': 'Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML.',\n",
       "   'abstract': 'The design of selective small molecules is often stymied by similar ligand binding pockets. Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degradation of cyclin-dependent kinase 6 (CDK6). Pharmacologic CDK6 degradation targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degradation of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36343264',\n",
       "   'title': 'Mapping prohormone processing by proteases in human enteroendocrine cells using genetically engineered organoid models.',\n",
       "   'abstract': 'Enteroendocrine cells (EECs) secrete hormones in response to ingested nutrients to control physiological processes such as appetite and insulin release. EEC hormones are synthesized as large proproteins that undergo proteolytic processing to generate bioactive peptides. Mutations in EEC-enriched proteases are associated with endocrinopathies. Due to the relative rarity of EECs and a paucity of in\\xa0vitro models, intestinal prohormone processing remains challenging to assess. Here, human gut organoids in which EECs can efficiently be induced are subjected to CRISPR-Cas9-mediated modification of EEC-expressed endopeptidase and exopeptidase genes. We employ mass spectrometry-based analyses to monitor peptide processing and identify glucagon production in intestinal EECs, stimulated upon bone morphogenic protein (BMP) signaling. We map the substrates and products of major EECs endo- and exopeptidases. Our studies provide a comprehensive description of peptide hormones produced by human EECs and define the roles of specific proteases in their generation.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/28424047',\n",
       "   'title': 'Deciphering mechanisms underlying the genetic variation of general production and liver quality traits in the overfed mule duck by pQTL analyses.',\n",
       "   'abstract': 'The aim of this study was to analyse the mechanisms that underlie phenotypic quantitative trait loci (QTL) in overfed mule ducks by identifying co-localized proteomic QTL (pQTL). The QTL design consisted of three families of common ducks that were progeny-tested by using 294 male mule ducks. This population of common ducks was genotyped using a genetic map that included 334 genetic markers located across 28 APL chromosomes (APL for Anas platyrhynchos). Mule ducks were phenotyped for 49 traits related to growth, metabolism, overfeeding ability and meat and fatty liver quality, and 326 soluble fatty liver proteins were quantified. One hundred and seventy-six pQTL and 80 phenotypic QTL were detected at the 5% chromosome-wide significance threshold. The great majority of the identified pQTL were trans-acting and localized on a chromosome other than that carrying the coding gene. The most significant pQTL (1% genome-wide significance) were found for alpha-enolase on APL18 and fatty acid synthase on APL24. Some proteins were associated with numerous pQTL (for example, 17 and 14 pQTL were detected for alpha-enolase and apolipoprotein A1, respectively) and pQTL hotspots were observed on some chromosomes (APL18, 24, 25 and 29). We detected 66 co-localized phenotypic QTL and pQTL for which the significance of the two-trait QTL (2t-QTL) analysis was higher than that of the strongest QTL using a single-trait approach. Among these, 16 2t-QTL were pleiotropic. For example, on APL15, melting rate and abundance of two alpha-enolase spots appeared to be impacted by a single locus that is involved in the glycolytic process. On APLZ, we identified a pleiotropic QTL that modified both the blood level of glucose at the beginning of the force-feeding period and the concentration of glutamate dehydrogenase, which, in humans, is involved in increased glucose absorption by the liver when the glutamate dehydrogenase 1 gene is mutated. We identified pleiotropic loci that affect metabolic pathways linked to glycolysis or lipogenesis, and in the end to fatty liver quality. Further investigation, via transcriptomics and metabolomics approaches, is required to confirm the biomarkers that were found to impact the genetic variability of these phenotypic traits.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32492067',\n",
       "   'title': 'A fully joint Bayesian quantitative trait locus mapping of human protein abundance in plasma.',\n",
       "   'abstract': 'Molecular quantitative trait locus (QTL) analyses are increasingly popular to explore the genetic architecture of complex traits, but existing studies do not leverage shared regulatory patterns and suffer from a large multiplicity burden, which hampers the detection of weak signals such as trans associations. Here, we present a fully multivariate proteomic QTL (pQTL) analysis performed with our recently proposed Bayesian method LOCUS on data from two clinical cohorts, with plasma protein levels quantified by mass-spectrometry and aptamer-based assays. Our two-stage study identifies 136 pQTL associations in the first cohort, of which >80% replicate in the second independent cohort and have significant enrichment with functional genomic elements and disease risk loci. Moreover, 78% of the pQTLs whose protein abundance was quantified by both proteomic techniques are confirmed across assays. Our thorough comparisons with standard univariate QTL mapping on (1) these data and (2) synthetic data emulating the real data show how LOCUS borrows strength across correlated protein levels and markers on a genome-wide scale to effectively increase statistical power. Notably, 15% of the pQTLs uncovered by LOCUS would be missed by the univariate approach, including several trans and pleiotropic hits with successful independent validation. Finally, the analysis of extensive clinical data from the two cohorts indicates that the genetically-driven proteins identified by LOCUS are enriched in associations with low-grade inflammation, insulin resistance and dyslipidemia and might therefore act as endophenotypes for metabolic diseases. While considerations on the clinical role of the pQTLs are beyond the scope of our work, these findings generate useful hypotheses to be explored in future research; all results are accessible online from our searchable database. Thanks to its efficient variational Bayes implementation, LOCUS can analyze jointly thousands of traits and millions of markers. Its applicability goes beyond pQTL studies, opening new perspectives for large-scale genome-wide association and QTL analyses. Diet, Obesity and Genes (DiOGenes) trial registration number: NCT00390637.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35425820',\n",
       "   'title': 'Association of Circulating Cathepsin B Levels With Blood Pressure and Aortic Dilation.',\n",
       "   'abstract': 'Hypertension is a key risk factor for spontaneous coronary artery dissection (SCAD) and aortic dilation. Circulating proteins play key roles in a range of biological processes and represent a major source of druggable targets. The aim of this study was to identify circulating proteins that were associated with blood pressure (BP), SCAD and aortic dilation. We identified shared genetic variants of BP and SCAD in genome-wide association studies, searched for circulating protein affected by these variants and examined the association of circulating protein levels with BP, aortic aneurysm and dissection (AAD) and aortic diameters by integrating data from circulating protein quantitative trait loci (pQTL) studies and genome wide association study (GWAS) in individuals from the UK Biobank using two-sample Mendelian randomization analysis methods. Single nucleotide polymorphisms (SNPs) in  The present study identified the association between circulating cathepsin B and BP and aortic diameters. The findings indicated that BP-associated genetic variants may influence aortic dilation risk by circulating proteins that regulate BP.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40269081',\n",
       "   'title': 'Multiomic insight into the involvement of cell aging related genes in the pathogenesis of endometriosis.',\n",
       "   'abstract': \"Endometriosis significantly impacts women's health and fertility, with cell aging playing a crucial role in its development. This study utilized a multi-omic summary Mendelian randomization (SMR) analysis, integrating genome-wide association studies (GWAS), expression quantitative trait loci (eQTLs), methylation quantitative trait loci (mQTLs), and protein quantitative trait loci (pQTLs). The goal was to identify genes that exhibit causal associations between cell aging and endometriosis. Validation was conducted using the FinnGen R10 and UK Biobank cohorts. The SMR and HEIDI tests evaluated the genetic variants linked to both cell aging and endometriosis risk. Colocalization analysis revealed shared genetic variants, uncovering significant associations between the two conditions. A total of 196 CpG sites in 78 genes, alongside 18 eQTL-associated genes and 7 pQTL-associated proteins, were identified. Notably, the MAP3K5 gene displayed contrasting methylation patterns linked to endometriosis risk. In validation cohorts, the THRB gene and ENG protein were confirmed as risk factors. The findings suggest a causal mechanism where specific methylation patterns downregulate the MAP3K5 gene, heightening endometriosis risk, highlighting it and associated pathways as potential therapeutic targets.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39739311',\n",
       "   'title': 'Spatial proteomics reveals sirtuin 1 to be a determinant of T-cell infiltration in human melanoma.',\n",
       "   'abstract': 'The tumour microenvironment significantly influences the clinical response of patients to therapeutic immune checkpoint inhibition (ICI), but a comprehensive understanding of the underlying immune-regulatory proteome is still lacking. To decipher targetable biologic processes that determine tumour-infiltrating lymphocytes (TiLs) as a cellular equivalent of clinical response to ICI. We mapped the spatial distribution of proteins in TiL-enriched vs. TiL-low compartments in melanoma by combining microscopy, matrix-assisted laser desorption mass spectrometry imaging and liquid chromatography-mass spectrometry, as well as computational data mining. Pharmacological modulation of sirtuin 1 (SIRT1) activity in syngeneic mouse models was used to evaluate the efficacy of pharmacological SIRT1 activation in two syngeneic melanoma mouse models, one known to be α-programmed cell death protein 1 (PD-1) sensitive and the other α-PD-1 resistant. Spatial proteomics and gene ontology-based enrichment analysis identified > 145 proteins enriched in CD8high tumour compartments, including negative regulators of mammalian target of rapamycin signalling such as SIRT1. Multiplexed immunohistochemistry confirmed that SIRT1 protein was expressed more in CD8high than in CD8low compartments. Further analysis of bulk and single-cell RNA sequencing data from melanoma tissue samples suggested the expression of SIRT1 by different lymphocyte subpopulations (CD8+ T cells, CD4+ T cells and B cells). Furthermore, we showed in vivo that pharmacological SIRT1 activation increased the immunological effect of α-PD-1 ICI against melanoma cells in mice, which was accompanied by an increase in T-cell infiltration and T-cell-related cytokines, including interferon (IFN)-γ, CCL4, CXCL9, CXCL10 and tumour necrosis factor-α. In silico analysis of large transcriptional data cohorts showed that SIRT1 was positively associated with the proinflammatory T-cell chemokines CXCL9, CXCL10 and IFN-γ, and prolonged overall survival of patients with melanoma. Our study deciphers the proteomics landscape in human melanoma, providing important information on the tumour microenvironment and identifying SIRT1 as having important prognostic and therapeutic implications.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39006426',\n",
       "   'title': 'Cross-Phenotype GWAS Supports Shared Genetic Susceptibility to Systemic Sclerosis and Primary Biliary Cholangitis.',\n",
       "   'abstract': 'An increased risk of primary biliary cholangitis (PBC) has been reported in patients with systemic sclerosis (SSc). Our study aims to investigate the shared genetic susceptibility between the two disorders and to define candidate causal genes using cross-phenotype GWAS meta-analysis. We performed cross-phenotype GWAS meta-analysis and colocalization analysis for SSc and PBC. We performed both genome-wide and locus-based analysis, including tissue and pathway enrichment analyses, fine-mapping, colocalization analyses with expression quantitative trait loci (eQTL) and protein quantitative trait loci (pQTL) datasets, and phenome-wide association studies (PheWAS). Finally, we used an integrative approach to prioritize candidate causal genes from the novel loci. We detected a strong genetic correlation between SSc and PBC (rg = 0.84, p = 1.7 × 10 Our study supports a strong shared genetic susceptibility between SSc and PBC. Through cross-phenotype analyses, we have prioritized several novel candidate causal genes and pathways for these disorders.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32186727',\n",
       "   'title': 'Multi-Trait Genome-Wide Association Studies Reveal Loci Associated with Maize Inflorescence and Leaf Architecture.',\n",
       "   'abstract': 'Maize inflorescence is a complex phenotype that involves the physical and developmental interplay of multiple traits. Given the evidence that genes could pleiotropically contribute to several of these traits, we used publicly available maize data to assess the ability of multivariate genome-wide association study (GWAS) approaches to identify pleiotropic quantitative trait loci (pQTL). Our analysis of 23 publicly available inflorescence and leaf-related traits in a diversity panel of n\\u2009=\\u2009281 maize lines genotyped with 376,336 markers revealed that the two multivariate GWAS approaches we tested were capable of identifying pQTL in genomic regions coinciding with similar associations found in previous studies. We then conducted a parallel simulation study on the same individuals, where it was shown that multivariate GWAS approaches yielded a higher true-positive quantitative trait nucleotide (QTN) detection rate than comparable univariate approaches for all evaluated simulation settings except for when the correlated simulated traits had a heritability of 0.9. We therefore conclude that the implementation of state-of-the-art multivariate GWAS approaches is a useful tool for dissecting pleiotropy and their more widespread implementation could facilitate the discovery of genes and other biological mechanisms underlying maize inflorescence.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37492104',\n",
       "   'title': 'Multi-layered genetic approaches to identify approved drug targets.',\n",
       "   'abstract': 'Drugs targeting genes linked to disease via evidence from human genetics have increased odds of approval. Approaches to prioritize such genes include genome-wide association studies (GWASs), rare variant burden tests in exome sequencing studies (Exome), or integration of a GWAS with expression/protein quantitative trait loci (eQTL/pQTL-GWAS). Here, we compare gene-prioritization approaches on 30 clinically relevant traits and benchmark their ability to recover drug targets. Across traits, prioritized genes were enriched for drug targets with odds ratios (ORs) of 2.17, 2.04, 1.81, and 1.31 for the GWAS, eQTL-GWAS, Exome, and pQTL-GWAS methods, respectively. Adjusting for differences in testable genes and sample sizes, GWAS outperforms e/pQTL-GWAS, but not the Exome approach. Furthermore, performance increased through gene network diffusion, although the node degree, being the best predictor (OR\\xa0= 8.7), revealed strong bias in literature-curated networks. In conclusion, we systematically assessed strategies to prioritize drug target genes, highlighting the promises and pitfalls of current approaches.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40001968',\n",
       "   'title': 'Assessing the Causal Relationship Between Plasma Proteins and Pulmonary Fibrosis: A Systematic Analysis Based on Mendelian Randomization.',\n",
       "   'abstract': 'Pulmonary fibrosis (PF) is a chronic interstitial lung disease characterized by the destruction of alveolar structures, the abnormal accumulation of extracellular matrix (ECM), and ultimately respiratory failure. Although previous studies have shown that plasma proteins play an important role in the onset and progression of PF, there is currently a lack of systematic studies on causal relationships. To address the identified gap, the study employs the MR method to identify potential drug targets associated with PF. Plasma protein data (pQTL, exposure) were sourced from Ferkingstad et al. ('},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40214710',\n",
       "   'title': 'Identifying Potential Drug Targets in Coronary Atherosclerosis: Insights from the Druggable Genome and Mendelian Randomization.',\n",
       "   'abstract': 'This study aims to identify therapeutic targets for coronary atherosclerosis (CA) using publicly available datasets while exploring its pathophysiologic mechanisms, mediators and potential side effects. We conducted a two-sample Mendelian randomization (MR) and single-cell MR analyses integrating identified druggable genes to evaluate the causal relationship between expression quantitative trait loci (eQTL) and CA in both peripheral and central tissues. Using peripheral protein quantitative trait loci (pQTL) data, we further validated the identified targets at the proteomic level through summary data-based MR (SMR) and HEIDI tests. Concurrently, mediation MR analysis was employed to investigate potential mechanistic pathways underlying the role of these targets in CA. Additionally, a phenotype-wide MR (Phe-MR) analysis was performed to explore other potential indications for the therapeutic application of the identified targets. In conclusion, we identified three CA-associated genes in peripheral tissues (VAMP8, MFGE8 and PDGFD) two CA-associated genes in central tissues (GGCX and NPEPPS). In addition, single-cell MR analyses revealed that GGCX was associated with increased CA risk in excitatory, inhibitory and oligodendrocyte precursor cells, whereas NPEPPS was associated with protection in oligodendrocyte lineage cells. Finally, Phe-MR analyses indicated possible indications and side effects of the targets. Our study provides genetic evidence for VAMP8, MFGE8, PDGFD, GGCX and NPEPPS as potential therapeutic targets for CA, highlighting their clinical relevance, associated risks and mediators, and providing valuable insights for the development of novel CA therapeutics.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38969939',\n",
       "   'title': 'Fine mapping of candidate effector genes for heart rate.',\n",
       "   'abstract': 'An elevated resting heart rate (RHR) is associated with increased cardiovascular mortality. Genome-wide association studies (GWAS) have identified\\u2009>\\u2009350 loci. Uniquely, in this study we applied genetic fine-mapping leveraging tissue specific chromatin segmentation and colocalization analyses to identify causal variants and candidate effector genes for RHR. We used RHR GWAS summary statistics from 388,237 individuals of European ancestry from UK Biobank and performed fine mapping using publicly available genomic annotation datasets. High-confidence causal variants (accounting for >\\u200975% posterior probability) were identified, and we collated candidate effector genes using a multi-omics approach that combined evidence from colocalisation with molecular quantitative trait loci (QTLs), and long-range chromatin interaction analyses. Finally, we performed druggability analyses to investigate drug repurposing opportunities. The fine mapping pipeline indicated 442 distinct RHR signals. For 90 signals, a single variant was identified as a high-confidence causal variant, of which 22 were annotated as missense. In trait-relevant tissues, 39 signals colocalised with cis-expression QTLs (eQTLs), 3 with cis-protein QTLs (pQTLs), and 75 had promoter interactions via Hi-C. In total, 262 candidate genes were highlighted (79% had promoter interactions, 15% had a colocalised eQTL, 8% had a missense variant and 1% had a colocalised pQTL), and, for the first time, enrichment in nervous system pathways. Druggability analyses highlighted ACHE, CALCRL, MYT1 and TDP1 as potential targets. Our genetic fine-mapping pipeline prioritised 262 candidate genes for RHR that warrant further investigation in functional studies, and we provide potential therapeutic targets to reduce RHR and cardiovascular mortality.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39671389',\n",
       "   'title': 'Exploration oxidative stress underlying gastroesophageal reflux disease and therapeutic targets identification: a multi-omics Mendelian randomization study.',\n",
       "   'abstract': \"Gastroesophageal reflux disease (GERD) is a chronic inflammatory gastrointestinal disease, which has no thoroughly effective or safe treatment. Elevated oxidative stress is a common consequence of chronic inflammatory conditions. We employed Summary-data based MR (SMR) analysis to assess the associations between gene molecular characteristics and GERD. Exposure data were the summary-level data on the levels of DNA methylation, gene expression, and protein expression, which obtained from related methylation, expression, and protein quantitative trait loci investigations (mQTL, eQTL, and pQTL). Outcome data, Genome-wide association study (GWAS) summary statistics of GERD, were extracted from the Ong's study (discovery), the Dönertaş's study (replication), and the FinnGen study (replication). Colocalization analysis was performed to determine if the detected signal pairs shared a causative genetic mutation. Oxidative stress related genes and druggable genes were imported to explore oxidative stress mechanism underlying GERD and therapeutic targets of GERD. The Drugbank database was utilized to conduct druggability evaluation. After multi-omics SMR analysis and colocalization analysis, we identified seven key genes for GERD, which were SUOX and SERPING1, DUSP13, SULT1A1, LMOD1, UBE2L6, and PSCA. SUOX was screened out to be the mediator, which suggest that GERD is related to oxidative stress. SERPING1, SULT1A1, and PSCA were selected to be the druggable genes. These findings offered strong support for the identification of GERD treatment targets in the future as well as for the study of the oxidative stress mechanism underlying GERD.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/16075419',\n",
       "   'title': 'Proteomic study of human bronchoalveolar lavage fluids from smokers with chronic obstructive pulmonary disease by combining surface-enhanced laser desorption/ionization-mass spectrometry profiling with mass spectrometric protein identification.',\n",
       "   'abstract': 'Bronchoalveolar lavage fluid (BALF) is an important diagnostic source to investigate cellular and molecular changes in the course of lung disorders. The pattern of soluble proteins in BALF obtained from patients at different stages of respiratory disorders may provide deeper insights in the molecular mechanisms of the disease. We used surface-enhanced laser desorption/ionization mass spectrometry (MS) for differential protein display combined with reversed-phase chromatography and subsequent matrix-assisted laser desorption/ionization-MS or nanoliquid chromatography MS/MS analysis for protein identification to compare the protein pattern of BALF samples obtained from ten smokers suffering from chronic obstructive pulmonary disease (COPD), eight clinically asymptomatic smokers, and eight nonsmokers without pulmonary disease. In this context, we were able to identify small proteins and peptides, either differentially expressed or secreted in the course of COPD or in a direct response to cigarette smoke. The concentrations of neutrophil defensins 1 and 2, S100A8 (calgranulin A), and S100A9 (calgranulin B) were elevated in BALFs of smokers with COPD when compared to asymptomatic smokers. Increased concentrations in S100A8 (Calgranulin A), salivary proline-rich peptide P-C, and lysozyme C were detected in BALFs of asymptomatic smokers when compared to nonsmokers, whereas salivary proline-rich peptide P-D and Clara cell phospholipid-binding protein (CC10) were reduced in their concentration. The identified proteins and peptides might be useful in the future as diagnostic markers for smoke-induced lung irritations and COPD.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39273172',\n",
       "   'title': 'Multi-Omics Integration Analysis Pinpoint Proteins Influencing Brain Structure and Function: Toward Drug Targets and Neuroimaging Biomarkers for Neuropsychiatric Disorders.',\n",
       "   'abstract': 'Integrating protein quantitative trait loci (pQTL) data and summary statistics from genome-wide association studies (GWAS) of brain image-derived phenotypes (IDPs) can benefit in identifying IDP-related proteins. Here, we developed a systematic omics-integration analytic framework by sequentially using proteome-wide association study (PWAS), Mendelian randomization (MR), and colocalization (COLOC) analyses to identify the potentially causal brain and plasma proteins for IDPs, followed by pleiotropy analysis, mediation analysis, and drug exploration analysis to investigate potential mediation pathways of pleiotropic proteins to neuropsychiatric disorders (NDs) as well as candidate drug targets. A total of 201 plasma proteins and 398 brain proteins were significantly associated with IDPs from PWAS analysis. Subsequent MR and COLOC analyses further identified 313 potentially causal IDP-related proteins, which were significantly enriched in neural-related phenotypes, among which 91 were further identified as pleiotropic proteins associated with both IDPs and NDs, including '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40128726',\n",
       "   'title': 'Multi-omic studies on the pathogenesis of Sepsis.',\n",
       "   'abstract': 'Sepsis is a life-threatening inflammatory condition, and its underlying genetic mechanisms are not yet fully elucidated. We applied methods such as Mendelian randomization (MR), genetic correlation analysis, and colocalization analysis to integrate multi-omics data and explore the relationship between genetically associated genes and sepsis, as well as sepsis-related mortality, with the goal of identifying key genetic factors and their potential mechanistic pathways. To identify therapeutic targets for sepsis and sepsis-related mortality, we conducted an MR analysis on 11,643 sepsis cases and 1,896 cases of 28-day sepsis mortality from the UK Biobank cohort. The exposure data consisted of 15,944 potential druggable genes (expression quantitative trait loci, eQTL) and 4,907 plasma proteins (protein quantitative trait loci, pQTL). We then performed sensitivity analysis, SMR analysis, reverse MR analysis, genetic correlation analysis, colocalization analysis, enrichment analysis, and protein-protein interaction network analysis on the overlapping genes. Validation was conducted using 17,133 sepsis cases from FinnGen R12. Drug prediction and molecular docking were subsequently used to further assess the therapeutic potential of the identified drug targets, while PheWAS was used to evaluate potential side effects. Finally, mediation analysis was conducted to identify the mediating role of related metabolites. The MR analysis results identified a significant causal relationship between 24 genes and sepsis. The robustness of these causal associations was further strengthened by SMR analysis, sensitivity analysis, and reverse MR analysis. Genetic correlation analysis revealed that only two of these genes were genetically correlated with sepsis. Colocalization analysis showed that only one gene was closely associated with sepsis, while validation using the FinnGen dataset identified three genes. In the MR analysis of 28-day sepsis mortality, seven genes were found to have significant associations, with reverse MR analysis excluding one gene. The remaining genes passed sensitivity analysis, with no significant genes identified in genetic correlation and colocalization analyses. Molecular docking demonstrated excellent binding affinity between drugs and proteins with available structural data. PheWAS at the gene level did not reveal any potential side effects of the related drugs. The identified drug targets, associated pathways, and metabolites have enhanced our understanding of the complex relationships between genes and sepsis. These genes and metabolites can serve as effective targets for sepsis treatment, paving new pathways in this field and laying a foundation for future research.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/34491321',\n",
       "   'title': 'The Effect of Phenotype and Genotype on the Plasma Proteome in Patients with Inflammatory Bowel Disease.',\n",
       "   'abstract': \"Protein profiling in patients with inflammatory bowel diseases [IBD] for diagnostic and therapeutic purposes is underexplored. This study analysed the association between phenotype, genotype, and the plasma proteome in IBD. A total of 92 inflammation-related proteins were quantified in plasma of 1028 patients with IBD (567 Crohn's disease [CD]; 461 ulcerative colitis [UC]) and 148 healthy individuals to assess protein-phenotype associations. Corresponding whole-exome sequencing and global screening array data of 919 patients with IBD were included to analyse the effect of genetics on protein levels (protein quantitative trait loci [pQTL] analysis). Intestinal mucosal RNA sequencing and faecal metagenomic data were used for complementary analyses. Thirty-two proteins were differentially abundant between IBD and healthy individuals, of which 22 proteins were independent of active inflammation; 69 proteins were associated with 15 demographic and clinical factors. Fibroblast growth factor-19 levels were decreased in CD patients with ileal disease or a history of ileocecal resection. Thirteen novel cis-pQTLs were identified and 10 replicated from previous studies. One trans-pQTL of the fucosyltransferase 2 [FUT2] gene [rs602662] and two independent cis-pQTLs of C-C motif chemokine 25 [CCL25] affected plasma CCL25 levels. Intestinal gene expression data revealed an overlapping cis-expression [e]QTL-variant [rs3745387] of the CCL25 gene. The FUT2 rs602662 trans-pQTL was associated with reduced abundances of faecal butyrate-producing bacteria. This study shows that genotype and multiple disease phenotypes strongly associate with the plasma inflammatory proteome in IBD, and identifies disease-associated pathways that may help to improve disease management in the future.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/28215025',\n",
       "   'title': 'Analysis of the genetic architecture of maize ear and grain morphological traits by combined linkage and association mapping.',\n",
       "   'abstract': 'Using combined linkage and association mapping, 26 stable QTL and six stable SNPs were detected across multiple environments for eight ear and grain morphological traits in maize. One QTL, PKS2, might play an important role in maize yield improvement. In the present study, one bi-parental population and an association panel were used to identify quantitative trait loci (QTL) for eight ear and grain morphological traits. A total of 108 QTL related to these traits were detected across four environments using an ultra-high density bin map constructed using recombinant inbred lines (RILs) derived from a cross between Ye478 and Qi319, and 26 QTL were identified in more than two environments. Furthermore, 64 single nucleotide polymorphisms (SNPs) were found to be significantly associated with the eight ear and grain morphological traits (-log'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37082146',\n",
       "   'title': 'Genetic dissection of the pluripotent proteome through multi-omics data integration.',\n",
       "   'abstract': 'Genetic background drives phenotypic variability in pluripotent stem cells (PSCs). Most studies to date have used transcript abundance as the primary molecular readout of cell state in PSCs. We performed a comprehensive proteogenomics analysis of 190 genetically diverse mouse embryonic stem cell (mESC) lines. The quantitative proteome is highly variable across lines, and we identified pluripotency-associated pathways that were differentially activated in the proteomics data that were not evident in transcriptome data from the same lines. Integration of protein abundance to transcript levels and chromatin accessibility revealed broad co-variation across molecular layers as well as shared and unique drivers of quantitative variation in pluripotency-associated pathways. Quantitative trait locus (QTL) mapping localized the drivers of these multi-omic signatures to genomic hotspots. This study reveals post-transcriptional mechanisms and genetic interactions that underlie quantitative variability in the pluripotent proteome and provides a regulatory map for mESCs that can provide a basis for future mechanistic studies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39857814',\n",
       "   'title': 'Mendelian Randomization Reveals Potential Causal Relationships Between DNA Damage Repair-Related Genes and Inflammatory Bowel Disease.',\n",
       "   'abstract': 'DNA damage repair (DDR) plays a key role in maintaining genomic stability and developing inflammatory bowel disease (IBD). However, no report about the causal association between DDR and IBD exists. Whether DDR-related genes are the precise causal association to IBD in etiology remains unclear. Herein, we employed a multi-omics summary data-based Mendelian randomization (SMR) approach to ascertain the potential causal effects of DDR-related genes in IBD. '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37468619',\n",
       "   'title': 'Organ Mapping Antibody Panels: a community resource for standardized multiplexed tissue imaging.',\n",
       "   'abstract': 'Multiplexed antibody-based imaging enables the detailed characterization of molecular and cellular organization in tissues. Advances in the field now allow high-parameter data collection (>60 targets); however, considerable expertise and capital are needed to construct the antibody panels employed by these methods. Organ mapping antibody panels are community-validated resources that save time and money, increase reproducibility, accelerate discovery and support the construction of a Human Reference Atlas.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38863531',\n",
       "   'title': 'Genome-wide association study and genomic selection of flax powdery mildew in Xinjiang Province.',\n",
       "   'abstract': 'Flax powdery mildew (PM), caused by '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36931692',\n",
       "   'title': 'Meta-analysis of erosive hand osteoarthritis identifies four common variants that associate with relatively large effect.',\n",
       "   'abstract': 'Erosive hand osteoarthritis (EHOA) is a severe subset of hand osteoarthritis (OA). It is unclear if EHOA is genetically different from other forms of OA. Sequence variants at ten loci have been associated with hand OA but none with EHOA. We performed meta-analysis of EHOA in 1484 cases and 550\\u2009680 controls, from 5 populations. To identify causal genes, we performed eQTL and plasma pQTL analyses, and developed one zebrafish mutant. We analysed associations of variants with other traits and estimated shared genetics between EHOA and other traits. Four common sequence variants associated with EHOA, all with relatively high effect. Rs17013495 ( We report on significant genetic associations with EHOA. The results support the view of EHOA as a form of severe hand OA and partly separate it from OA in larger joints.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39215079',\n",
       "   'title': 'Mendelian randomization reveals potential causal relationships between cellular senescence-related genes and multiple cancer risks.',\n",
       "   'abstract': 'Cellular senescence is widely acknowledged as having strong associations with cancer. However, the intricate relationships between cellular senescence-related (CSR) genes and cancer risk remain poorly explored, with insights on causality remaining elusive. In this study, Mendelian Randomization (MR) analyses were used to draw causal inferences from 866 CSR genes as exposures and summary statistics for 18 common cancers as outcomes. We focused on genetic variants affecting gene expression, DNA methylation, and protein expression quantitative trait loci (cis-eQTL, cis-mQTL, and cis-pQTL, respectively), which were strongly linked to CSR genes alterations. Variants were selected as instrumental variables (IVs) and analyzed for causality with cancer using both summary-data-based MR (SMR) and two-sample MR (TSMR) approaches. Bayesian colocalization was used to unravel potential regulatory mechanisms underpinning risk variants in cancer, and further validate the robustness of MR results. We identified five CSR genes (CNOT6, DNMT3B, MAP2K1, TBPL1, and SREBF1), 18 DNA methylation genes, and LAYN protein expression which were all causally associated with different cancer types. Beyond causality, a comprehensive analysis of gene function, pathways, and druggability values was also conducted. These findings provide a robust foundation for unravelling CSR genes molecular mechanisms and promoting clinical drug development for cancer.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39495227',\n",
       "   'title': 'Mendelian Randomization Estimates the Effects of Plasma and Cerebrospinal Fluid Proteins on Intelligence, Fluid Intelligence Score, and Cognitive Performance.',\n",
       "   'abstract': 'Observational studies have revealed associations between levels of plasma and cerebrospinal fluid (CSF) proteins and cognition-related traits. However, these associations may be influenced by confounding factors inherent in observational research. This study aims to identify plasma and CSF proteins associated with intelligence, fluid intelligence score, and cognitive performance through the application of Mendelian randomization (MR). Proteomic quantitative trait locus (pQTL) data for plasma and CSF proteins were sourced from existing genome-wide association study (GWAS). Intelligence, fluid intelligence score, and cognitive performance GWAS summary statistics provided comprehensive data for two-sample MR analysis. Extensive sensitivity analyses, including Steiger testing, reverse MR analysis, and Bayesian co-localization, were conducted to validate associations and identify shared genetic variants. Phenotype scanning explored potential pleiotropic effects. MR analysis identified several proteins in plasma and CSF significantly associated with intelligence, fluid intelligence scores, and cognitive performance. For intelligence, negatively associated proteins in plasma include endoplasmic reticulum aminopeptidase 2 (ERAP2) and secretogranin III (SCG3), while positively associated proteins are myeloperoxidase (MPO), signal regulatory protein alpha (SIRPA), regulator of microtubule dynamics 1 (RMDN1), and endoplasmic reticulum lectin 1 (ERLEC1). In CSF, C1-esterase inhibitor and carboxypeptidase E (CBPE) both exhibited positive associations with intelligence. For fluid intelligence scores, negatively associated proteins in plasma are copine 1 (CPNE1) and SCG3, while positively associated proteins are nudix hydrolase 12 (NUDT12) and RMDN1.\\xa0In CSF, Macrophage Stimulating Protein (MSP) demonstrated a significant negative impact. For cognitive performance, negatively associated proteins in plasma include ERAP2, tyrosine kinase with immunoglobulin-like and EGF-like domains 1 (TIE1), and SCG3, while positively associated proteins are NUDT12, RMDN1, ERLEC1, and ectonucleotide pyrophosphatase/phosphodiesterase family member 5 (ENPP5). In CSF, C1-esterase inhibitor was positively associated, while MSP and soluble tyrosine kinase with immunoglobulin-like and EGF-like domains 1(sTie-1) showed a negative association. Bayesian co-localization analysis revealed significant genetic overlaps between SIRPA, RMDN1, and ERLEC1 in plasma with intelligence; NUDT12 and SCG3 in plasma with fluid intelligence scores; and TIE1, NUDT12, RMDN1, ERLEC1, and ENPP5 in plasma with cognitive performance. Additionally, significant co-localization was identified between C1-esterase inhibitor and CBPE in CSF with intelligence, as well as between C1-esterase inhibitor and sTie-1 in CSF with cognitive performance. Reverse causality analysis confirmed the causal direction from proteins to cognitive traits. This study identifies specific plasma and CSF proteins that significantly impact intelligence, fluid intelligence scores, and cognitive performance. These proteins could serve as biomarkers and targets for future research and therapeutic interventions aimed at sustaining cognitive abilities and reducing impairment risks.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40311696',\n",
       "   'title': 'Comprehensive genetic analysis based on multi - omics reveals novel therapeutic targets for mitral valve prolapse and drug molecular dynamics simulation.',\n",
       "   'abstract': 'Mitral valve prolapse (MVP), the most prevalent primary valvular disease, serves as a direct risk factor for multiple cardiovascular disorders and exhibits a high prevalence in the general population. As no specific pharmacological therapies currently exist for MVP, the identification of precise therapeutic targets is imperative. We conducted comprehensive causal genetic inference by integrating genetic data from expression quantitative trait loci (eQTL) and genome-wide association studies (GWAS). Analytical approaches included Mendelian Randomization (MR), colocalization analysis, Summary-data-based Mendelian Randomization (SMR), Linkage Disequilibrium Score Regression (LDSC), and High-Definition Likelihood (HDL) analysis. Protein quantitative trait loci (pQTL) were utilized to validate gene expression. Replication analyses were performed using additional exposure datasets. Methylation quantitative trait loci (mQTL) were employed to elucidate regulatory roles of methylation sites on genes and disease pathogenesis. Phenome-Wide Association Study (PheWAS) was conducted to predict potential adverse effects of gene-targeted therapies. Drug candidates targeting identified genes were predicted via the Drug Signature Database (DSigDB) and validated through molecular docking. Core targets were identified using the STRING database, followed by molecular dynamics simulations. Two-sample MR analysis showed that genetically predicted 266 genes had positive or negative causal relationships with MVP. Colocalization analysis indicated that 9 genes had a posterior probability greater than 0.75. Subsequent SMR analysis excluded the gene GAPVD1. HDL analysis showed that except for the gene PTPN1, the remaining 7 genes were all significantly genetically associated with MVP, and LDSC analysis further showed that only NMB was associated with MVP. Validation using pQTL data confirmed that increased NMB protein expression reduced the risk of MVP. Replication analysis further verified this conclusion. In addition, SMR analysis of methylation sites for 8 genes indicated that multiple methylation sites played a key role in gene regulation of mitral valve prolapse. PheWAS results showed that targeted therapy for 8 genes did not detect other causal associations at the genome-wide significance level. Molecular docking showed that quercetin had good binding ability with 8 target genes. The STRING database identified 3 core target proteins, and molecular dynamics simulations further verified the binding ability of quercetin with core target proteins. This study successfully predicted the potential of multiple druggable genes as effective therapeutic targets for MVP through genetic methods, validated the potential of quercetin as a drug, and provided new ideas for drug treatment strategies for MVP.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38232702',\n",
       "   'title': 'Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia.',\n",
       "   'abstract': 'Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex\\xa0vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37352370',\n",
       "   'title': 'Genetic regulators of sputum mucin concentration and their associations with COPD phenotypes.',\n",
       "   'abstract': 'Hyper-secretion and/or hyper-concentration of mucus is a defining feature of multiple obstructive lung diseases, including chronic obstructive pulmonary disease (COPD). Mucus itself is composed of a mixture of water, ions, salt and proteins, of which the gel-forming mucins, MUC5AC and MUC5B, are the most abundant. Recent studies have linked the concentrations of these proteins in sputum to COPD phenotypes, including chronic bronchitis (CB) and acute exacerbations (AE). We sought to determine whether common genetic variants influence sputum mucin concentrations and whether these variants are also associated with COPD phenotypes, specifically CB and AE. We performed a GWAS to identify quantitative trait loci for sputum mucin protein concentration (pQTL) in the Sub-Populations and InteRmediate Outcome Measures in COPD Study (SPIROMICS, n = 708 for total mucin, n = 215 for MUC5AC, MUC5B). Subsequently, we tested for associations of mucin pQTL with CB and AE using regression modeling (n = 822-1300). Replication analysis was conducted using data from COPDGene (n = 5740) and by examining results from the UK Biobank. We identified one genome-wide significant pQTL for MUC5AC (rs75401036) and two for MUC5B (rs140324259, rs10001928). The strongest association for MUC5B, with rs140324259 on chromosome 11, explained 14% of variation in sputum MUC5B. Despite being associated with lower MUC5B, the C allele of rs140324259 conferred increased risk of CB (odds ratio (OR) = 1.42; 95% confidence interval (CI): 1.10-1.80) as well as AE ascertained over three years of follow up (OR = 1.41; 95% CI: 1.02-1.94). Associations between rs140324259 and CB or AE did not replicate in COPDGene. However, in the UK Biobank, rs140324259 was associated with phenotypes that define CB, namely chronic mucus production and cough, again with the C allele conferring increased risk. We conclude that sputum MUC5AC and MUC5B concentrations are associated with common genetic variants, and the top locus for MUC5B may influence COPD phenotypes, in particular CB.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38576019',\n",
       "   'title': 'Systematic proteome-wide Mendelian randomization using the human plasma proteome to identify therapeutic targets for lung adenocarcinoma.',\n",
       "   'abstract': 'Lung adenocarcinoma (LUAD) is the predominant histological subtype of lung cancer and the leading cause of cancer-related mortality. Identifying effective drug targets is crucial for advancing LUAD treatment strategies. This study employed proteome-wide Mendelian randomization (MR) and colocalization analyses. We collected data on 1394 plasma proteins from a protein quantitative trait loci (pQTL) study involving 4907 individuals. Genetic associations with LUAD were derived from the Transdisciplinary Research in Cancer of the Lung (TRICL) study, including 11,245 cases and 54,619 controls. We integrated pQTL and LUAD genome-wide association studies (GWASs) data to identify candidate proteins. MR utilizes single nucleotide polymorphisms (SNPs) as genetic instruments to estimate the causal effect of exposure on outcome, while Bayesian colocalization analysis determines the probability of shared causal genetic variants between traits. Our study applied these methods to assess causality between plasma proteins and LUAD. Furthermore, we employed a two-step MR to quantify the proportion of risk factors mediated by proteins on LUAD. Finally, protein-protein interaction (PPI) analysis elucidated potential links between proteins and current LUAD medications. We identified nine plasma proteins significantly associated with LUAD. Increased levels of ALAD, FLT1, ICAM5, and VWC2 exhibited protective effects, with odds ratios of 0.79 (95% CI 0.72-0.87), 0.39 (95% CI 0.28-0.55), 0.91 (95% CI 0.72-0.87), and 0.85 (95% CI 0.79-0.92), respectively. Conversely, MDGA2 (OR, 1.13; 95% CI 1.08-1.19), NTM (OR, 1.12; 95% CI 1.09-1.16), PMM2 (OR, 1.35; 95% CI 1.18-1.53), RNASET2 (OR, 1.15; 95% CI 1.08-1.21), and TFPI (OR, 4.58; 95% CI 3.02-6.94) increased LUAD risk. Notably, none of the nine proteins showed evidence of reverse causality. Bayesian colocalization indicated that RNASET2, TFPI, and VWC2 shared the same variant with LUAD. Furthermore, NTM and FLT1 demonstrated interactions with targets of current LUAD medications. Additionally, FLT1 and TFPI are currently under evaluation as therapeutic targets, while NTM, RNASET2, and VWC2 are potentially druggable. These findings shed light on LUAD pathogenesis, highlighting the tumor-promoting effects of RNASET2, TFPI, and NTM, along with the protective effects of VWC2 and FLT1, providing a significant biological foundation for future LUAD therapeutic targets. Our proteome-wide MR analysis highlighted RNASET2, TFPI, VWC2, NTM, and FLT1 as potential drug targets for further clinical investigation in LUAD. However, the specific mechanisms by which these proteins influence LUAD remain elusive. Targeting these proteins in drug development holds the potential for successful clinical trials, providing a pathway to prioritize and reduce costs in LUAD therapeutics.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40108295',\n",
       "   'title': 'Multi-omic biomarkers associated with multiple sclerosis: from Mendelian randomization to drug prediction.',\n",
       "   'abstract': 'Currently, the treatment and prevention of multiple sclerosis (MS) continue to encounter significant challenges. Mendelian randomization (MR) analysis has emerged as a crucial research method in the pursuit of new therapeutic strategies. Accordingly, we hypothesize that there exists a causal association between genetic variants of specific plasma proteins and MS through MR mechanisms, and that key therapeutic targets can be precisely identified by integrating multi-omics analytical approaches. In this study, we developed a comprehensive analytical framework aimed at identifying and validating potential therapeutic targets for MS. The framework commenced with a two-sample Mendelian randomization (MR) study utilizing two large plasma protein quantitative trait locus (pQTL) datasets. Building on this foundation, we performed Bayesian co-localization analysis of coding genes, followed by a full phenotype-wide association study (PheWAS) on the co-positive genes identified through both analytical methods. This approach allowed us to explore the functions of key genes and the mechanisms of co-morbidity associated with the disease. Subsequently, we integrated protein-protein interaction (PPI) network analysis, gene ontology (GO) analysis, and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis to facilitate drug prediction and molecular docking studies. This study conducted a systematic analysis between two large plasma pQTLs datasets and MS. In the MR analysis, the MR analysis of Icelandic plasma pQTLs and MS identified 88 positive plasma proteins, while the MR analysis of the UK Biobank database pQTLs and MS identified 122 positive plasma proteins. By comparison, uroporphyrinogen III synthase (UROS) and glutathione S-transferase theta 2B (GSTT2B) were found to be the positive proteins shared by the two datasets. After false discovery rate (FDR) correction, signal transducer and activator of transcription 3 (STAT3) was a significantly positive protein in the analysis of Icelandic plasma pQTLs. In the analysis of the UK Biobank database pQTLs, advanced glycosylation end product-specific receptor (AGER), allograft inflammatory factor 1 (AIF1), butyrophilin subfamily 1 member A1 (BTN1A1), cluster of differentiation 58 (CD58), desmoglein 4 (DSG4), ecotropic viral integration site 5 (EVI5), tumor necrosis factor (TNF), and tumor necrosis factor receptor superfamily member 14 (TNFRSF14) were significantly positive proteins. After Bonferroni correction, AGER, CD58, EVI5, and TNF remained significantly positive proteins in the analysis of the UK Biobank database pQTLs. In the Bayesian colocalization analysis, EVI5 (PPH4\\u2009=\\u20090.9800), O-GlcNAcase (OGA) (PPH4\\u2009=\\u20090.8569), and TNFRSF14 (PPH4\\u2009=\\u20090.8904) were the common positive genes in the two analysis methods. In conclusion, EVI5, OGA, and TNFRSF14 may be potential therapeutic targets for MS. Through the comprehensive application of MR analysis and Bayesian colocalization analysis, we have successfully identified that EVI5, OGA, and TNFRSF14 may be key therapeutic targets for MS. These findings may provide a scientific basis for the development of novel immunotherapies, combination treatment regimens, or targeted intervention strategies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38997278',\n",
       "   'title': 'CoPheScan: phenome-wide association studies accounting for linkage disequilibrium.',\n",
       "   'abstract': 'Phenome-wide association studies (PheWAS) facilitate the discovery of associations between a single genetic variant with multiple phenotypes. For variants which impact a specific protein, this can help identify additional therapeutic indications or on-target side effects of intervening on that protein. However, PheWAS is restricted by an inability to distinguish confounding due to linkage disequilibrium (LD) from true pleiotropy. Here we describe CoPheScan (Coloc adapted Phenome-wide Scan), a Bayesian approach that enables an intuitive and systematic exploration of causal associations while simultaneously addressing LD confounding. We demonstrate its performance through simulation, showing considerably better control of false positive rates than a conventional approach not accounting for LD. We used CoPheScan to perform PheWAS of protein-truncating variants and fine-mapped variants from disease and pQTL studies, in 2275 disease phenotypes from the UK Biobank. Our results identify the complexity of known pleiotropic genes such as APOE, and suggest a new causal role for TGM3 in skin cancer.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39247875',\n",
       "   'title': \"Genome-Wide Mendelian Randomization Identifies Ferroptosis-Related Drug Targets for Alzheimer's Disease.\",\n",
       "   'abstract': \"Alzheimer's disease (AD) currently lacks effective disease-modifying treatments. Recent research suggests that ferroptosis could be a potential therapeutic target. Mendelian randomization (MR) is a widely used method for identifying novel therapeutic targets. Employ genetic information to evaluate the causal impact of ferroptosis-related genes on the risk of AD. 564 ferroptosis-related genes were obtained from FerrDb. We derived genetic instrumental variables for these genes using four brain quantitative trait loci (QTL) and two blood QTL datasets. Summary-data-based Mendelian randomization (SMR) and two-sample MR methods were applied to estimate the causal effects of ferroptosis-related genes on AD. Using extern transcriptomic datasets and triple-transgenic mouse model of AD (3xTg-AD) to further validate the gene targets identified by the MR analysis. We identified 17 potential AD risk gene targets from GTEx, 13 from PsychENCODE, and 22 from BrainMeta (SMR  This study provides genetic evidence supporting the potential therapeutic benefits of targeting the six druggable genes for AD treatment, especially for \"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/28796488',\n",
       "   'title': 'Direct Interaction of Chivosazole F with Actin Elicits Cell Responses Similar to Latrunculin A but Distinct from Chondramide.',\n",
       "   'abstract': 'The microbial metabolite Chivosazole F has been described to affect the cytoskeleton and to inhibit actin polymerization in vitro. Applying orthogonal genomic and proteomics approaches, we now show for the first time that Chivosazole F exerts its effect by directly interacting with actin and demonstrate the cellular impact of Chivosazole F in an unbiased, genome-wide context in yeast and in mammalian cells. Furthermore, mutation-based resistance mapping identifies two SNPs located in the putative Chivosazole F binding site of actin. Comparing chemogenomic profiles and responses to the Chivosazole F-resistant SNPs shows a partially conserved mechanism of action for Chivosazole F and Latrunculin A, but clear divergence from Chondramide. In addition, C14orf80 is an evolutionarily highly conserved ORF, lacking any functional annotation. As editing of C14orf80 leads to Chivosazole F hyper-resistance, we propose a function for this gene product in counteracting perturbation of actin filaments.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38824092',\n",
       "   'title': 'Proteome-Wide Multicenter Mendelian Randomization Analysis to Identify Novel Therapeutic Targets for Lung Cancer.',\n",
       "   'abstract': 'Lung cancer (LC) remains a leading cause of cancer mortality worldwide, underscoring the urgent need for novel therapeutic targets. The integration of Mendelian randomization (MR) with proteomic data presents a novel approach to identifying potential targets for LC treatment. This study utilized a proteome-wide MR analysis, leveraging publicly available data from genome-wide association studies (GWAS) and protein quantitative trait loci (pQTL) studies. We analyzed genetic association data for LC from the TRICL-ILCCO Consortium and proteomic data from the Decode cohort. The MR framework was employed to estimate the causal effects of specific proteins on LC risk, supplemented by external validation, co-localization analyses, and exploration of protein-protein interaction (PPI) networks. Our analysis identified five proteins (TFPI, ICAM5, SFTPB, COL6A3, EPHB1) with significant associations to LC risk. External validation confirmed the potential therapeutic relevance of ICAM5 and SFTPB. Co-localization analyses and PPI network exploration provided further insights into the biological pathways involved and their potential mechanistic roles in LC pathogenesis. The study highlights the power of integrating genomic and proteomic data through MR analysis to uncover novel therapeutic targets for lung cancer. The identified proteins, particularly ICAM5 and SFTPB, offer promising directions for future research and development of targeted therapies, demonstrating the potential to advance personalized medicine in lung cancer treatment.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38527854',\n",
       "   'title': 'Connecting dementia risk loci to the CSF proteome identifies pathophysiological leads for dementia.',\n",
       "   'abstract': \"Genome-wide association studies have successfully identified many genetic risk loci for dementia, but exact biological mechanisms through which genetic risk factors contribute to dementia remains unclear. Integrating CSF proteomic data with dementia risk loci could reveal intermediate molecular pathways connecting genetic variance to the development of dementia. We tested to what extent effects of known dementia risk loci can be observed in CSF levels of 665 proteins [proximity extension-based (PEA) immunoassays] in a deeply-phenotyped mixed memory clinic cohort [n = 502, mean age (standard deviation, SD) = 64.1 (8.7) years, 181 female (35.4%)], including patients with Alzheimer's disease (AD, n = 213), dementia with Lewy bodies (DLB, n = 50) and frontotemporal dementia (FTD, n = 93), and controls (n = 146). Validation was assessed in independent cohorts (n = 99 PEA platform, n = 198, mass reaction monitoring-targeted mass spectroscopy and multiplex assay). We performed additional analyses stratified according to diagnostic status (AD, DLB, FTD and controls separately), to explore whether associations between CSF proteins and genetic variants were specific to disease or not. We identified four AD risk loci as protein quantitative trait loci (pQTL): CR1-CR2 (rs3818361, P = 1.65 × 10-8), ZCWPW1-PILRB (rs1476679, P = 2.73 × 10-32), CTSH-CTSH (rs3784539, P = 2.88 × 10-24) and HESX1-RETN (rs186108507, P = 8.39 × 10-8), of which the first three pQTLs showed direct replication in the independent cohorts. We identified one AD-specific association between a rare genetic variant of TREM2 and CSF IL6 levels (rs75932628, P = 3.90 × 10-7). DLB risk locus GBA showed positive trans effects on seven inter-related CSF levels in DLB patients only. No pQTLs were identified for FTD loci, either for the total sample as for analyses performed within FTD only. Protein QTL variants were involved in the immune system, highlighting the importance of this system in the pathophysiology of dementia. We further identified pQTLs in stratified analyses for AD and DLB, hinting at disease-specific pQTLs in dementia. Dissecting the contribution of risk loci to neurobiological processes aids in understanding disease mechanisms underlying dementia.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37076045',\n",
       "   'title': 'Proteomics: Progress and Promise of High-Throughput Proteomics in Chronic Kidney Disease.',\n",
       "   'abstract': 'Current proteomic tools permit the high-throughput analysis of the blood proteome in large cohorts, including those enriched for chronic kidney disease (CKD) or its risk factors. To date, these studies have identified numerous proteins associated with cross-sectional measures of kidney function, as well as with the longitudinal risk of CKD progression. Representative signals that have emerged from the literature include an association between levels of testican-2 and favorable kidney prognosis and an association between levels of TNFRSF1A and TNFRSF1B and worse kidney prognosis. For these and other associations, however, understanding whether the proteins play a causal role in kidney disease pathogenesis remains a fundamental challenge, especially given the strong impact that kidney function can have on blood protein levels. Prior to investing in dedicated animal models or randomized trials, methods that leverage the availability of genotyping in epidemiologic cohorts-including Mendelian randomization, colocalization analyses, and proteome-wide association studies-can add evidence for causal inference in CKD proteomics research. In addition, integration of large-scale blood proteome analyses with urine and tissue proteomics, as well as improved assessment of posttranslational protein modifications (e.g., carbamylation), represent important future directions. Taken together, these approaches seek to translate progress in large-scale proteomic profiling into the promise of improved diagnostic tools and therapeutic target identification in kidney disease.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39716299',\n",
       "   'title': 'Integrated Mendelian randomization and single-cell RNA-sequencing analyses identified OAS1 as a novel therapeutic target for erectile dysfunction via targeting fibroblasts.',\n",
       "   'abstract': 'Clinically, phosphodiesterase type 5 inhibitors (PDE5-Is) remain the first-line therapy for erectile dysfunction (ED) patients; however, approximately 35% of these patients are still failing to respond to the therapeutic effects. So, urgent needs are required to identify novel therapeutic targets for ED. Hence, in this report, it was the first time for us to integrate single-cell RNA-sequencing (scRNA-Seq), mendelian randomization (MR) analysis with expression quantitative trait loci (eQTL), and protein quantitative trait loci (pQTL) data to find new treatment targets for ED. Disease-causing changes were revealed by MR analysis, and it showed that the OAS1 eQTL/cis-eQTL/cis-pQTL was causally related to ED, significantly reducing its risks (all P\\u2009<\\u20090.05). Disease-induced changes were revealed by scRNA-Seq, and it suggested that OAS1 mainly played its role in ED via targeting fibroblasts. We further concluded that the positive regulation of OAS1 gene expression could lead to the vicious circle of ED. As a result, drugs targeting OAS1 in the future might provide more potential opportunities and flexibility for treating ED. In conclusion, our study identified OAS1 as a gene of interest in the context of ED via targeting fibroblasts through integrated MR and scRNA-Seq analyses. While these findings highlighted the potential of OAS1 as a therapeutic target, further experimental and clinical studies were still required to validate its functional role and therapeutic relevance in ED pathology.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37550624',\n",
       "   'title': 'Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases.',\n",
       "   'abstract': 'Human plasma contains a wide variety of circulating proteins. These proteins can be important clinical biomarkers in disease and also possible drug targets. Large scale genomics studies of circulating proteins can identify genetic variants that lead to relative protein abundance. We conducted a meta-analysis on genome-wide association studies of autosomal chromosomes in 22,997 individuals of primarily European ancestry across 12 cohorts to identify protein quantitative trait loci (pQTL) for 92 cardiometabolic associated plasma proteins. We identified 503 (337 cis and 166 trans) conditionally independent pQTLs, including several novel variants not reported in the literature. We conducted a sex-stratified analysis and found that 118 (23.5%) of pQTLs demonstrated heterogeneity between sexes. The direction of effect was preserved but there were differences in effect size and significance. Additionally, we annotate trans-pQTLs with nearest genes and report plausible biological relationships. Using Mendelian randomization, we identified causal associations for 18 proteins across 19 phenotypes, of which 10 have additional genetic colocalization evidence. We highlight proteins associated with a constellation of cardiometabolic traits including angiopoietin-related protein 7 (ANGPTL7) and Semaphorin 3F (SEMA3F). Through large-scale analysis of protein quantitative trait loci, we provide a comprehensive overview of common variants associated with plasma proteins. We highlight possible biological relationships which may serve as a basis for further investigation into possible causal roles in cardiometabolic diseases.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35482474',\n",
       "   'title': 'Biomarkers of disease progression in people with psoriasis: a scoping review.',\n",
       "   'abstract': 'Identification of those at risk of more severe psoriasis and/or associated morbidities offers opportunity for early intervention, reduced disease burden and more cost-effective healthcare. Prognostic biomarkers of disease progression have thus been the focus of intense research, but none are part of routine practice. To identify and catalogue candidate biomarkers of disease progression in psoriasis for the translational research community. A systematic search of CENTRAL, Embase, LILACS and MEDLINE was performed for relevant articles published between 1990 and December 2021. Eligibility criteria were studies involving patients with psoriasis (any age, n\\u2009≥\\u200950) reporting biomarkers associated with disease progression. The main outcomes were any measure of skin severity or any prespecified psoriasis comorbidity. Data were extracted by one reviewer and checked by a second; studies meeting minimal quality criteria (longitudinal design and/or use of methods to control for confounding) were formally assessed for bias. Candidate biomarkers were identified by an expert multistakeholder group using a majority voting consensus exercise, and mapped to relevant cellular and molecular pathways. Of 181 included studies, most investigated genomic or proteomic biomarkers associated with disease severity (n\\u2009=\\u2009145) or psoriatic arthritis (n\\u2009=\\u200930). Methodological and reporting limitations compromised interpretation of findings, most notably a lack of longitudinal studies, and inadequate control for key prognostic factors. The following candidate biomarkers with future potential utility were identified for predicting disease severity: LCE3D, interleukin (IL)23R, IL23A, NFKBIL1 loci, HLA-C*06:02 (genomic), IL-17A, IgG aHDL, GlycA, I-FABP and kallikrein 8 (proteomic), tyramine (metabolomic); psoriatic arthritis: HLA-C*06:02, HLA-B*27, HLA-B*38, HLA-B*08, and variation at the IL23R and IL13 loci (genomic); IL-17A, CXCL10, Mac-2 binding protein, integrin b5, matrix metalloproteinase-3 and macrophage-colony stimulating factor (proteomic) and tyramine and mucic acid (metabolomic); and type 2 diabetes mellitus: variation in IL12B and IL23R loci (genomic). No biomarkers were supported by sufficient evidence for clinical use without further validation. This review provides a comprehensive catalogue of investigated biomarkers of disease progression in psoriasis. Future studies must address the common methodological limitations identified herein to expedite discovery and validation of biomarkers for clinical use. What is already known about this topic? The current treatment paradigm in psoriasis is reactive. There is a need to develop effective risk-stratified management approaches that can proactively attenuate the substantial burden of disease. Prognostic biomarkers of disease progression have therefore been the focus of intense research. What does this study add? This review is the first to scope, collate and catalogue research investigating biomarkers of disease progression in psoriasis. The review identifies potentially promising candidate biomarkers for further investigation and highlights common important limitations that should be considered when designing and conducting future studies in this area.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40211130',\n",
       "   'title': 'Investigating potential drug targets for the treatment of glioblastoma: a Mendelian randomization study.',\n",
       "   'abstract': 'Glioblastoma (GBM), one of the most aggressive brain tumors, has a 5-year survival rate of less than 5%. Current standard therapies, including surgery, radiotherapy, and temozolomide (TMZ) chemotherapy, are limited by drug resistance and the blood-brain barrier. Integrating expression quantitative trait loci (eQTL) and protein quantitative trait loci (pQTL) data has shown promise in uncovering disease mechanisms and therapeutic targets. This study combined eQTL and pQTL analyses to identify potential GBM-related genes and circulating plasma proteins for therapeutic exploration. Using transcriptomic data from The Cancer Genome Atlas (TCGA), we identified 2,528 differentially expressed genes, including GPX7 and CXCL10. eQTL-MR analysis identifies GBM-associated differentially expressed genes and constructs a protein-protein interaction (PPI) network.Integrating pQTL data from the deCODE database, pQTL-MR, and colocalization analyses validated the therapeutic potential of GPX7 and CXCL10.These findings provide new perspectives on GBM biology and suggest actionable targets for therapy. Despite limitations due to sample size and population-specific data, this study highlights GPX7 and CXCL10 as promising candidates for further investigation and lays the foundation for targeted GBM treatments.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/26883866',\n",
       "   'title': 'Blood group antigen loci demonstrate multivariate genetic associations with circulating cellular adhesion protein levels in the Multi-Ethnic Study of Atherosclerosis.',\n",
       "   'abstract': 'The cellular adhesion pathway is critical in the pathophysiology of atherosclerosis, and genetic factors contributing to regulation of circulating levels of related proteins may be relevant to risk prediction of cardiovascular disease. In contrast to conducting separate genome-wide protein quantitative trait loci (pQTL) mapping analyses of each individual protein, joint genetic association analyses of multiple quantitative traits can leverage cross-trait co-variation and identify simultaneous regulatory effects on protein levels across the pathway. We conducted a multi-pQTL (mpQTL) analysis of 15 proteins related to cellular adhesion assayed on 2313 participants from the Multi-Ethnic Study of Atherosclerosis (MESA). We applied the MQFAM multivariate association analysis method in PLINK on normalized protein level residuals derived from univariate linear regression, adjusting for age, sex, and principal components of ancestry. Race/ethnicity-stratified analyses identified nine genome-wide significant (P < 5e-08) loci associated with co-variation of protein levels. Although the majority of these SNPs were in proximity to structural genes of the assayed proteins, we discovered multiple loci demonstrating co-association with the circulation of at least two proteins. Of these, two significant loci specific to non-Hispanic white participants, rs17074898 at ALOX5AP (P = 1.78E-08) and rs7521237 at KIAA1614 (P = 2.2E-08), would not have met statistical significance using univariate analyses. Moreover, common patterns of multi-protein associations were discovered at the ABO locus across race/ethnicity. These results indicate the biological relevance of blood group antigens on regulation of circulating cellular adhesion pathway proteins while also demonstrating race/ethnicity-specific co-regulatory effects.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/34376796',\n",
       "   'title': 'Analyses of biomarker traits in diverse UK biobank participants identify associations missed by European-centric analysis strategies.',\n",
       "   'abstract': 'Despite the dramatic underrepresentation of non-European populations in human genetics studies, researchers continue to exclude participants of non-European ancestry, as well as variants rare in European populations, even when these data are available. This practice perpetuates existing research disparities and can lead to important and large effect size associations being missed. Here, we conducted genome-wide association studies (GWAS) of 31 serum and urine biomarker quantitative traits in African (n\\u2009=\\u20099354), East Asian (n\\u2009=\\u20092559), and South Asian (n\\u2009=\\u20099823) ancestry UK Biobank (UKBB) participants. We adjusted for all known GWAS catalog variants for each trait, as well as novel signals identified in a recent European ancestry-focused analysis of UKBB participants. We identify 7 novel signals in African ancestry and 2 novel signals in South Asian ancestry participants (p\\u2009<\\u20091.61E-10). Many of these signals are highly plausible, including a cis pQTL for the gene encoding gamma-glutamyl transferase and PIEZO1 and G6PD variants with impacts on HbA1c through likely erythrocytic mechanisms. This work illustrates the importance of using the genetic data we already have in diverse populations, with novel discoveries possible in even modest sample sizes.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39500920',\n",
       "   'title': 'Network medicine informed multiomics integration identifies drug targets and repurposable medicines for Amyotrophic Lateral Sclerosis.',\n",
       "   'abstract': 'Amyotrophic Lateral Sclerosis (ALS) is a devastating, immensely complex neurodegenerative disease by lack of effective treatments. We developed a network medicine methodology via integrating human brain multi-omics data to prioritize drug targets and repurposable treatments for ALS. We leveraged non-coding ALS loci effects from genome-wide associated studies (GWAS) on human brain expression quantitative trait loci (QTL) (eQTL), protein QTL (pQTL), splicing QTL (sQTL), methylation QTL (meQTL), and histone acetylation QTL (haQTL). Using a network-based deep learning framework, we identified 105 putative ALS-associated genes enriched in known ALS pathobiological pathways. Applying network proximity analysis of predicted ALS-associated genes and drug-target networks under the human protein-protein interactome (PPI) model, we identified potential repurposable drugs (i.e., Diazoxide and Gefitinib) for ALS. Subsequent validation established preclinical evidence for top-prioritized drugs. In summary, we presented a network-based multi-omics framework to identify drug targets and repurposable treatments for ALS and other neurodegenerative disease if broadly applied.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37683633',\n",
       "   'title': 'Integration of genetic fine-mapping and multi-omics data reveals candidate effector genes for hypertension.',\n",
       "   'abstract': 'Genome-wide association studies of blood pressure (BP) have identified >1,000 loci, but the effector genes and biological pathways at these loci are mostly unknown. Using published association summary statistics, we conducted annotation-informed fine-mapping incorporating tissue-specific chromatin segmentation and colocalization to identify causal variants and candidate effector genes for systolic BP, diastolic BP, and pulse pressure. We observed 532 distinct signals associated with ≥2 BP traits and 84 with all three. For >20% of signals, a single variant accounted for >75% posterior probability, 65 were missense variants in known (SLC39A8, ADRB2, and DBH) and previously unreported BP candidate genes (NRIP1 and MMP14). In disease-relevant tissues, we colocalized >80 and >400 distinct signals for each BP trait with cis-eQTLs and regulatory regions from promoter capture Hi-C, respectively. Integrating mouse, human disorder, gene expression and tissue abundance data, and literature review, we provide consolidated evidence for 436 BP candidate genes for future functional validation and discover several potential drug targets.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/28791018',\n",
       "   'title': 'Genetic and Functional Associations with Decreased Anti-inflammatory Tumor Necrosis Factor Alpha Induced Protein 3 in Macrophages from Subjects with Axial Spondyloarthritis.',\n",
       "   'abstract': 'Tumor necrosis factor alpha-induced protein 3 (TNFAIP3) is an anti-inflammatory protein implicated in multiple autoimmune and rheumatologic conditions. We hypothesized that lower levels of TNFAIP3 contributes to excessive cytokine production in response to inflammatory stimuli in axial spondyloarthritis (AxSpA). A further aim was to determine the immune signaling and genetic variation regulating TNFAIP3 expression in individual subjects. Blood-derived macrophages from 50 AxSpA subjects and 30 healthy controls were assessed for TNFAIP3 expression. Cell lysates were also analyzed for NF-κB, mitogen-activated protein (MAP) kinase and STAT3 phosphorylation, and supernatants for cytokine production. Coding and regulatory regions in the  Mean TNFAIP3 was significantly lower in spondyloarthritis macrophages than controls ( Our results suggest that both immune functional and genetic variations contribute to the regulation of TNFAIP3 levels in individual subjects. Decreased expression of TNFAIP3 in AxSpA macrophages correlated with increased LPS-induced TNF-α, and thus, TNFAIP3 dysregulation may be a contributor to excessive inflammatory responses in spondyloarthritis subjects.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/34208838',\n",
       "   'title': \"Analysis of Genetic Variants Associated with Levels of Immune Modulating Proteins for Impact on Alzheimer's Disease Risk Reveal a Potential Role for SIGLEC14.\",\n",
       "   'abstract': \"Genome-wide association studies (GWAS) have identified immune-related genes as risk factors for Alzheimer's disease (AD), including \"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36780661',\n",
       "   'title': 'Integrating Genetics, Transcriptomics, and Proteomics in Lung Tissue to Investigate Chronic Obstructive Pulmonary Disease.',\n",
       "   'abstract': 'The integration of transcriptomic and proteomic data from lung tissue with chronic obstructive pulmonary disease (COPD)-associated genetic variants could provide insight into the biological mechanisms of COPD. Here, we assessed associations between lung transcriptomics and proteomics with COPD in 98 subjects from the Lung Tissue Research Consortium. Low correlations between transcriptomics and proteomics were generally observed, but higher correlations were found for COPD-associated proteins. We integrated COPD risk SNPs or SNPs near COPD-associated proteins with lung transcripts and proteins to identify regulatory '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37715107',\n",
       "   'title': 'Decreased CNNM2 expression in prefrontal cortex affects sensorimotor gating function, cognition, dendritic spine morphogenesis and risk of schizophrenia.',\n",
       "   'abstract': 'Genome-wide association studies (GWASs) have reported multiple single nucleotide polymorphisms (SNPs) associated with schizophrenia, yet the underlying molecular mechanisms are largely unknown. In this study, we aimed to identify schizophrenia relevant genes showing alterations in mRNA and protein expression associated with risk SNPs at the 10q24.32-33 GWAS locus. We carried out the quantitative trait loci (QTL) and summary data-based Mendelian randomization (SMR) analyses, using the PsychENCODE dorsolateral prefrontal cortex (DLPFC) expression QTL (eQTL) database, as well as the ROSMAP and Banner DLPFC protein QTL (pQTL) datasets. The gene CNNM2 (encoding a magnesium transporter) at 10q24.32-33 was identified to be a robust schizophrenia risk gene, and was highly expressed in human neurons according to single cell RNA-seq (scRNA-seq) data. We further revealed that reduced Cnnm2 in the mPFC of mice led to impaired cognition and compromised sensorimotor gating function, and decreased Cnnm2 in primary cortical neurons altered dendritic spine morphogenesis, confirming the link between CNNM2 and endophenotypes of schizophrenia. Proteomics analyses showed that reduced Cnnm2 level changed expression of proteins associated with neuronal structure and function. Together, these results identify a robust gene in the pathogenesis of schizophrenia.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35987429',\n",
       "   'title': 'Codon Usage and mRNA Stability are Translational Determinants of Cellular Response to Canonical Ferroptosis Inducers.',\n",
       "   'abstract': 'Ferroptosis is a non-apoptotic cell death mechanism characterized by the generation of lipid peroxides. While many effectors in the ferroptosis pathway have been mapped, its epitranscriptional regulation is not yet fully understood. Ferroptosis can be induced via system xCT inhibition (Class I) or GPX4 inhibition (Class II). Previous works have revealed important differences in cellular response to different ferroptosis inducers. Importantly, blocking mRNA transcription or translation appears to protect cells against Class I ferroptosis inducing agents but not Class II. In this work, we examined the impact of blocking transcription (via Actinomycin D) or translation (via Cycloheximide) on Erastin (Class I) or RSL3 (Class II) induced ferroptosis. Blocking transcription or translation protected cells against Erastin but was detrimental against RSL3. Cycloheximide led to increased levels of GSH alone or when co-treated with Erastin via the activation of the reverse transsulfuration pathway. RNA sequencing analysis revealed early activation of a strong alternative splice program before observed changes in transcription. mRNA stability analysis revealed divergent mRNA stability changes in cellular response to Erastin or RSL3. Importantly, codon optimality biases were drastically different in either condition. Our data also implicated translation repression and rate as an important determinant of the cellular response to ferroptosis inducers. Given that mRNA stability and codon usage can be influenced via the tRNA epitranscriptome, we evaluated the role of a tRNA modifying enzyme in ferroptosis stress response. Alkbh1, a tRNA demethylase, led to translation repression and increased the resistance to Erastin but made cells more sensitive to RSL3.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37286633',\n",
       "   'title': 'Large scale proteomic studies create novel privacy considerations.',\n",
       "   'abstract': 'Privacy protection is a core principle of genomic but not proteomic research. We identified independent single nucleotide polymorphism (SNP) quantitative trait loci (pQTL) from COPDGene and Jackson Heart Study (JHS), calculated continuous protein level genotype probabilities, and then applied a naïve Bayesian approach to link SomaScan 1.3K proteomes to genomes for 2812 independent subjects from COPDGene, JHS, SubPopulations and InteRmediate Outcome Measures In COPD Study (SPIROMICS) and Multi-Ethnic Study of Atherosclerosis (MESA). We correctly linked 90-95% of proteomes to their correct genome and for 95-99% we identify the 1% most likely links. The linking accuracy in subjects with African ancestry was lower (~\\u200960%) unless training included diverse subjects. With larger profiling (SomaScan 5K) in the Atherosclerosis Risk Communities (ARIC) correct identification was\\u2009>\\u200999% even in mixed ancestry populations. We also linked proteomes-to-proteomes and used the proteome only to determine features such as sex, ancestry, and first-degree relatives. When serial proteomes are available, the linking algorithm can be used to identify and correct mislabeled samples. This work also demonstrates the importance of including diverse populations in omics research and that large proteomic datasets (>\\u20091000 proteins) can be accurately linked to a specific genome through pQTL knowledge and should not be considered unidentifiable.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/34857953',\n",
       "   'title': 'Large-scale integration of the plasma proteome with genetics and disease.',\n",
       "   'abstract': 'The plasma proteome can help bridge the gap between the genome and diseases. Here we describe genome-wide association studies (GWASs) of plasma protein levels measured with 4,907 aptamers in 35,559 Icelanders. We found 18,084 associations between sequence variants and levels of proteins in plasma (protein quantitative trait loci; pQTL), of which 19% were with rare variants (minor allele frequency (MAF)\\u2009<\\u20091%). We tested plasma protein levels for association with 373 diseases and other traits and identified 257,490 associations. We integrated pQTL and genetic associations with diseases and other traits and found that 12% of 45,334 lead associations in the GWAS Catalog are with variants in high linkage disequilibrium with pQTL. We identified 938 genes encoding potential drug targets with variants that influence levels of possible biomarkers. Combining proteomics, genomics and transcriptomics, we provide a valuable resource that can be used to improve understanding of disease pathogenesis and to assist with drug discovery and development.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32108428',\n",
       "   'title': 'Advanced Cell Mapping Visualizations for Single Cell Functional Proteomics Enabling Patient Stratification.',\n",
       "   'abstract': 'Highly multiplexed single-cell functional proteomics has emerged as one of the next-generation toolkits for a deeper understanding of functional heterogeneity in cell. Different from the conventional population-based bulk and single-cell RNA-Seq assays, the microchip-based proteomics at the single-cell resolution enables a unique identification of highly polyfunctional cell subsets that co-secrete many proteins from live single cells and most importantly correlate with patient response to a therapy. The 32-plex IsoCode chip technology has defined a polyfunctional strength index (PSI) of pre-infusion anti-CD19 chimeric antigen receptor (CAR)-T products, that is significantly associated with patient response to the CAR-T cell therapy. To complement the clinical relevance of the PSI, a comprehensive visualization toolkit of 3D uniform manifold approximation and projection (UMAP) and t-distributed stochastic neighbor embedding (t-SNE) in a proteomic analysis pipeline is developed, providing more advanced analytical algorithms for more intuitive data visualizations. The UMAP and t-SNE of anti-CD19 CAR-T products reveal distinct cytokine profiles between nonresponders and responders and demonstrate a marked upregulation of antitumor-associated cytokine signatures in CAR-T cells from responding patients. Using this powerful while user-friendly analytical tool, the multi-dimensional single-cell data can be dissected from complex immune responses and uncover underlying mechanisms, which can promote correlative biomarker discovery, improved bioprocessing, and personalized treatment development.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39844246',\n",
       "   'title': 'Multi-omics analysis reveals novel causal pathways in psoriasis pathogenesis.',\n",
       "   'abstract': 'To elucidate the genetic and molecular mechanisms underlying psoriasis by employing an integrative multi-omics approach, using summary-data-based Mendelian randomization (SMR) to infer causal relationships among DNA methylation, gene expression, and protein levels in relation to psoriasis risk. We conducted SMR analyses integrating genome-wide association study (GWAS) summary statistics with methylation quantitative trait loci (mQTL), expression quantitative trait loci (eQTL), and protein quantitative trait loci (pQTL) data. Publicly available datasets were utilized, including psoriasis GWAS data from the European Molecular Biology Laboratory-European Bioinformatics Institute and the UK Biobank. Heterogeneity in dependent instruments (HEIDI) test and colocalization analyses were performed to identify shared causal variants, and multi-omics integration was employed to construct potential regulatory pathways. Our analyses identified significant causal associations between DNA methylation, gene expression, protein abundance, and psoriasis risk. We discovered two pathways involving the long non-coding RNA RP11-977G19.11 and apolipoprotein F (APOF). Methylation at sites cg26804944 and cg02705573 was negatively associated with RP11-977G19.11 expression. Reduced expression of RP11-977G19.11 was linked to increased APOF levels, which were positively associated with a higher risk of psoriasis. Methylation at sites cg00172967, cg00294382, and cg24773560 was positively associated with RP11-977G19.11 expression. Elevated expression of RP11-977G19.11 was associated with decreased APOF levels, reducing the risk of psoriasis. Colocalization analysis highlighted APOF as a key protein in psoriasis pathogenesis. Validation using skin tissue, EBV-transformed lymphocytes data and inflammation-related protein panels confirmed the associations of RP11-977G19.11 and APOF with psoriasis. Our multi-omics analysis provides preliminary evidence for potential molecular mechanisms in psoriasis pathogenesis. Through the integration of GWAS and molecular QTL data, we identify candidate pathways that may be relevant to disease biology. While these findings require extensive experimental validation, they offer a framework for future investigations into the molecular basis of psoriasis.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/40263317',\n",
       "   'title': 'Genetics of circulating proteins in newborn babies at high risk of type 1 diabetes.',\n",
       "   'abstract': 'Type 1 diabetes is a chronic, autoimmune disease characterized by the destruction of insulin-producing β-cells in the pancreas. Early detection can facilitate timely intervention, potentially delaying or preventing disease onset. Circulating proteins reflect dysregulated biological processes and offer insights into early disease mechanisms. Here, we construct a genome-wide pQTL map of 1985 proteins in 695 newborn babies (median age 2\\u2009days) at increased genetic risk of developing Type 1 diabetes. We identify 535 pQTLs (352 cis-pQTLs, 183 trans-pQTLs), 62 of which characteristic of newborns. We show colocalization of pQTLs for CTRB1, APOBR, IL7R, CPA1, and PNLIPRP1 with Type 1 diabetes GWAS signals, and Mendelian randomization causally implicates each of these five proteins in the aetiology of Type 1 diabetes. Our study illustrates the utility of newborn molecular profiles for discovering potential drug targets for childhood diseases of significant concern.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39793580',\n",
       "   'title': 'Multivariate proteome-wide association study to identify causal proteins for Alzheimer disease.',\n",
       "   'abstract': \"Alzheimer disease (AD) is a complex and progressive neurodegenerative disorder that accounts for the majority of individuals with dementia. Here, we aim to identify causal plasma proteins for AD, shedding light on the etiology of AD. We utilized the latest large-scale plasma proteomic data from the UK Biobank Pharma Proteomics Project (UKB-PPP) and AD genome-wide association study (GWAS) summary data from the International Genomics of Alzheimer's Project (IGAP). Via a robust univariate instrumental variable (IV) regression method, we identified causal proteins through cis-protein quantitative trait loci (pQTLs) and (both cis- and trans-)pQTLs. To further reduce potential false positives due to high linkage disequilibrium (LD) of some pQTLs and high correlations among some proteins, we developed a robust multivariate IV regression method, called two-stage constrained maximum likelihood (MV-2ScML), to distinguish direct and confounding/mediating effects of proteins; some key features of the method include its robustness to invalid IVs and applicability to GWAS summary data. Our work highlights some differences between using cis-pQTLs and trans-pQTLs and critical values of multivariate analysis for fine-mapping causal proteins, providing insights into plasma protein pathways to AD.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/27504496',\n",
       "   'title': 'Trans-pQTL study identifies immune crosstalk between Parkinson and Alzheimer loci.',\n",
       "   'abstract': 'Given evidence from genetic studies, we hypothesized that there may be a shared component to the role of myeloid function in Parkinson and Alzheimer disease (PD and AD) and assessed whether PD susceptibility variants influenced protein expression of well-established AD-associated myeloid genes in human monocytes. We repurposed data in which AD-related myeloid proteins CD33, TREM1, TREM2, TREML2, TYROBP, and PTK2B were measured by flow cytometry in monocytes from 176 participants of the PhenoGenetic Project (PGP) and Harvard Aging Brain Study. Linear regression was used to identify associations between 24 PD risk variants and protein expression. The 2 cohorts were meta-analyzed in a discovery analysis, and the 4 most strongly suggestive results were validated in an independent cohort of 50 PGP participants. We discovered and validated an association between the PD risk allele rs12456492(G) in the RIT2 locus and increased CD33 expression (p joint = 3.50 × 10(-5)) and found strongly suggestive evidence that rs11060180(A) in the CCDC62/HIP1R locus decreased PTK2B expression (p joint = 1.12 × 10(-4)). Furthermore, in older individuals, increased CD33 expression on peripheral monocytes was associated with a greater burden of parkinsonism (p = 0.047), particularly bradykinesia (p = 6.64 × 10(-3)). We find that the rs12456492 PD risk variant affects expression of AD-associated protein CD33 in peripheral monocytes, which suggests that genetic factors for these 2 diseases may converge to influence overlapping innate immune-mediated mechanisms that contribute to neurodegeneration. Furthermore, the effect of the rs12456492(G) PD risk allele on increased CD33 suggests that the inhibition of certain myeloid functions may contribute to PD susceptibility, as is the case for AD.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/33851187',\n",
       "   'title': 'Mendelian randomisation identifies alternative splicing of the FAS death receptor as a mediator of severe COVID-19.',\n",
       "   'abstract': 'Severe COVID-19 is characterised by immunopathology and epithelial injury. Proteomic studies have identified circulating proteins that are biomarkers of severe COVID-19, but cannot distinguish correlation from causation. To address this, we performed Mendelian randomisation (MR) to identify proteins that mediate severe COVID-19. Using protein quantitative trait loci (pQTL) data from the SCALLOP consortium, involving meta-analysis of up to 26,494 individuals, and COVID-19 genome-wide association data from the Host Genetics Initiative, we performed MR for 157 COVID-19 severity protein biomarkers. We identified significant MR results for five proteins: FAS, TNFRSF10A, CCL2, EPHB4 and LGALS9. Further evaluation of these candidates using sensitivity analyses and colocalization testing provided strong evidence to implicate the apoptosis-associated cytokine receptor FAS as a causal mediator of severe COVID-19. This effect was specific to severe disease. Using RNA-seq data from 4,778 individuals, we demonstrate that the pQTL at the '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38940271',\n",
       "   'title': \"Proteome-wide association study using cis and trans variants and applied to blood cell and lipid-related traits in the Women's Health Initiative study.\",\n",
       "   'abstract': \"In most Proteome-Wide Association Studies (PWAS), variants near the protein-coding gene (±1\\u2009Mb), also known as cis single nucleotide polymorphisms\\xa0(SNPs), are used to predict protein levels, which are then tested for association with phenotypes. However, proteins can be regulated through variants outside of the cis region. An intermediate GWAS step to identify protein quantitative trait loci (pQTL) allows for the inclusion of trans SNPs outside the cis region in protein-level prediction models. Here, we assess the prediction of 540 proteins in 1002 individuals from the Women's Health Initiative (WHI), split equally into a GWAS set, an elastic net training set, and a testing set. We compared the testing r\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37410486',\n",
       "   'title': 'Association of Risk Variants in the CFH Gene With Elevated Levels of Coagulation and Complement Factors in Idiopathic Multifocal Choroiditis.',\n",
       "   'abstract': 'Idiopathic multifocal choroiditis (MFC) is poorly understood, thereby hindering optimal treatment and monitoring of patients. To identify the genes and pathways associated with idiopathic MFC. This was a case-control genome-wide association study (GWAS) and protein study of blood plasma samples conducted from March 2006 to February 2022. This was a multicenter study involving 6 Dutch universities. Participants were grouped into 2 cohorts: cohort 1 consisted of Dutch patients with idiopathic MFC and controls, and cohort 2 consisted of patients with MFC and controls. Plasma samples from patients with idiopathic MFC who had not received treatment were subjected to targeted proteomics. Idiopathic MFC was diagnosed according to the Standardization of Uveitis Nomenclature (SUN) Working Group guidelines for punctate inner choroidopathy and multifocal choroiditis with panuveitis. Data were analyzed from July 2021 to October 2022. Genetic variants associated with idiopathic MFC and risk variants associated with plasma protein concentrations in patients. This study included a total of 4437 participants in cohort 1 (170 [3.8%] Dutch patients with idiopathic MFC and 4267 [96.2%] controls; mean [SD] age, 55 [18] years; 2443 female [55%]) and 1344 participants in cohort 2 (52 [3.9%] patients with MFC and 1292 [96.1%] controls; 737 male [55%]). The primary GWAS association mapped to the CFH gene with genome-wide significance (lead variant the A allele of rs7535263; odds ratio [OR], 0.52; 95% CI, 0.41-0.64; P\\u2009=\\u20099.3\\u2009×\\u200910-9). There was no genome-wide significant association with classical human leukocyte antigen (HLA) alleles (lead classical allele, HLA-A*31:01; P\\u2009=\\u2009.002). The association with rs7535263 showed consistent direction of effect in an independent cohort of 52 cases and 1292 control samples (combined meta-analysis OR,\\u20090.58; 95% CI, 0.38-0.77; P\\u2009=\\u20093.0\\u2009×\\u200910-8). In proteomic analysis of 87 patients, the risk allele G of rs7535263 in the CFH gene was strongly associated with increased plasma concentrations of factor H-related (FHR) proteins (eg, FHR-2, likelihood ratio test, adjusted P\\u2009=\\u20091.1\\u2009×\\u200910-3) and proteins involved in platelet activation and the complement cascade. Results suggest that CFH gene variants increase systemic concentrations of key factors of the complement and coagulation cascades, thereby conferring susceptibility to idiopathic MFC. These findings suggest that the complement and coagulation pathways may be key targets for the treatment of idiopathic MFC.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39112975',\n",
       "   'title': 'Circulating inflammatory cytokines and the risk of sepsis: a bidirectional mendelian randomization analysis.',\n",
       "   'abstract': 'Sepsis is a life-threatening condition that is characterized by multiorgan dysfunction and caused by dysregulated cytokine networks, which are closely associated with sepsis progression and outcomes. However, currently available treatment strategies that target cytokines have failed. Thus, this study aimed to investigate the interplay between genetically predicted circulating concentrations of cytokines and the outcomes of sepsis and to identify potential targets for sepsis treatment. Data related to 35 circulating cytokines in 31,112 individuals (including 11,643 patients with sepsis) were included in genome-wide association studies (GWASs) from the UK Biobank and FinnGen consortia. A bidirectional two-sample Mendelian randomization (MR) analysis was performed using single nucleotide polymorphisms (SNPs) to evaluate the causal effects of circulating cytokines on sepsis outcomes and other cytokines. A total of 35 inflammatory cytokine genes were identified in the GWASs, and 11 cytokines, including Interleukin-1 receptor antagonist (IL-1ra), macrophage inflammatory protein 1 (MIP1α), IL-16, et al., were associated with sepsis outcome pairs according to the selection criteria of the cis-pQTL instrument. Multiple MR methods verified that genetically predicted high circulating levels of IL-1ra or MIP1α were negatively correlated with genetic susceptibility to risk of sepsis, including sepsis (28-day mortality), septicaemia, streptococcal and pneumonia-derived septicaemia (P\\u2009≤\\u20090.01). Furthermore, genetic susceptibility of sepsis outcomes except sepsis (28-day mortality) markedly associated with the circulating levels of five cytokines, including active plasminogen activator inhibitor (PAI), interleukin 7 (IL-7), tumour necrosis factor alpha (TNF-α), beta nerve growth factor (NGF-β), hepatic growth factor (HGF) (P\\u2009<\\u20090.05). Finally, we observed that the causal interaction network between MIP1α\\xa0or IL-1ra and other cytokines (P\\u2009<\\u20090.05). This comprehensive MR analysis provides insights into the potential causal mechanisms that link key cytokines, particularly MIP1α, with risk of sepsis, and the findings suggest that targeting MIP1α may be a potential strategy for preventing sepsis.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39715222',\n",
       "   'title': 'Identifying potential drug targets for myocardial infarction through Mendelian randomization.',\n",
       "   'abstract': 'This study explored the associations between plasma and cerebrospinal fluid (CSF) proteins and myocardial infarction (MI) risk. Identifying specific proteins as biomarkers for MI could enhance our understanding of disease mechanisms and inform clinical practice. We combined protein quantitative trait loci (pQTL) data for plasma and CSF proteins with genome-wide association study (GWAS) summary statistics for MI. Mendelian Randomization (MR) analyses were conducted to establish causal relationships, supported by Bayesian colocalization and Spearman correlation analyses. For plasma proteins, we used pQTL data from Cheng et al. to select 738 cis-acting SNPs associated with 734 proteins. The \"TwoSampleMR\" method and inverse-variance weighted MR were applied for evaluations. In plasma, CD8A and HDHD2 were identified as protective factors against MI, while DPEP1 was linked to increased risk. In CSF, CD30 Ligand was associated with MI risk. Bayesian colocalization supported the association for CD8A in plasma. No significant correlation was found between plasma and CSF results, suggesting distinct mechanisms for these biomarkers. Our study identified several plasma and CSF proteins linked to MI risk, offering new insights into the disease\\'s biological underpinnings. These findings could guide future research on MI biomarkers and contribute to improved prevention and treatment strategies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30562114',\n",
       "   'title': 'Identification of Common and Rare Genetic Variation Associated With Plasma Protein Levels Using Whole-Exome Sequencing and Mass Spectrometry.',\n",
       "   'abstract': 'Identifying genetic variation associated with plasma protein levels, and the mechanisms by which they act, could provide insight into alterable processes involved in regulation of protein levels. Although protein levels can be affected by genetic variants, their estimation can also be biased by missense variants in coding exons causing technical artifacts. Integrating genome sequence genotype data with mass spectrometry-based protein level estimation could reduce bias, thereby improving detection of variation that affects RNA or protein metabolism. Here, we integrate the blood plasma protein levels of 664 proteins from 165 participants of the Tromsø Study, measured via tandem mass tag mass spectrometry, with whole-exome sequencing data to identify common and rare genetic variation associated with peptide and protein levels (protein quantitative trait loci [pQTLs]). We additionally use literature and database searches to prioritize putative functional variants for each pQTL. We identify 109 independent associations (36 protein and 73 peptide) and use genotype data to exclude 49 (4 protein and 45 peptide) as technical artifacts. We describe 2 particular cases of rare variation: 1 associated with the complement pathway and 1 with platelet degranulation. We identify putative functional variants and show that pQTLs act through diverse molecular mechanisms that affect both RNA and protein metabolism. We show that although the majority of pQTLs exert their effects by modulating RNA metabolism, many affect protein levels directly. Our work demonstrates the extent by which pQTL studies are affected by technical artifacts and highlights how prioritizing the functional variant in pQTL studies can lead to insights into the molecular steps by which a protein may be regulated.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/27295212',\n",
       "   'title': 'Integration of multi-omics data for prediction of phenotypic traits using random forest.',\n",
       "   'abstract': 'In order to find genetic and metabolic pathways related to phenotypic traits of interest, we analyzed gene expression data, metabolite data obtained with GC-MS and LC-MS, proteomics data and a selected set of tuber quality phenotypic data from a diploid segregating mapping population of potato. In this study we present an approach to integrate these\\u2009~\\u2009omics data sets for the purpose of predicting phenotypic traits. This gives us networks of relatively small sets of interrelated\\u2009~\\u2009omics variables that can predict, with higher accuracy, a quality trait of interest. We used Random Forest regression for integrating multiple\\u2009~\\u2009omics data for prediction of four quality traits of potato: tuber flesh colour, DSC onset, tuber shape and enzymatic discoloration. For tuber flesh colour beta-carotene hydroxylase and zeaxanthin epoxidase were ranked first and forty-fourth respectively both of which have previously been associated with flesh colour in potato tubers. Combining all the significant genes, LC-peaks, GC-peaks and proteins, the variation explained was 75\\xa0%, only slightly more than what gene expression or LC-MS data explain by themselves which indicates that there are correlations among the variables across data sets. For tuber shape regressed on the gene expression, LC-MS, GC-MS and proteomics data sets separately, only gene expression data was found to explain significant variation. For DSC onset, we found 12 significant gene expression, 5 metabolite levels (GC) and 2 proteins that are associated with the trait. Using those 19 significant variables, the variation explained was 45\\xa0%. Expression QTL (eQTL) analyses showed many associations with genomic regions in chromosome 2 with also the highest explained variation compared to other chromosomes. Transcriptomics and metabolomics analysis on enzymatic discoloration after 5\\xa0min resulted in 420 significant genes and 8 significant LC metabolites, among which two were putatively identified as caffeoylquinic acid methyl ester and tyrosine. In this study, we made a strategy for selecting and integrating multiple\\u2009~\\u2009omics data using random forest method and selected representative individual peaks for networks based on eQTL, mQTL or pQTL information. Network analysis was done to interpret how a particular trait is associated with gene expression, metabolite and protein data.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39284855',\n",
       "   'title': \"Genomic study of taste perception genes in African Americans reveals SNPs linked to Alzheimer's disease.\",\n",
       "   'abstract': \"While previous research has shown the potential links between taste perception pathways and brain-related conditions, the area involving Alzheimer's disease remains incompletely understood. Taste perception involves neurotransmitter signaling, including serotonin, glutamate, and dopamine. Disruptions in these pathways are implicated in neurodegenerative diseases. The integration of olfactory and taste signals in flavor perception may impact brain health, evident in olfactory dysfunction as an early symptom in neurodegenerative conditions. Shared immune response and inflammatory pathways may contribute to the association between altered taste perception and conditions like neurodegeneration, present in Alzheimer's disease. This study consists of an exploration of expression-quantitative trait loci (eQTL), utilizing whole-blood transcriptome profiles, of 28 taste perception genes, from a combined cohort of 475 African American subjects. This comprehensive dataset was subsequently intersected with single-nucleotide polymorphisms (SNPs) identified in Genome-Wide Association Studies (GWAS) of Alzheimer's Disease (AD). Finally, the investigation delved into assessing the association between eQTLs reported in GWAS of AD and the profiles of 741 proteins from the Olink Neurological Panel. The eQTL analysis unveiled 3,547 statistically significant SNP-Gene associations, involving 412 distinct SNPs that spanned all 28 taste genes. In 17 GWAS studies encompassing various traits, a total of 14 SNPs associated with 12 genes were identified, with three SNPs consistently linked to Alzheimer's disease across four GWAS studies. All three SNPs demonstrated significant associations with the down-regulation of TAS2R41, and two of them were additionally associated with the down-regulation of TAS2R60. In the subsequent pQTL analysis, two of the SNPs linked to TAS2R41 and TAS2R60 genes (rs117771145 and rs10228407) were correlated with the upregulation of two proteins, namely EPHB6 and ADGRB3. Our investigation introduces a new perspective to the understanding of Alzheimer's disease, emphasizing the significance of bitter taste receptor genes in its pathogenesis. These discoveries set the stage for subsequent research to delve into these receptors as promising avenues for both intervention and diagnosis. Nevertheless, the translation of these genetic insights into clinical practice requires a more profound understanding of the implicated pathways and their pertinence to the disease's progression across diverse populations.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/21346809',\n",
       "   'title': 'Genome-wide linkage analysis of global gene expression in loin muscle tissue identifies candidate genes in pigs.',\n",
       "   'abstract': 'Nearly 6,000 QTL have been reported for 588 different traits in pigs, more than in any other livestock species. However, this effort has translated into only a few confirmed causative variants. A powerful strategy for revealing candidate genes involves expression QTL (eQTL) mapping, where the mRNA abundance of a set of transcripts is used as the response variable for a QTL scan. We utilized a whole genome expression microarray and an F(2) pig resource population to conduct a global eQTL analysis in loin muscle tissue, and compared results to previously inferred phenotypic QTL (pQTL) from the same experimental cross. We found 62 unique eQTL (FDR <10%) and identified 3 gene networks enriched with genes subject to genetic control involved in lipid metabolism, DNA replication, and cell cycle regulation. We observed strong evidence of local regulation (40 out of 59 eQTL with known genomic position) and compared these eQTL to pQTL to help identify potential candidate genes. Among the interesting associations, we found aldo-keto reductase 7A2 (AKR7A2) and thioredoxin domain containing 12 (TXNDC12) eQTL that are part of a network associated with lipid metabolism and in turn overlap with pQTL regions for marbling, % intramuscular fat (% fat) and loin muscle area on Sus scrofa (SSC) chromosome 6. Additionally, we report 13 genomic regions with overlapping eQTL and pQTL involving 14 local eQTL. Results of this analysis provide novel candidate genes for important complex pig phenotypes.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39191728',\n",
       "   'title': 'Identifying novel proteins underlying bipolar disorder via integrating pQTLs of the plasma, CSF, and brain with GWAS summary data.',\n",
       "   'abstract': 'Bipolar disorder (BD) presents a significant challenge due to its chronic and relapsing nature, with its underlying pathogenesis remaining elusive. This study employs Mendelian randomization (MR), a widely recognized genetic approach, to unveil intricate causal associations between proteins and BD, leveraging protein quantitative trait loci (pQTL) as key exposures. We integrate pQTL data from brain, cerebrospinal fluid (CSF), and plasma with genome-wide association study (GWAS) findings of BD within a comprehensive systems analysis framework. Our analyses, including two-sample MR, Steiger filtering, and Bayesian colocalization, reveal noteworthy associations. Elevated levels of AGRP, FRZB, and IL36A in CSF exhibit significant associations with increased BD_ALL risk, while heightened levels of CTSF and LRP8 in CSF, and FLRT3 in plasma, correlate with decreased BD_ALL risk. Specifically for Bipolar I disorder (BD_I), increased CSF AGRP levels are significantly linked to heightened BD_I risk, whereas elevated CSF levels of CTSF and LRP8, and plasma FLRT3, are associated with reduced BD_I risk. Notably, genes linked to BD-related proteins demonstrate substantial enrichment in functional pathways such as \"antigen processing and presentation,\" \"metabolic regulation,\" and \"regulation of myeloid cell differentiation.\" In conclusion, our findings provide beneficial evidence to support the potential causal relationship between IL36A, AGRP, FRZB, LRP8 in cerebrospinal fluid, and FLRT3 in plasma, and BD and BD_I, providing insights for future mechanistic studies and therapeutic development.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39358768',\n",
       "   'title': 'Association between lipid-lowering agents with intervertebral disc degeneration, sciatica and low back pain: a drug-targeted mendelian randomized study and cross-sectional observation.',\n",
       "   'abstract': \"Abnormal lipid metabolism is linked to intervertebral disc degeneration (IVDD), sciatica, and low back pain (LBP), but it remains unclear whether targeted interventions can prevent these issues. This study investigated the causal effects of lipid-lowering drug use on IVDD, sciatica, and LBP development. Single-nucleotide polymorphisms (SNPs) linked to total cholesterol (TC), low-density-lipoprotein cholesterol (LDL-C), and non-high-density-lipoprotein cholesterol (non-HDL-C) were obtained from the Global Lipids Genetics Consortium's genome-wide association study (GWAS). Genes near HMGCR, PCSK9, and NPC1L1 were selected to represent therapeutic inhibition targets. Using Mendelian randomization (MR) focusing on these drug targets, we identified causal effects of PCSK9, HMGCR, and NPC1L1 on the risk of developing IVDD, sciatica, and LBP, with coronary heart disease risk serving as a positive control. Using summary data from Mendelian randomization (SMR) analysis, we evaluated potential therapeutic targets for IVDD, sciatica, and LBP through protein quantitative trait loci (pQTL). The genetic associations with IVDD, sciatica, LBP, and coronary heart disease were derived from FinnGen (discovery) and UK Biobank (replication). Additionally, a cross-sectional observational study was performed using data from the National Health and Nutrition Examination Survey (NHANES) to further investigate the connection between LBP and statin use, with a sample size of 4343 participants. Odds ratios (ORs) and corresponding 95% confidence intervals (CIs) were calculated to assess the outcomes. The NHANES-based cross-sectional study indicated that non-statin use was associated with an increased risk of developing LBP (OR\\u2009=\\u20091.29, 95% CI [1.04, 1.59], P\\u2009=\\u20090.019). Moreover, Inverse-variance weighting (IVW) analysis revealed that NPC1L1-mediated reductions in TC, LDL-C, and non-HDL-C concentrations were associated with a decreased risk of developing IVDD (P\\u2009=\\u20099.956E-03; P\\u2009=\\u20093.516E-02; P\\u2009=\\u20091.253E-04). Similarly, PCSK9-mediated reductions in LDL-C and TC concentrations were linked to a lower risk of developing sciatica (P\\u2009=\\u20093.825E-02; P\\u2009=\\u20092.709E-02). Sensitivity analysis confirmed the stability and reliability of the MR results. MST1 (macrophage stimulating 1) levels was inversely associated with IVDD, sciatica, and LBP risks. The results of cross-sectional study suggested that non-use of statins was positively correlated with LBP. The results of Mendelian randomization study suggest that NPC1L1 could lower the risk of developing IVDD by reducing TC, LDL-C, and non-HDL-C levels. Additionally, PCSK9 may reduce the risk of developing sciatica by lowering LDL-C and TC levels. In contrast, HMGCR appears to have no significant effect on IVDD, sciatica, or LBP development. Nonetheless, further research is needed to verify these preliminary results. MST1 warrants further exploration as a potential therapeutic target. It is necessary to do further research to validate these findings.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39215100',\n",
       "   'title': 'µMap proximity labeling in living cells reveals stress granule disassembly mechanisms.',\n",
       "   'abstract': 'Phase-separated condensates are membrane-less intracellular structures comprising dynamic protein interactions that organize essential biological processes. Understanding the composition and dynamics of these organelles advances our knowledge of cellular behaviors and disease pathologies related to granule dysregulation. In this study, we apply microenvironment mapping with a HaloTag-based platform (HaloMap) to characterize intracellular stress granule dynamics in living cells. After validating the robustness and sensitivity of this approach, we then profile the stress granule proteome throughout the formation and disassembly and under pharmacological perturbation. These experiments reveal several ubiquitin-related modulators, including the HECT (homologous to E6AP C terminus) E3 ligases ITCH and NEDD4L, as well as the ubiquitin receptor toll-interacting protein TOLLIP, as key mediators of granule disassembly. In addition, we identify an autophagy-related pathway that promotes granule clearance. Collectively, this work establishes a general photoproximity labeling approach for unraveling intracellular protein interactomes and uncovers previously unexplored regulatory mechanisms of stress granule dynamics.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39217396',\n",
       "   'title': 'Genetic causal association between lipidomic profiles, inflammatory proteomics, and aortic stenosis: a Mendelian randomization investigation.',\n",
       "   'abstract': 'Aortic stenosis (AS) is a prevalent and serious valvular heart disease with a complex etiology involving genetic predispositions, lipid dysregulation, and inflammation. The specific roles of lipid and protein biomarkers in AS development are not fully elucidated. This study aimed to elucidate the causal relationships between lipidome, inflammatory proteins, and AS using Mendelian randomization (MR), identifying potential therapeutic targets. Utilizing data from large-scale genome-wide association studies (GWAS) and genome-wide protein quantitative trait loci (pQTL) studies, we conducted MR analyses on 179 plasma lipidome and 91 inflammatory proteins to assess their causal associations with AS. Our approach included Inverse Variance Weighting (IVW), Wald ratio, and robust adjusted profile score (RAPS) analyses to refine these associations. MR-Egger regression was used to address directional horizontal pleiotropy. Our MR analysis showed that genetically predicted 50 lipids were associated with AS, including 38 as risk factors [(9 Sterol ester, 18 Phosphatidylcholine, 4 Phosphatidylethanolamine, 1 Phosphatidylinositol and 6 Triacylglycerol)] and 12 as protective. Sterol ester (27:1/17:1) emerged as the most significant risk factor with an odds ratio (OR) of 3.11. Additionally, two inflammatory proteins, fibroblast growth factor 19 (FGF19) (OR\\u2009=\\u20090.830, P\\u2009=\\u20090.015), and interleukin 6 (IL-6) (OR\\u2009=\\u20090.729, P\\u2009=\\u20091.79E-04) were significantly associated with reduced AS risk. However, a two-step MR analysis showed no significant mediated correlations between these proteins and the lipid-AS pathway. This study reveals complex lipid and protein interactions in AS, identifying potential molecular targets for therapy. These results go beyond traditional lipid profiling and significantly advance our genetic and molecular understanding of AS, highlighting potential pathways for intervention and prevention.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39076991',\n",
       "   'title': 'Integrated analyses of single-cell transcriptome and Mendelian randomization reveal the protective role of FCRL3 in multiple sclerosis.',\n",
       "   'abstract': 'Multiple sclerosis (MS) represents a multifaceted autoimmune ailment, prompting the development and widespread utilization of numerous therapeutic interventions. However, extant medications for MS have proven inadequate in mitigating relapses and halting disease progression. Innovative drug targets for preventing multiple sclerosis are still required. The objective of this study is to discover novel therapeutic targets for MS by integrating single-cell transcriptomics and Mendelian randomization analysis. The study integrated MS genome-wide association study (GWAS) data, single-cell transcriptomics (scRNA-seq), expression quantitative trait loci (eQTL), and protein quantitative trait loci (pQTL) data for analysis and utilized two-sample Mendelian randomization study to comprehend the causal relationship between proteins and MS. Sequential analyses involving colocalization and Phenome-wide association studies (PheWAS) were conducted to validate the causal role of candidate genes. Following stringent quality control preprocessing of scRNA-seq data, 1,123 expression changes across seven peripheral cell types were identified. Among the seven most prevalent cell types, 97 genes exhibiting at least one eQTL were discerned. Examination of MR associations between 28 proteins with available index pQTL signals and the risk of MS outcomes was conducted. Co-localization analyses and PheWAS indicated that FCRL3 may exert influence on MS. The integration of scRNA-seq and MR analysis facilitated the identification of potential therapeutic targets for MS. Notably, FCRL3, implicated in immune function, emerged as a significant drug target in the deCODE databases. This research underscores the importance of FCRL3 in MS therapy and advocates for further investigation and clinical trials targeting FCRL3.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39512814',\n",
       "   'title': 'CDKN1A as a target of senescence in heart failure: insights from a multiomics study.',\n",
       "   'abstract': 'Cardiomyocyte senescence plays a crucial role as a pathological mechanism in heart failure (HF). However, the exact triggering factors and underlying causes of HF onset and progression are still not fully understood. By integrating multi-omics data, this study aimed to determine the genetic associations between cardiomyocyte and HF using cell senescence-related genes (SRGs). The study utilized the CellAge database and the SenMayo dataset, combined with high-resolution single-cell RNA sequencing (scRNA-seq) data, to identify SRG and examine differences in cardiac cell expression. To explore the causal relationship with HF using Mendelian Randomization (MR). Genetic variations influencing gene expression, DNA methylation, and protein expression (cis-eQTL, cis-mQTL, and cis-pQTL) were analyzed using the two-sample MR (TSMR) and summary-data-based MR (SMR). Additionally, Bayesian colocalization analysis, germline genetic variation, and bulk RNA data were employed to strengthen the reliability of the results. The application potential of therapeutic targets is ultimately assessed by evaluating their druggability. The expression of 39 SRGs in cardiomyocytes was identified. In the discovery set revealed that  Based on multi-omics data, this study uncovered the reciprocal regulation of cardiomyocyte senescence through '},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39637861',\n",
       "   'title': \"Deciphering proteins in Alzheimer's disease: A new Mendelian randomization method integrated with AlphaFold3 for 3D structure prediction.\",\n",
       "   'abstract': \"Hidden confounding biases hinder identifying causal protein biomarkers for Alzheimer's disease in non-randomized studies. While Mendelian randomization (MR) can mitigate these biases using protein quantitative trait loci (pQTLs) as instrumental variables, some pQTLs violate core assumptions, leading to biased conclusions. To address this, we propose MR-SPI, a novel MR method that selects valid pQTL instruments using Leo Tolstoy's Anna Karenina principle and performs robust post-selection inference. Integrating MR-SPI with AlphaFold3, we developed a computational pipeline to identify causal protein biomarkers and predict 3D structural changes. Applied to genome-wide proteomics data from 54,306\\xa0UK Biobank participants and 455,258 subjects (71,880 cases and 383,378 controls) for a genome-wide association study of Alzheimer's disease, we identified seven proteins (TREM2, PILRB, PILRA, EPHA1, CD33, RET, and CD55) with structural alterations due to missense mutations. These findings offer insights into the etiology and potential drug targets for Alzheimer's disease.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/37563310',\n",
       "   'title': 'Genetics of circulating inflammatory proteins identifies drivers of immune-mediated disease risk and therapeutic targets.',\n",
       "   'abstract': 'Circulating proteins have important functions in inflammation and a broad range of diseases. To identify genetic influences on inflammation-related proteins, we conducted a genome-wide protein quantitative trait locus (pQTL) study of 91 plasma proteins measured using the Olink Target platform in 14,824 participants. We identified 180 pQTLs (59 cis, 121 trans). Integration of pQTL data with eQTL and disease genome-wide association studies provided insight into pathogenesis, implicating lymphotoxin-α in multiple sclerosis. Using Mendelian randomization (MR) to assess causality in disease etiology, we identified both shared and distinct effects of specific proteins across immune-mediated diseases, including directionally discordant effects of CD40 on risk of rheumatoid arthritis versus multiple sclerosis and inflammatory bowel disease. MR implicated CXCL5 in the etiology of ulcerative colitis (UC) and we show elevated gut CXCL5 transcript expression in patients with UC. These results identify targets of existing drugs and provide a powerful resource to facilitate future drug target prioritization.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38302916',\n",
       "   'title': 'Circadian clock-related genome-wide mendelian randomization identifies putatively genes for ulcerative colitis and its comorbidity.',\n",
       "   'abstract': 'Circadian rhythm is crucial to the function of the immune system. Disorders of the circadian rhythm can contribute to inflammatory diseases such as Ulcerative colitis (UC). This Mendelian Randomization (MR) analysis applies genetic tools to represent the aggregated statistical results of exposure to circadian rhythm disorders and UC and its comorbidities, allowing for causal inferences. Summary statistics of protein, DNA methylation and gene expression quantitative trait loci in individuals of European ancestry (pQTL, mQTL, and eQTL, respectively) were used. Genetic variants located within or near 152 circadian clock-related genes and closely related to circadian rhythm disorders were selected as instrumental variables. Causal relationships with UC and its comorbidities were then estimated through employed Summary data-based Mendelian Randomization (SMR) and Inverse-Variance-Weighted MR (IVW-MR). Through preliminary SMR analysis, we identified a potential causal relationship between circadian clock-related genes and UC along with its comorbidities, which was further confirmed by IVW-MR analysis. Our study identified strong evidence of positive correlation involving seven overlapping genes (CSNK1E, OPRL1, PIWIL2, RORC, MAX, PPP5C, and AANAT) through MWAS and TWAS in UC, four overlapping genes (OPRL1, CHRNB2, FBXL17, and SIRT1) in UC with PSC, and three overlapping genes (ARNTL, USP7, and KRAS) in UC with arthropathy. This SMR study demonstrates the causal effect of circadian rhythm disorders in UC and its comorbidities. Furthermore, our investigation pinpointed candidate genes that could potentially serve as drug targets.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/36168886',\n",
       "   'title': 'Differences and commonalities in the genetic architecture of protein quantitative trait loci in European and Arab populations.',\n",
       "   'abstract': 'Polygenic scores (PGS) can identify individuals at risk of adverse health events and guide genetics-based personalized medicine. However, it is not clear how well PGS translate between different populations, limiting their application to well-studied ethnicities. Proteins are intermediate traits linking genetic predisposition and environmental factors to disease, with numerous blood circulating protein levels representing functional readouts of disease-related processes. We hypothesized that studying the genetic architecture of a comprehensive set of blood-circulating proteins between a European and an Arab population could shed fresh light on the translatability of PGS to understudied populations. We therefore conducted a genome-wide association study with whole-genome sequencing data using 1301 proteins measured on the SOMAscan aptamer-based affinity proteomics platform in 2935 samples of Qatar Biobank and evaluated the replication of protein quantitative traits (pQTLs) from European studies in an Arab population. Then, we investigated the colocalization of shared pQTL signals between the two populations. Finally, we compared the performance of protein PGS derived from a Caucasian population in a European and an Arab cohort. We found that the majority of shared pQTL signals (81.8%) colocalized between both populations. About one-third of the genetic protein heritability was explained by protein PGS derived from a European cohort, with protein PGS performing ~20% better in Europeans when compared to Arabs. Our results are relevant for the translation of PGS to non-Caucasian populations, as well as for future efforts to extend genetic research to understudied populations.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/38660492',\n",
       "   'title': 'Mendelian randomization analysis identifies druggable genes and drugs repurposing for chronic obstructive pulmonary disease.',\n",
       "   'abstract': 'Chronic obstructive pulmonary disease (COPD) is a prevalent condition that significantly impacts public health. Unfortunately, there are few effective treatment options available. Mendelian randomization (MR) has been utilized to repurpose existing drugs and identify new therapeutic targets. The objective of this study is to identify novel therapeutic targets for COPD. Cis-expression quantitative trait loci (cis-eQTL) were extracted for 4,317 identified druggable genes from genomics and proteomics data of whole blood (eQTLGen) and lung tissue (GTEx Consortium). Genome-wide association studies (GWAS) data for doctor-diagnosed COPD, spirometry-defined COPD (Forced Expiratory Volume in one second [FEV1]/Forced Vital Capacity [FVC] <0.7), and FEV1 were obtained from the cohort of FinnGen, UK Biobank and SpiroMeta consortium. We employed Summary-data-based Mendelian Randomization (SMR), HEIDI test, and colocalization analysis to assess the causal effects of druggable gene expression on COPD and lung function. The reliability of these druggable genes was confirmed by eQTL two-sample MR and protein quantitative trait loci (pQTL) SMR, respectively. The potential effects of druggable genes were assessed through the phenome-wide association study (PheWAS). Information on drug repurposing for COPD was collected from multiple databases. A total of 31 potential druggable genes associated with doctor-diagnosed COPD, spirometry-defined COPD, and FEV1 were identified through SMR, HEIDI test, and colocalization analysis. Among them, 22 genes (e.g., MMP15, PSMA4, ERBB3, and LMCD1) were further confirmed by eQTL two-sample MR and protein SMR analyses. Gene-level PheWAS revealed that ERBB3 expression might reduce inflammation, while GP9 and MRC2 were associated with other traits. The drugs Montelukast (targeting the MMP15 gene) and MARIZOMIB (targeting the PSMA4 gene) may reduce the risk of spirometry-defined COPD. Additionally, an existing small molecule inhibitor of the APH1A gene has the potential to increase FEV Our findings identified 22 potential drug targets for COPD and lung function. Prioritizing clinical trials that target these identified druggable genes with existing drugs or novel medications will be beneficial for the development of COPD treatments.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39911230',\n",
       "   'title': 'Identification of potential druggable targets for endometriosis through Mendelian randomization analysis.',\n",
       "   'abstract': 'Endometriosis (EM) is a widely recognized disorder in gynecological endocrinology. Although hormonal therapies are frequently employed for EM, their side effects and outcome limitations underscore the need to explore the genetic basis and potential drug targets for developing innovative therapeutic approaches. This study aimed to identify both cerebrospinal fluid (CSF) and plasma protein markers as promising therapeutic targets for EM. We utilized Mendelian randomization (MR) analysis to explore potential disease-causing proteins, utilizing genetic datasets from genome-wide association studies (GWAS) and protein quantitative trait loci (pQTL) analyses. We applied a range of validation techniques, including reverse causality detection, phenotype scanning, Bayesian co-localization (BC) analysis, and external validations to substantiate our findings. Additionally, we conducted a protein-protein interaction (PPI) network as well as functional enrichment analyses to unveil potential associations among target proteins. MR analysis revealed that a decrease of one standard deviation (SD) in plasma R-Spondin 3 (RSPO3) level had a protective effect on EM (OR = 1.0029; 95% confidence interval (95% CI): 1.0015-1.0043; P = 3.2567e-05; Bonferroni P < 5.63 × 10 In conclusion, this comprehensive study offers valuable insights into potential drug targets for EM, with RSPO3 emerging as a promising candidate. Additionally, mechanistic roles of FN1, glycan degradation pathway, LGALS3, CPE, and FUT5 in EM warrant further investigation.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/35292653',\n",
       "   'title': 'Mapping the sequence specificity of heterotypic amyloid interactions enables the identification of aggregation modifiers.',\n",
       "   'abstract': 'Heterotypic amyloid interactions between related protein sequences have been observed in functional and disease amyloids. While sequence homology seems to favour heterotypic amyloid interactions, we have no systematic understanding of the structural rules determining such interactions nor whether they inhibit or facilitate amyloid assembly. Using structure-based thermodynamic calculations and extensive experimental validation, we performed a comprehensive exploration of the defining role of sequence promiscuity in amyloid interactions. Using tau as a model system we demonstrate that proteins with local sequence homology to tau amyloid nucleating regions can modify fibril nucleation, morphology, assembly and spreading of aggregates in cultured cells. Depending on the type of mutation such interactions inhibit or promote aggregation in a manner that can be predicted from structure. We find that these heterotypic amyloid interactions can result in the subcellular mis-localisation of these proteins. Moreover, equilibrium studies indicate that the critical concentration of aggregation is altered by heterotypic interactions. Our findings suggest a structural mechanism by which the proteomic background can modulate the aggregation propensity of amyloidogenic proteins and we\\xa0discuss how such sequence-specific proteostatic perturbations could contribute to the selective cellular susceptibility of amyloid disease progression.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32645325',\n",
       "   'title': 'The Global Phosphorylation Landscape of SARS-CoV-2 Infection.',\n",
       "   'abstract': 'The causative agent of the coronavirus disease 2019 (COVID-19) pandemic, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has infected millions and killed hundreds of thousands of people worldwide, highlighting an urgent need to develop antiviral therapies. Here we present a quantitative mass spectrometry-based phosphoproteomics survey of SARS-CoV-2 infection in Vero E6 cells, revealing dramatic rewiring of phosphorylation on host and viral proteins. SARS-CoV-2 infection promoted casein kinase II (CK2) and p38 MAPK activation, production of diverse cytokines, and shutdown of mitotic kinases, resulting in cell cycle arrest. Infection also stimulated a marked induction of CK2-containing filopodial protrusions possessing budding viral particles. Eighty-seven drugs and compounds were identified by mapping global phosphorylation profiles to dysregulated kinases and pathways. We found pharmacologic inhibition of the p38, CK2, CDK, AXL, and PIKFYVE kinases to possess antiviral efficacy, representing potential COVID-19 therapies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39935908',\n",
       "   'title': \"Genetic and Plasma Proteomic Approaches to Identify Therapeutic Targets for Graves' Disease and Graves' Ophthalmopathy.\",\n",
       "   'abstract': \"The blood proteome is a major source of biomarkers and therapeutic targets. We aimed to identify the causal proteins and potential targets for Graves' disease (GD) and Graves' ophthalmopathy (GO) via systematic genetic analyses. Genome-wide association studies (GWASs) on the UK Biobank- Pharma Proteomics Project (UKB-PPP) collected 2923 Olink proteins from 54,219 participants. We conducted a proteome-wide Mendelian randomization (MR) study with cis-pQTLs to identify candidate proteins for GD and GO risk. Colocalization analysis and the Heidi test were used to examine whether the identified proteins and diseases shared the same variant. More proteins with potential causal associations were identified in Summary-data-based MR (SMR) analyses using trans-pQTLs. Then, downstream analyses were performed to detect protein interactions, gene function, cell type-specific expression and druggable information. This study genetically predicted levels of 62 plasma proteins were associated with GD risk. Four proteins (CD40, TINAGL1, GMPR and CXCL10) were prioritized with the evidence of sharing the same variants with GD. Specifically, some proteins had potential associations with GD with trans-pQTLs mapping in CD40. The four prioritized protein-coding genes were mainly enriched in the regulation of apoptotic and death processes. In addition, GMPR was associated with both GO and GD in a consistent direction. BTN1A1 and FCRL1 were prioritized as the causal proteins for GO onset and were not associated with GD. By synthesizing proteomic and genetic data, we identified several protein biomarkers for GD, with one linked to both GD and GO and two other protein biomarkers specific to GO onset, which provides valuable insights into the etiology and potential therapeutic targets for the two diseases.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30111768',\n",
       "   'title': 'Genome-wide mapping of plasma protein QTLs identifies putatively causal genes and pathways for cardiovascular disease.',\n",
       "   'abstract': \"Identifying genetic variants associated with circulating protein concentrations (protein quantitative\\xa0trait loci; pQTLs) and integrating them with variants from genome-wide association studies (GWAS) may illuminate the proteome's causal role in disease and bridge a knowledge gap regarding SNP-disease associations. We provide the results of GWAS of 71 high-value cardiovascular disease proteins in 6861 Framingham Heart Study participants and independent external replication. We report the mapping of over 16,000 pQTL variants and their functional relevance. We provide an integrated plasma protein-QTL database. Thirteen proteins harbor pQTL variants that match coronary disease-risk variants from GWAS or test causal for coronary disease by Mendelian randomization. Eight\\xa0of these proteins predict new-onset cardiovascular disease events in Framingham participants. We demonstrate that identifying pQTLs, integrating them with GWAS results, employing Mendelian randomization, and prospectively testing protein-trait associations holds potential for elucidating causal genes, proteins, and pathways for cardiovascular disease and may identify targets for its prevention and treatment.\"},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/32474106',\n",
       "   'title': 'Genetic variants entail type 2 diabetes as an innate immune disorder.',\n",
       "   'abstract': 'Type 2 Diabetes (T2D) is characterized by alteration in the circulatory levels of key inflammatory proteins, where our body strives to eliminate the perturbing factor through inflammation as a final resort to restore homeostasis. Plasma proteins play a crucial role to orchestrate this immune response. Over the past two decades, rigorous genetic efforts taken to comprehend T2D physiology have been partially successful and have left behind a dearth of knowledge of its causality. Here, we have investigated how the reported genetic variants of T2D are associated with circulatory levels of key plasma proteins. We identified 99 T2D genetic variants that serve as strong pQTL (protein Quantitative Trait Loci) for 72 plasma proteins, of which 4 proteins namely Small nuclear ribonucleoprotein F [SNRPF] (p\\xa0=\\xa02.99\\xa0×\\xa010'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/39680155',\n",
       "   'title': 'Identification of potential pathogenic genes for urolithiasis through multi-omics Mendelian randomization analysis.',\n",
       "   'abstract': 'Urolithiasis, a common urological disorder affecting about 10% of the global population, is known for its high recurrence rate, yet its genetic mechanisms remain poorly understood. This study aimed to fill this gap by identifying potential pathogenic genes associated with urolithiasis using a multi-omics Mendelian randomization approach. We conducted a comprehensive analysis that integrated genome-wide association studies (GWAS), expression quantitative trait loci (eQTL), methylation quantitative trait loci (mQTL), and protein quantitative trait loci (pQTL) data. Summary Data-Based Mendelian Randomization (SMR) and Bayesian colocalization analyses were employed to investigate causal relationships between gene expression and urolithiasis, while external validation and multivariable MR controlled for confounding factors. Seven genes were identified as significantly associated with urolithiasis, with LMAN2, NUCKS1, and L3MBTL3 highlighted as key contributors. LMAN2 was positively associated with urolithiasis risk (SMR b\\u2009=\\u20090.842, FDR\\u2009<\\u20090.05), with evidence that increased LMAN2 expression elevates stone formation likelihood, supported by findings from DNA methylation and protein level analyses. Conversely, NUCKS1 and L3MBTL3 exhibited protective effects, with NUCKS1 expression negatively associated with urolithiasis and supported by methylation at the cg12081870 site. Bayesian colocalization analysis showed strong shared genetic bases for NUCKS1 and L3MBTL3 with urolithiasis, with further multivariable MR confirming these associations were independent of BMI, smoking, alcohol consumption, and serum calcium levels. Genetic correlation analysis revealed significant positive genetic correlations between LMAN2 and urolithiasis (rg\\u2009=\\u20091.12, P\\u2009=\\u20098.11e-11), while NUCKS1 (rg\\u2009=\\u2009-\\xa00.60, P\\u2009=\\u20093.10e-03) and L3MBTL3 (rg\\u2009=\\u2009-\\xa00.38, P\\u2009=\\u20091.20e-03) showed strong negative correlations. These findings provide critical insights into the genetic basis of urolithiasis, identifying LMAN2, NUCKS1, and L3MBTL3 as potential biomarkers and therapeutic targets, offering a pathway toward personalized treatment strategies.'},\n",
       "  {'pid': 'http://www.ncbi.nlm.nih.gov/pubmed/30683686',\n",
       "   'title': 'Human Testis Phosphoproteome Reveals Kinases as Potential Targets in Spermatogenesis and Testicular Cancer.',\n",
       "   'abstract': 'Spermatogenesis is a complex cell differentiation process that includes marked genetic, cellular, functional and structural changes. It requires tight regulation, because disturbances in any of the spermatogenic processes would lead to fertility deficiencies as well as disorders in offspring. To increase our knowledge of signal transduction during sperm development, we carried out a large-scale identification of the phosphorylation events that occur in the human male gonad. Metal oxide affinity chromatography using TiO'}]}"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load json file\n",
    "def load_data(file_path):\n",
    "    with open(file_path, 'r', encoding='utf-8') as f:\n",
    "        data = json.load(f)\n",
    "\n",
    "    return data['data']\n",
    "\n",
    "# test training daya\n",
    "#input_file_data = 'datasets/test_golden_answers/batch_1/parsed_data_final_test_batch_1.json'\n",
    "# input_file_data = 'datasets/test/test_golden_answers/batch_2/retrieved_articles_sampled_test_batch_2.json'\n",
    "# input_file_data = 'datasets/test/test_golden_answers/batch_3/retrieved_articles_sampled_test_batch_3.json'\n",
    "\n",
    "# training data\n",
    "#input_file_data = 'datasets/training/parsed_data_final.json'\n",
    "\n",
    "# test not golden\n",
    "input_file_data = \"datasets/test/batch_3/retrieved_articles_sampled_test_batch_3.json\"\n",
    "\n",
    "# test output files\n",
    "#output_file = 'results/bm25_results_test_batch_1.json'\n",
    "# output_file = 'results/bm25_results_test_batch_2.json'\n",
    "# output_file = 'results/bm25_results_test_batch_3.json'\n",
    "\n",
    "# training output files\n",
    "output_file ='results/bm25_results_batch_3_not_golden.json'\n",
    "\n",
    "\n",
    "\n",
    "data = load_data(file_path=input_file_data)\n",
    "\n",
    "data[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "f1008c4a",
   "metadata": {},
   "outputs": [],
   "source": [
    "def tokenize(text):\n",
    "    text = text.lower()\n",
    "    text = re.sub(r'[^a-z0-9\\s]', '', text)\n",
    "    return text.split()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "165bb53e",
   "metadata": {},
   "outputs": [],
   "source": [
    "def prepare_corpus(articles):\n",
    "    \"\"\"\n",
    "    Prepares tokenized documents for BM25 from article title + abstract.\n",
    "    Returns (tokenized_corpus, article_refs)\n",
    "    \"\"\"\n",
    "    corpus = []\n",
    "    article_refs = []\n",
    "\n",
    "    for article in articles:\n",
    "        title = article.get('title', '')\n",
    "        abstract = article.get('abstract', '')\n",
    "        text = f\"{title} {abstract}\".strip()\n",
    "\n",
    "        if not text:\n",
    "            continue\n",
    "\n",
    "        tokens = tokenize(text)\n",
    "        if tokens:\n",
    "            corpus.append(tokens)\n",
    "            article_refs.append(article)  # store original for scoring\n",
    "\n",
    "    return corpus, article_refs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "2665f776",
   "metadata": {},
   "outputs": [],
   "source": [
    "def rank_articles_bm25(question, articles):\n",
    "    corpus, article_refs = prepare_corpus(articles)\n",
    "    if not corpus:\n",
    "        return []\n",
    "\n",
    "    bm25 = BM25Okapi(corpus)\n",
    "    query = tokenize(question)\n",
    "\n",
    "    scores = bm25.get_scores(query)\n",
    "    ranked_indices = sorted(range(len(scores)), key=lambda i: scores[i], reverse=True)\n",
    "\n",
    "    top_10 = []\n",
    "    for i in ranked_indices[:10]:\n",
    "        article = article_refs[i]\n",
    "        top_10.append({\n",
    "            'pid': article.get('pid', ''),\n",
    "            'title': article.get('title', ''),\n",
    "            'abstract': article.get('abstract', ''),\n",
    "            'score': float(scores[i])\n",
    "        })\n",
    "\n",
    "    return top_10"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "140b78a2",
   "metadata": {},
   "outputs": [],
   "source": [
    "def save_results(results, output_file):\n",
    "    with open(output_file, 'w', encoding='utf-8') as f:\n",
    "        json.dump(results, f, indent=2, ensure_ascii=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "bd006e56",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_snippets(question, top_articles):\n",
    "    \"\"\"\n",
    "    Extract BioASQ-style snippets from the top-ranked articles.\n",
    "    Returns a list of snippet dicts with section labels and character offsets.\n",
    "    \"\"\"\n",
    "    query_terms = [re.sub(r'[^a-z0-9]', '', t.lower()) for t in re.findall(r'\\w+', question)]\n",
    "    snippets = []\n",
    "\n",
    "    for article in top_articles:\n",
    "        pid = article.get('pid', '')\n",
    "        #doc_url = f\"https://www.ncbi.nlm.nih.gov/pubmed/{pid}\"\n",
    "\n",
    "        for section in ['title', 'abstract']:\n",
    "            field_text = article.get(section, '')\n",
    "            text_lower = field_text.lower()\n",
    "            text_norm = re.sub(r'[^a-z0-9\\s]', '', text_lower)\n",
    "\n",
    "            # Map normalized text back to original for offset tracking\n",
    "            match_offsets = []\n",
    "            for term in query_terms:\n",
    "                for m in re.finditer(r'\\b' + re.escape(term) + r'\\b', text_norm):\n",
    "                    start, end = m.start(), m.end()\n",
    "                    match_offsets.append((start, end))\n",
    "\n",
    "            if not match_offsets:\n",
    "                continue\n",
    "\n",
    "            snippet_start = min(offset[0] for offset in match_offsets)\n",
    "            snippet_end = max(offset[1] for offset in match_offsets)\n",
    "\n",
    "            # Use original text offsets — approximate based on normalized mapping\n",
    "            snippet_text = field_text[snippet_start:snippet_end].strip()\n",
    "\n",
    "            snippets.append({\n",
    "                \"beginSection\": section,\n",
    "                \"endSection\": section,\n",
    "                \"text\": snippet_text,\n",
    "                \"document\": pid,\n",
    "                \"offsetInBeginSection\": snippet_start,\n",
    "                \"offsetInEndSection\": snippet_end\n",
    "            })\n",
    "\n",
    "    return snippets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "dd241c2e",
   "metadata": {},
   "outputs": [],
   "source": [
    "def rank_all_questions_bm25(data):\n",
    "    results_by_question = []\n",
    "\n",
    "    for entry in tqdm(data, desc=\"Processing questions...\"):\n",
    "        question = entry['question']\n",
    "        articles = entry.get('all_retreived_articles', [])\n",
    "        top_articles = rank_articles_bm25(question, articles)\n",
    "        snippets = extract_snippets(question, top_articles)\n",
    "\n",
    "        results_by_question.append({\n",
    "            'id': entry['qid'],\n",
    "            'question': question,\n",
    "            'top_10_articles': top_articles, \n",
    "            'snippets': snippets\n",
    "        })\n",
    "\n",
    "    return { 'data' : results_by_question }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "5b2a8534",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processing questions...: 100%|██████████| 85/85 [00:01<00:00, 43.69it/s]\n"
     ]
    }
   ],
   "source": [
    "results_bm25 = rank_all_questions_bm25(data=data)\n",
    "save_results(results_bm25, output_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f7eb84d7",
   "metadata": {},
   "source": [
    "Code down will be used to convert the result file to the format wanted for submission on BioASQ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "db27a081",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "816d9e85",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "air_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
